{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import lxml\n",
    "import glob\n",
    "import bs4\n",
    "from lxml import etree \n",
    "import numpy as np\n",
    "from nltk.tokenize import RegexpTokenizer\n",
    "import nltk\n",
    "import string\n",
    "from nltk.corpus import stopwords\n",
    "#https://www.sketchengine.eu/user-guide/user-manual/corpora/compare-corpora/#toggle-id-1 to compare corpora"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###first, we're going to try to parse with etree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#construct paths\n",
    "#note: these are NOT full text. they link to abstracts. \n",
    "pathMAP3K2=\"/Users/liangrj/Desktop/pubmed_result_MAP3K2.xml\"\n",
    "pathMAP3K3=\"/Users/liangrj/Desktop/pubmed_result_MAP3K3.xml\"\n",
    "pathMAP3K5=\"/Users/liangrj/Desktop/pubmed_result_MAP3K5.xml\"\n",
    "pathMAP3K6=\"/Users/liangrj/Desktop/pubmed_result_MAP3K6.xml\"\n",
    "pathMAP3K15=\"/Users/liangrj/Desktop/pubmed_result_MAP3K15.xml\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#open all the corpuses. these read as strings initially, so we need to encode to UTF8 into bytes\n",
    "#could easily do for-loop, but fuck it\n",
    "with open(pathMAP3K2) as f:\n",
    "    MAP3K2corp = f.read().encode('UTF-8')\n",
    "with open(pathMAP3K3) as g:\n",
    "    MAP3K3corp = g.read().encode('UTF-8')\n",
    "with open(pathMAP3K5) as h:\n",
    "    MAP3K5corp = h.read().encode('UTF-8')\n",
    "with open(pathMAP3K6) as i:\n",
    "    MAP3K6corp = i.read().encode('UTF-8')\n",
    "with open(pathMAP3K15) as j:\n",
    "    MAP3K15corp = j.read().encode('UTF-8')\n",
    "with open('/Users/liangrj/Desktop/pubmedsample18n0001.xml') as z:\n",
    "    text = z.read().encode('UTF-8')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#test number of documents in each corpus. I am going to be first using MAP3K5,6,and 15 to determine MAP3K4 function.\n",
    "#thus, I will only use those for now\n",
    "et1 = etree.fromstring(MAP3K5corp) #needs to be either UTF8 OR string to be parsed. NOT soup. \n",
    "et2 = etree.fromstring(MAP3K6corp)\n",
    "et3 = etree.fromstring(MAP3K15corp)\n",
    "articlesMAP3K5_tree=et1.findall('PubmedArticle')\n",
    "articlesMAP3K6_tree=et2.findall('PubmedArticle')\n",
    "articlesMAP3K15_tree=et3.findall('PubmedArticle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def show(e):\n",
    "    print(etree.tostring(e, xml_declaration=True).decode(\"utf-8\")) \n",
    "#try to get the labels of the etree\n",
    "#string_list_MAP3K5=[]\n",
    "#show(articlesMAP3K5[0]) #have to do a for loop to make a list. we can create a series of strings from here. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1 Role of germline aberrations affecting \n",
      "In approximately 10% of all gastric cancer (GC) cases, a heritable cause is suspected. A subset of these cases have a causative germline  CDH1 CTNNA1 MAP3K6 MYD88\n",
      "2 Cyclin-dependent kinase 5 regulates MAPK/ERK signaling in the skin of mice.\n",
      "Cyclin-dependent kinase 5 (CDK5) is a proline-directed serine/threonine kinase that has been shown to play important roles in many tissues except the nervous system. We previously reported that CDK5 showed differential expression in the transcriptome profiles of the skin of alpacas with different hair colors. To understand the functional role of CDK5 in hair color determination, we constructed CDK5-knockdown mice and identified the effect on the mitogen-activated protein kinase (MAPK) pathway in the mouse skin. Quantitative real-time polymerase chain reaction, co-immunoprecipitation, and western blotting were performed to analyze the effects of CDK5-knockdown on the MAPK pathway in mice. The results showed that MAP3K6 was inhibited by phosphorylated CDK5 through its activator CDK7. The decrease in MAP3K6 levels caused down-regulation of MEK1 and ERK expression, leading to the up-regulation of miR-143-3p, which targets MAP3K6 via Dicer. Taken together, our findings indicate that CDK5 functions in regulating the MAPK pathway. Given that MAP3K6 was inhibited in two directions, this mechanism can provide insight into the contributions of the MAPK/ERK pathway to the inhibition of melanin production.\n",
      "3 Genetic predisposition to gastric cancer.\n",
      "Gastric cancer ranks as the third leading cause of cancer mortality worldwide and confers a 5-year survival of 20%. While most gastric cancers are sporadic, ~1%-3% can be attributed to inherited cancer predisposition syndromes. Germline E-cadherin/CDH1 mutations have been identified in families with an autosomal dominant inherited predisposition to diffuse gastric cancer. The cumulative risk of gastric cancer for CDH1 mutation carriers by age 80 years is reportedly 70% for men and 56% for women. Female mutation carriers also have an estimated 42% risk for developing lobular breast cancer by age 80 years. However, most individuals meeting clinical criteria for hereditary diffuse gastric cancer syndrome (HDGC) do not have a germline CDH1 mutation, and germline CDH1 mutation carriers do not all exhibit similar clinical outcomes in terms of age of diagnosis or cancer types. E-cadherin (CDH1) as the one known causative gene for HDGC accounts for only 40% of cases, leaving 60% with an unknown genetic diagnosis. In addition to HDGC, we will review other genetic syndromes with elevated gastric cancer risk, as well as newly implicated alterations in other genes (CTNNA1, DOT1L, FBXO24, PRSS1, MAP3K6, MSR1, and INSR) that may affect gastric cancer susceptibility and age-specific penetrance.\n",
      "4 Hereditary diffuse gastric cancer: What the clinician should know.\n",
      "Hereditary diffuse gastric cancer (HDGC) is an inherited autosomal dominant syndrome with a penetrance of up to 80% affecting diverse geographic populations. While it has been shown to be caused mainly by germline alterations in the E-cadherin gene (CDH1), problematically, the genetic diagnosis remains unknown in up to 60% of patients. Given the important knowledge gaps regarding the syndrome, asymptomatic carriers of CDH1 mutations are advised for a prophylactic total gastrectomy. Intensive annual endoscopic surveillance is the alternative for carriers who decline gastrectomy. As HDGCs have a prolonged indolent phase, this provides a window of opportunity for surveillance and treatment. Recent findings of other gene defects in CTNNA1 and MAP3K6, as well as further characterization of CDH1 mutations and their pathogenicity will change the way HDGC patients are counselled for screening, surveillance and treatment. This review will bring the reader up to date with these changes and discuss future directions for research; namely more accurate risk stratification and surveillance methods to improve clinical care of HDGC patients. \n",
      "5 Recognition of and recent issues in hereditary diffuse gastric cancer.\n",
      "In East Asian countries, gastric cancer incidence is high, but detection rates for germline CDH1 mutations that cause hereditary diffuse gastric cancers (HDGCs) are low. Consequently, screens and genetic testing for HDGC are often considered unimportant. Since the first germline truncating CDH1 mutations in Japanese patients were reported, some HDGC cases have been reported, and some of these involve large germline rearrangements and de novo mutation of CDH1. New methods for mutation detection--such as multiplex ligation-dependent probe amplification, array comparative genomic hybridization, and exome sequencing--have become available, as have new experimental models, including novel gene-knockout mice and gastric organoids. Because of these advances, searches for candidate genes (e.g., CTNNA1, MAP3K6) and our understanding of HDGC pathogenesis have improved in recent years; moreover, there have been substantial changes in the field since the current HDGC consensus guidelines were released. This review focuses on recent issues and advances in the study of HDGC. For example, lobular breast cancer cases and de novo occurrences of DGC are unlikely to meet the existing criteria for genetic testing, but current evidence indicates that some such cases may be good candidates for genetic testing. It is important to recognize that HDGC is a syndrome and that lobular breast cancer can be the first manifestation of this syndrome. CDH1 testing, including analyses of large genomic rearrangements, should be recommended even in countries where few HDGC cases have been reported.\n",
      "6 A rapid and high content assay that measures cyto-ID-stained autophagic compartments and estimates autophagy flux with potential clinical applications.\n",
      "The lack of a rapid and quantitative autophagy assay has substantially hindered the development and implementation of autophagy-targeting therapies for a variety of human diseases. To address this critical issue, we developed a novel autophagy assay using the newly developed Cyto-ID fluorescence dye. We first verified that the Cyto-ID dye specifically labels autophagic compartments with minimal staining of lysosomes and endosomes. We then developed a new Cyto-ID fluorescence spectrophotometric assay that makes it possible to estimate autophagy flux based on measurements of the Cyto-ID-stained autophagic compartments. By comparing to traditional autophagy approaches, we found that this assay yielded a more sensitive, yet less variable, quantification of the stained autophagic compartments and the estimate of autophagy flux. Furthermore, we tested the potential application of this autophagy assay in high throughput research by integrating it into an RNA interference (RNAi) screen and a small molecule screen. The RNAi screen revealed WNK2 and MAP3K6 as autophagy-modulating genes, both of which inhibited the MTOR pathway. Similarly, the small molecule screen identified sanguinarine and actinomycin D as potent autophagy inducers in leukemic cells. Moreover, we successfully detected autophagy responses to kinase inhibitors and chloroquine in normal or leukemic mice using this assay. Collectively, this new Cyto-ID fluorescence spectrophotometric assay provides a rapid, reliable quantification of autophagic compartments and estimation of autophagy flux with potential applications in developing autophagy-related therapies and as a test to monitor autophagy responses in patients being treated with autophagy-modulating drugs. \n",
      "7 Germline mutations in MAP3K6 are associated with familial gastric cancer.\n",
      "Gastric cancer is among the leading causes of cancer-related deaths worldwide. While heritable forms of gastric cancer are relatively rare, identifying the genes responsible for such cases can inform diagnosis and treatment for both hereditary and sporadic cases of gastric cancer. Mutations in the E-cadherin gene, CDH1, account for 40% of the most common form of familial gastric cancer (FGC), hereditary diffuse gastric cancer (HDGC). The genes responsible for the remaining forms of FGC are currently unknown. Here we examined a large family from Maritime Canada with FGC without CDH1 mutations, and identified a germline coding variant (p.P946L) in mitogen-activated protein kinase kinase kinase 6 (MAP3K6). Based on conservation, predicted pathogenicity and a known role of the gene in cancer predisposition, MAP3K6 was considered a strong candidate and was investigated further. Screening of an additional 115 unrelated individuals with non-CDH1 FGC identified the p.P946L MAP3K6 variant, as well as four additional coding variants in MAP3K6 (p.F849Sfs*142, p.P958T, p.D200Y and p.V207G). A somatic second-hit variant (p.H506Y) was present in DNA obtained from one of the tumor specimens, and evidence of DNA hypermethylation within the MAP3K6 gene was observed in DNA from the tumor of another affected individual. These findings, together with previous evidence from mouse models that MAP3K6 acts as a tumor suppressor, and studies showing the presence of somatic mutations in MAP3K6 in non-hereditary gastric cancers and gastric cancer cell lines, point towards MAP3K6 variants as a predisposing factor for FGC. \n",
      "8 Salt-responsive ERF1 regulates reactive oxygen species-dependent signaling during the initial response to salt stress in rice.\n",
      "Early detection of salt stress is vital for plant survival and growth. Still, the molecular processes controlling early salt stress perception and signaling are not fully understood. Here, we identified salt-responsive ERF1 (SERF1), a rice (Oryza sativa) transcription factor (TF) gene that shows a root-specific induction upon salt and hydrogen peroxide (H2O2) treatment. Loss of SERF1 impairs the salt-inducible expression of genes encoding members of a mitogen-activated protein kinase (MAPK) cascade and salt tolerance-mediating TFs. Furthermore, we show that SERF1-dependent genes are H2O2 responsive and demonstrate that SERF1 binds to the promoters of MAPK kinase kinase6 (MAP3K6), MAPK5, dehydration-responsive element bindinG2A (DREB2A), and zinc finger protein179 (ZFP179) in vitro and in vivo. SERF1 also directly induces its own gene expression. In addition, SERF1 is a phosphorylation target of MAPK5, resulting in enhanced transcriptional activity of SERF1 toward its direct target genes. In agreement, plants deficient for SERF1 are more sensitive to salt stress compared with the wild type, while constitutive overexpression of SERF1 improves salinity tolerance. We propose that SERF1 amplifies the reactive oxygen species-activated MAPK cascade signal during the initial phase of salt stress and translates the salt-induced signal into an appropriate expressional response resulting in salt tolerance.\n",
      "9 Genetic and structural variation in the gastric cancer kinome revealed through targeted deep sequencing.\n",
      "Genetic alterations in kinases have been linked to multiple human pathologies. To explore the landscape of kinase genetic variation in gastric cancer (GC), we used targeted, paired-end deep sequencing to analyze 532 protein and phosphoinositide kinases in 14 GC cell lines. We identified 10,604 single-nucleotide variants (SNV) in kinase exons including greater than 300 novel nonsynonymous SNVs. Family-wise analysis of the nonsynonymous SNVs revealed a significant enrichment in mitogen-activated protein kinase (MAPK)-related genes (P < 0.01), suggesting a preferential involvement of this kinase family in GC. A potential antioncogenic role for MAP2K4, a gene exhibiting recurrent alterations in 2 lines, was functionally supported by siRNA knockdown and overexpression studies in wild-type and MAP2K4 variant lines. The deep sequencing data also revealed novel, large-scale structural rearrangement events involving kinases including gene fusions involving CDK12 and the ERBB2 receptor tyrosine kinase in MKN7 cells. Integrating SNVs and copy number alterations, we identified Hs746T as a cell line exhibiting both splice-site mutations and genomic amplification of MET, resulting in MET protein overexpression. When applied to primary GCs, we identified somatic mutations in 8 kinases, 4 of which were recurrently altered in both primary tumors and cell lines (MAP3K6, STK31, FER, and CDKL5). These results demonstrate that how targeted deep sequencing approaches can deliver unprecedented multilevel characterization of a medically and pharmacologically relevant gene family. The catalog of kinome genetic variants assembled here may broaden our knowledge on kinases and provide useful information on genetic alterations in GC.\n",
      "10 Global gene expression profiles of MT knockout and wild-type mice in the condition of doxorubicin-induced cardiomyopathy.\n",
      "Increasing evidence from in vivo and in vitro studies has indicated that MT exerts protective effects against DOX-induced cardiotoxicity; however the underlying precise mechanisms still remain an enigma. Therefore, the present study was designed using MT knockout mice in concert with genomic approaches to explore the possible molecular and cellular mechanisms in terms of the genetic network changes. MT-I/II null (MT⁻/⁻) mice and corresponding wild-type mice (MT+/+) were administrated with a single dose of DOX (15 mg/kg, i.p.) or equal volume of saline. Animals were sacrificed on the 4th day after DOX administration and samples were collected for further analyses. Global gene expression profiles of cardiac mRNA from two genotype mice revealed that 381 characteristically MT-responsive genes were identified between MT+/+ mice and MT⁻/⁻ mice in response to DOX, including fos, ucp3, car3, atf3, map3k6, etc. Functional analysis implied MAPK signaling pathway, p53 signaling pathway, Jak-STAT signaling pathway, PPAR signaling pathway, Wnt signaling pathway, etc. might be involved to mediate the protection of DOX cardiomyopathy by MT. Results from the present study not only validated the previously reported possible mechanisms of MT protection against DOX toxicity, but also provided new clues into the molecular mechanisms involved in this process.\n",
      "11 Impact of nose-only exposure system on pulmonary gene expression.\n",
      "Nose-only exposure is used to study the distribution and toxicity of airborne contaminants. Restraint of animals in nose-only tubes causes stress, but the impact on pulmonary mRNA levels is unknown. Since stress and xenobiotics activate common pathways, we assessed whether nose-only exposure would alter expression of toxicologically relevant genes in the lungs. To identify candidate genes for further analysis, we first interrogated microarray data to examine time-dependent changes in gene expression in air-control animals from a nose-only inhalation study involving male wild-type C57BL/6 mice and transgenic tumor necrosis factor (TNF)-alpha over-expressing littermates. Comparison of transcript levels immediately and 24 h after a single 4-h nose-only exposure to air revealed differential expression of 280 genes (false discovery rate-adjusted, p < .05). Functional analysis revealed enrichment of immune response, apoptosis, and signalling terms, consistent with effects of restraint stress. We then selected a subset of target genes for comparison of naive animals and air-exposed animals from the inhalation study by real-time polymerase chain reaction (PCR). Expression of genes involved in stress (BNIP, sestrin-1, CDKN1A [p21], GADD45 gamma), glucocorticoid-response (GILZ, Sgk), and signal transduction (MAP3K6, C/EBP-delta) was increased as a result of nose-only exposure (p < .05). In contrast, proinflammatory factors (lymphotoxin-beta, chemokine receptor CXCR5) were decreased (p < .05). Immune gene responses observed in wild-type animals were reduced in animals with lung inflammation, indicating that pathological states can modify the response to nose-only exposure. Observed responses may warrant consideration in the evaluation of materials delivered by nose-only inhalation, and suggest that incorporation of naive animals into nose-only studies should be considered as a best practice.\n",
      "12 Mitogen-activated protein 3 kinase 6 mediates angiogenic and tumorigenic effects via vascular endothelial growth factor expression.\n",
      "Genome-wide screening using a small interfering RNA (siRNA) library has revealed novel molecules that are involved in a wide range of physiological responses. The expression of vascular endothelial growth factor (VEGF) is increased under hypoxic conditions, and plays a crucial role in tumor angiogenesis and tissue responses to ischemia. Here, we used a siRNA expression vector library to elucidate molecules that modify VEGF expression. Screening using an siRNA library revealed that MAPKKK6 (MEKK6/MAP3K6) regulates VEGF expression under both normoxic and hypoxic conditions in vitro, although the biological function of MAP3K6 remains unknown. Attenuation of VEGF expression by MAP3K6 inhibition was demonstrated by transient transfection of double-stranded RNA as well as by stable transfection of short hairpin RNA-expressing vectors against MAP3K6. Conditioned medium of MAP3K6-knocked down cells attenuated both endothelial proliferation and capillary network formation in a VEGF-dependent manner in vitro. In addition, tumor cells with down-regulation of MAP3K6 expression showed significant suppression of tumor growth in vivo, which was accompanied by significant repression of vessel formation and VEGF expression in these tumors. The results of this study suggest that MAP3K6 regulates VEGF expression in both normoxia and hypoxia, and that regulation of VEGF by MAP3K6 may play a crucial role in both angiogenesis and tumorigenesis.\n",
      "13 Acute phase gene expression in mice exposed to the marine neurotoxin domoic acid.\n",
      "Domoic acid is a rigid analog of the neurotransmitter glutamate and a potent agonist of kainate subtype glutamate receptors. Persistent activation of these receptor subtypes results in rapid excitotoxicity, calcium dependent cell death and neuronal lesions in areas of the brain where kainate pathways are concentrated. To better understand responses to domoic acid induced excitotoxicity, microarrays were used to profile gene expression in mouse brain following domoic acid exposure. Adult female mice were subjected intraperitoneally to domoic acid at the lethal dose 50, killed and dissected at 30, 60 and 240 min post-injection. Total brain RNA from treated mice was compared with time-matched controls on Agilent 22K feature microarrays. Real-time PCR was performed on selected genes. For the 30, 60 and 240 min time points, 3.96%, 3.94% and 4.36% of the genes interrogated were differentially expressed (P-value < or = 0.01), respectively. Rigorous filtering of the data resulted in a set of 56 genes used for trending analysis and K-medians and agglomerative clustering. The earliest genes induced consisted primarily of early response gene families (Jun, Fos, Ier, Egr, growth arrest and DNA damage 45) and the inflammatory response element cyclooxygenase 2. Some later responding genes involved glucocorticoid responses (Gilz, Sgk), cold inducible proteins (Cirbp, Rbm3), Map kinases (Map3k6) and NF-kappaB inhibition. Real-time PCR in male mice from an additional study confirmed the expression of several of these genes across gender. The transcriptional profile induced by domoic acid shared similarity with expression profiles of brain ischemia and other excitotoxins, suggesting a common transcriptional response.\n"
     ]
    }
   ],
   "source": [
    "#articlesMAP3K6 is list of lxml.etree._Element. This parser is interrupted every time there is a tag interruption\n",
    "count = 0\n",
    "\n",
    "for article in articlesMAP3K6_tree: #shows all articles in organized format\n",
    "    count=count+1\n",
    "    abstract_text_children=\"\"\n",
    "    abstract_text_path=article.findall('MedlineCitation')[0].findall('Article')[0].findall('Abstract')[0].findall('AbstractText')[0]\n",
    "    \n",
    "    title = article.findall('MedlineCitation')[0].findall('Article')[0].findall('ArticleTitle')[0].text\n",
    "    abstract_text = article.findall('MedlineCitation')[0].findall('Article')[0].findall('Abstract')[0].findall('AbstractText')[0].text\n",
    "    \n",
    "    if len(abstract_text_path.getchildren())>0:\n",
    "        for child in abstract_text_path.getchildren(): \n",
    "            abstract_text_children= abstract_text_children + \" \" + child.text\n",
    "    print(str(count)+\" \" +title)\n",
    "    print(abstract_text + abstract_text_children)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# move to beautifulsoup to get rid of the fucking tags. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#I want to do beautifulsoup in conjunction. Above are corpora as bytes. I want them as soups now.\n",
    "MAP3K2soup = bs4.BeautifulSoup(MAP3K2corp, 'lxml')\n",
    "MAP3K3soup = bs4.BeautifulSoup(MAP3K3corp, 'lxml')\n",
    "MAP3K5soup = bs4.BeautifulSoup(MAP3K5corp, 'lxml')\n",
    "MAP3K6soup = bs4.BeautifulSoup(MAP3K6corp, 'lxml')\n",
    "MAP3K15soup = bs4.BeautifulSoup(MAP3K15corp, 'lxml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Role of germline aberrations affecting CTNNA1, MAP3K6 and MYD88 in gastric cancer susceptibility.'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "MAP3K6soup.articletitle.getText() #just gives first articletitle's title. ignores tags!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "soups_MAP3K6soup=MAP3K6soup.find_all('pubmedarticle') #separates into several soups\n",
    "soups_MAP3K2soup=MAP3K2soup.find_all('pubmedarticle')\n",
    "soups_MAP3K3soup=MAP3K3soup.find_all('pubmedarticle')\n",
    "soups_MAP3K5soup=MAP3K5soup.find_all('pubmedarticle')\n",
    "soups_MAP3K15soup=MAP3K15soup.find_all('pubmedarticle')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Role of germline aberrations affecting CTNNA1, MAP3K6 and MYD88 in gastric cancer susceptibility.\n",
      "In approximately 10% of all gastric cancer (GC) cases, a heritable cause is suspected. A subset of these cases have a causative germline CDH1 mutation; however, in most cases the cause remains unknown. Our objective was to assess to what extent these remaining cases may be explained by germline mutations in the novel candidate GC predisposing genes CTNNA1, MAP3K6 or MYD88.\n",
      "We sequenced a large cohort of unexplained young and/or familial patients with GC (n=286) without a CDH1germline mutation for germline variants affecting CTNNA1, MAP3K6 and MYD88 using a targeted next-generation sequencing approach based on single-molecule molecular inversion probes.\n",
      "Predicted deleterious germline variants were not encountered in MYD88, but recurrently observed in CTNNA1 (n=2) and MAP3K6 (n=3) in our cohort of patients with GC. In contrast to deleterious variants in CTNNA1, deleterious variants in MAP3K6 also occur frequently in the general population.\n",
      "Based on our results MAP3K6 should no longer be considered a GC predisposition gene, whereas deleterious CTNNA1 variants are confirmed as an infrequent cause of GC susceptibility. Biallelic MYD88 germline mutations are at most a very rare cause of GC susceptibility as no additional cases were identified.\n"
     ]
    }
   ],
   "source": [
    "#find the text I want from just one soup:  \n",
    "print(soups_MAP3K6soup[0].articletitle.getText()) #print single abstract's title\n",
    "\n",
    "#print all of a single abstract's text with a for loop \n",
    "abstract_text_list=soups_MAP3K6soup[0].find_all('abstracttext')\n",
    "type(abstract_text_list)\n",
    "for abstract_text in abstract_text_list:\n",
    "    print(abstract_text.getText())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"Publisher\">\n",
       " <pmid version=\"1\">29330337</pmid>\n",
       " <daterevised>\n",
       " <year>2018</year>\n",
       " <month>01</month>\n",
       " <day>13</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1468-6244</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <pubdate>\n",
       " <year>2018</year>\n",
       " <month>Jan</month>\n",
       " <day>12</day>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Journal of medical genetics</title>\n",
       " <isoabbreviation>J. Med. Genet.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Role of germline aberrations affecting <i>CTNNA1</i>, <i>MAP3K6</i> and <i>MYD88</i> in gastric cancer susceptibility.</articletitle>\n",
       " <elocationid eidtype=\"pii\" validyn=\"Y\">jmedgenet-2017-104962</elocationid>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1136/jmedgenet-2017-104962</elocationid>\n",
       " <abstract>\n",
       " <abstracttext label=\"BACKGROUND\" nlmcategory=\"BACKGROUND\">In approximately 10% of all gastric cancer (GC) cases, a heritable cause is suspected. A subset of these cases have a causative germline <i>CDH1</i> mutation; however, in most cases the cause remains unknown. Our objective was to assess to what extent these remaining cases may be explained by germline mutations in the novel candidate GC predisposing genes <i>CTNNA1</i>, <i>MAP3K6</i> or <i>MYD88</i>.</abstracttext>\n",
       " <abstracttext label=\"METHODS\" nlmcategory=\"METHODS\">We sequenced a large cohort of unexplained young and/or familial patients with GC (n=286) without a <i>CDH1</i>germline mutation for germline variants affecting <i>CTNNA1</i>, <i>MAP3K6</i> and <i>MYD88</i> using a targeted next-generation sequencing approach based on single-molecule molecular inversion probes.</abstracttext>\n",
       " <abstracttext label=\"RESULTS\" nlmcategory=\"RESULTS\">Predicted deleterious germline variants were not encountered in <i>MYD88</i>, but recurrently observed in <i>CTNNA1</i> (n=2) and <i>MAP3K6</i> (n=3) in our cohort of patients with GC. In contrast to deleterious variants in <i>CTNNA1</i>, deleterious variants in <i>MAP3K6</i> also occur frequently in the general population.</abstracttext>\n",
       " <abstracttext label=\"CONCLUSIONS\" nlmcategory=\"CONCLUSIONS\">Based on our results <i>MAP3K6</i> should no longer be considered a GC predisposition gene, whereas deleterious <i>CTNNA1</i> variants are confirmed as an infrequent cause of GC susceptibility. Biallelic <i>MYD88</i> germline mutations are at most a very rare cause of GC susceptibility as no additional cases were identified.</abstracttext>\n",
       " <copyrightinformation>© Article author(s) (or their employer(s) unless otherwise stated in the text of the article) 2018. All rights reserved. No commercial use is permitted unless otherwise expressly granted.</copyrightinformation>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Weren</lastname>\n",
       " <forename>Robbert D A</forename>\n",
       " <initials>RDA</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>van der Post</lastname>\n",
       " <forename>Rachel S</forename>\n",
       " <initials>RS</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Vogelaar</lastname>\n",
       " <forename>Ingrid P</forename>\n",
       " <initials>IP</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>van Krieken</lastname>\n",
       " <forename>J Han</forename>\n",
       " <initials>JH</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Spruijt</lastname>\n",
       " <forename>Liesbeth</forename>\n",
       " <initials>L</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Lubinski</lastname>\n",
       " <forename>Jan</forename>\n",
       " <initials>J</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Genetics and Pathology, Pomeranian Medical University, Szczecin, Poland.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Jakubowska</lastname>\n",
       " <forename>Anna</forename>\n",
       " <initials>A</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Genetics and Pathology, Pomeranian Medical University, Szczecin, Poland.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Teodorczyk</lastname>\n",
       " <forename>Urszula</forename>\n",
       " <initials>U</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Genetics and Pathology, Pomeranian Medical University, Szczecin, Poland.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Aalfs</lastname>\n",
       " <forename>Cora M</forename>\n",
       " <initials>CM</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Clinical Genetics, Academic Medical Center, Amsterdam, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>van Hest</lastname>\n",
       " <forename>Liselotte P</forename>\n",
       " <initials>LP</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Clinical Genetics, VU University Medical Center, Amsterdam, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Oliveira</lastname>\n",
       " <forename>Carla</forename>\n",
       " <initials>C</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Expression Regulation in Cancer Group, Instituto de Investigação e Inovação em Saúde, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Cancer Genetics, Institute of Molecular Pathology and Immunology of the University of Porto, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology and Oncology, Faculty of Medicine, University of Porto, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Kamping</lastname>\n",
       " <forename>Eveline J</forename>\n",
       " <initials>EJ</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Schackert</lastname>\n",
       " <forename>Hans K</forename>\n",
       " <initials>HK</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Surgical Research, Universitätsklinikum Carl Gustav Carus, Technische Universität Dresden, Dresden, Germany.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ranzani</lastname>\n",
       " <forename>Guglielmina N</forename>\n",
       " <initials>GN</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Biology and Biotechnology, University of Pavia, Pavia, Italy.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Gómez García</lastname>\n",
       " <forename>Encarna B</forename>\n",
       " <initials>EB</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Clinical Genetics, Maastricht University Medical Center, Maastricht, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hes</lastname>\n",
       " <forename>Frederik J</forename>\n",
       " <initials>FJ</initials>\n",
       " <identifier source=\"ORCID\">http://orcid.org/0000-0002-3025-0882</identifier>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Clinical Genetics, Leiden University Medical Center, Leiden, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Holinski-Feder</lastname>\n",
       " <forename>Elke</forename>\n",
       " <initials>E</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medizinische Klinik und Poliklinik IV, Klinikum der Universität München, Munich, Germany.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Genuardi</lastname>\n",
       " <forename>Maurizio</forename>\n",
       " <initials>M</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Institute of Genomic Medicine, Catholic University of the Sacred Heart, Milan, Italy.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ausems</lastname>\n",
       " <forename>Margreet G E M</forename>\n",
       " <initials>MGEM</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Genetics, University Medical Center Utrecht, Utrecht, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Sijmons</lastname>\n",
       " <forename>Rolf H</forename>\n",
       " <initials>RH</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Wagner</lastname>\n",
       " <forename>Anja</forename>\n",
       " <initials>A</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Clinical Genetics, Erasmus University Medical Center, Rotterdam, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>van der Kolk</lastname>\n",
       " <forename>Lizet E</forename>\n",
       " <initials>LE</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Family Cancer Clinic, The Netherlands Cancer Institute, Amsterdam, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Cats</lastname>\n",
       " <forename>Annemieke</forename>\n",
       " <initials>A</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Gastrointestinal Oncology, The Netherlands Cancer Institute/Antoni van Leeuwenhoek, Amsterdam, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Bjørnevoll</lastname>\n",
       " <forename>Inga</forename>\n",
       " <initials>I</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Medical Genetics, St Olav's Hospital, Trondheim, Norway.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hoogerbrugge</lastname>\n",
       " <forename>Nicoline</forename>\n",
       " <initials>N</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ligtenberg</lastname>\n",
       " <forename>Marjolijn J L</forename>\n",
       " <initials>MJL</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Radboud university medical center, Nijmegen, The Netherlands.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2018</year>\n",
       " <month>01</month>\n",
       " <day>12</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>England</country>\n",
       " <medlineta>J Med Genet</medlineta>\n",
       " <nlmuniqueid>2985087R</nlmuniqueid>\n",
       " <issnlinking>0022-2593</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <keywordlist owner=\"NOTNLM\">\n",
       " <keyword majortopicyn=\"N\">cancer: gastric</keyword>\n",
       " <keyword majortopicyn=\"N\">ctnna1 – map3k6 – myd88</keyword>\n",
       " <keyword majortopicyn=\"N\">heritability</keyword>\n",
       " <keyword majortopicyn=\"N\">next generation sequencing</keyword>\n",
       " </keywordlist>\n",
       " <coistatement>Competing interests: None declared.</coistatement>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2017</year>\n",
       " <month>08</month>\n",
       " <day>07</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"revised\">\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>19</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2017</year>\n",
       " <month>12</month>\n",
       " <day>11</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2018</year>\n",
       " <month>1</month>\n",
       " <day>14</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2018</year>\n",
       " <month>1</month>\n",
       " <day>14</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2018</year>\n",
       " <month>1</month>\n",
       " <day>14</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>aheadofprint</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">29330337</articleid>\n",
       " <articleid idtype=\"pii\">jmedgenet-2017-104962</articleid>\n",
       " <articleid idtype=\"doi\">10.1136/jmedgenet-2017-104962</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"In-Process\">\n",
       " <pmid version=\"1\">29132690</pmid>\n",
       " <daterevised>\n",
       " <year>2017</year>\n",
       " <month>12</month>\n",
       " <day>22</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1618-0372</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>120</volume>\n",
       " <issue>1</issue>\n",
       " <pubdate>\n",
       " <year>2018</year>\n",
       " <month>Jan</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Acta histochemica</title>\n",
       " <isoabbreviation>Acta Histochem.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Cyclin-dependent kinase 5 regulates MAPK/ERK signaling in the skin of mice.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>15-21</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"pii\" validyn=\"Y\">S0065-1281(17)30297-0</elocationid>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1016/j.acthis.2017.10.009</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Cyclin-dependent kinase 5 (CDK5) is a proline-directed serine/threonine kinase that has been shown to play important roles in many tissues except the nervous system. We previously reported that CDK5 showed differential expression in the transcriptome profiles of the skin of alpacas with different hair colors. To understand the functional role of CDK5 in hair color determination, we constructed CDK5-knockdown mice and identified the effect on the mitogen-activated protein kinase (MAPK) pathway in the mouse skin. Quantitative real-time polymerase chain reaction, co-immunoprecipitation, and western blotting were performed to analyze the effects of CDK5-knockdown on the MAPK pathway in mice. The results showed that MAP3K6 was inhibited by phosphorylated CDK5 through its activator CDK7. The decrease in MAP3K6 levels caused down-regulation of MEK1 and ERK expression, leading to the up-regulation of miR-143-3p, which targets MAP3K6 via Dicer. Taken together, our findings indicate that CDK5 functions in regulating the MAPK pathway. Given that MAP3K6 was inhibited in two directions, this mechanism can provide insight into the contributions of the MAPK/ERK pathway to the inhibition of melanin production.</abstracttext>\n",
       " <copyrightinformation>Copyright © 2017 Elsevier GmbH. All rights reserved.</copyrightinformation>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Liu</lastname>\n",
       " <forename>Xuexian</forename>\n",
       " <initials>X</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhang</lastname>\n",
       " <forename>Pengqian</forename>\n",
       " <initials>P</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Ecology Research, Beijing Milu Ecological Research Center, Nanhaizi, Daxing District, Beijing 102600, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ji</lastname>\n",
       " <forename>Kaiyuan</forename>\n",
       " <initials>K</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhang</lastname>\n",
       " <forename>Junzhen</forename>\n",
       " <initials>J</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Yang</lastname>\n",
       " <forename>Shanshan</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Du</lastname>\n",
       " <forename>Bin</forename>\n",
       " <initials>B</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hu</lastname>\n",
       " <forename>Shuaipeng</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Fan</lastname>\n",
       " <forename>Ruiwen</forename>\n",
       " <initials>R</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>College of Animal Science and Veterinary Medicine, Shanxi Agricultural University, Mingxian South Road, Taigu 030801, China. Electronic address: 2687299379@qq.com.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>11</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>Germany</country>\n",
       " <medlineta>Acta Histochem</medlineta>\n",
       " <nlmuniqueid>0370320</nlmuniqueid>\n",
       " <issnlinking>0065-1281</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <keywordlist owner=\"NOTNLM\">\n",
       " <keyword majortopicyn=\"N\">CDK5</keyword>\n",
       " <keyword majortopicyn=\"N\">CDK5-Knockdown mouse</keyword>\n",
       " <keyword majortopicyn=\"N\">MAP3K6</keyword>\n",
       " <keyword majortopicyn=\"N\">MEK1</keyword>\n",
       " <keyword majortopicyn=\"N\">Pigment</keyword>\n",
       " </keywordlist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2017</year>\n",
       " <month>08</month>\n",
       " <day>21</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"revised\">\n",
       " <year>2017</year>\n",
       " <month>10</month>\n",
       " <day>28</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2017</year>\n",
       " <month>10</month>\n",
       " <day>30</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>15</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>15</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>15</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">29132690</articleid>\n",
       " <articleid idtype=\"pii\">S0065-1281(17)30297-0</articleid>\n",
       " <articleid idtype=\"doi\">10.1016/j.acthis.2017.10.009</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">27899187</pmid>\n",
       " <datecompleted>\n",
       " <year>2017</year>\n",
       " <month>05</month>\n",
       " <day>26</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2017</year>\n",
       " <month>10</month>\n",
       " <day>24</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1532-8708</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>43</volume>\n",
       " <issue>5</issue>\n",
       " <pubdate>\n",
       " <year>2016</year>\n",
       " <month>Oct</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Seminars in oncology</title>\n",
       " <isoabbreviation>Semin. Oncol.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Genetic predisposition to gastric cancer.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>554-559</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"pii\" validyn=\"Y\">S0093-7754(16)30025-2</elocationid>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1053/j.seminoncol.2016.08.006</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Gastric cancer ranks as the third leading cause of cancer mortality worldwide and confers a 5-year survival of 20%. While most gastric cancers are sporadic, ~1%-3% can be attributed to inherited cancer predisposition syndromes. Germline E-cadherin/CDH1 mutations have been identified in families with an autosomal dominant inherited predisposition to diffuse gastric cancer. The cumulative risk of gastric cancer for CDH1 mutation carriers by age 80 years is reportedly 70% for men and 56% for women. Female mutation carriers also have an estimated 42% risk for developing lobular breast cancer by age 80 years. However, most individuals meeting clinical criteria for hereditary diffuse gastric cancer syndrome (HDGC) do not have a germline CDH1 mutation, and germline CDH1 mutation carriers do not all exhibit similar clinical outcomes in terms of age of diagnosis or cancer types. E-cadherin (CDH1) as the one known causative gene for HDGC accounts for only 40% of cases, leaving 60% with an unknown genetic diagnosis. In addition to HDGC, we will review other genetic syndromes with elevated gastric cancer risk, as well as newly implicated alterations in other genes (CTNNA1, DOT1L, FBXO24, PRSS1, MAP3K6, MSR1, and INSR) that may affect gastric cancer susceptibility and age-specific penetrance.</abstracttext>\n",
       " <copyrightinformation>Copyright Â© 2016 Elsevier Inc. All rights reserved.</copyrightinformation>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Petrovchich</lastname>\n",
       " <forename>Iva</forename>\n",
       " <initials>I</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Clinical Cancer Genetics Program, Division of Oncology, Departments of Medicine and Genetics, Stanford University School of Medicine, Stanford, CA.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ford</lastname>\n",
       " <forename>James M</forename>\n",
       " <initials>JM</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Clinical Cancer Genetics Program, Division of Oncology, Departments of Medicine and Genetics, Stanford University School of Medicine, Stanford, CA. Electronic address: jmf@stanford.edu.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D016454\">Review</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2016</year>\n",
       " <month>09</month>\n",
       " <day>22</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Semin Oncol</medlineta>\n",
       " <nlmuniqueid>0420432</nlmuniqueid>\n",
       " <issnlinking>0093-7754</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"C513384\">CDH1 protein, human</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"C495273\">CTNNA1 protein, human</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D015820\">Cadherins</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D051178\">alpha Catenin</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015820\">Cadherins</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005260\">Female</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D020022\">Genetic Predisposition to Disease</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008297\">Male</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009154\">Mutation</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013274\">Stomach Neoplasms</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000175\">diagnosis</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000209\">etiology</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051178\">alpha Catenin</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " <keywordlist owner=\"NOTNLM\">\n",
       " <keyword majortopicyn=\"Y\">CDH1 gene</keyword>\n",
       " <keyword majortopicyn=\"Y\">E-cadherin</keyword>\n",
       " <keyword majortopicyn=\"Y\">Genetic testing</keyword>\n",
       " <keyword majortopicyn=\"Y\">Stomach cancer</keyword>\n",
       " </keywordlist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2016</year>\n",
       " <month>05</month>\n",
       " <day>25</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2016</year>\n",
       " <month>08</month>\n",
       " <day>10</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2016</year>\n",
       " <month>12</month>\n",
       " <day>1</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2016</year>\n",
       " <month>12</month>\n",
       " <day>3</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2017</year>\n",
       " <month>5</month>\n",
       " <day>27</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">27899187</articleid>\n",
       " <articleid idtype=\"pii\">S0093-7754(16)30025-2</articleid>\n",
       " <articleid idtype=\"doi\">10.1053/j.seminoncol.2016.08.006</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"PubMed-not-MEDLINE\">\n",
       " <pmid version=\"1\">26380059</pmid>\n",
       " <datecompleted>\n",
       " <year>2015</year>\n",
       " <month>09</month>\n",
       " <day>18</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2017</year>\n",
       " <month>02</month>\n",
       " <day>20</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print\">\n",
       " <journal>\n",
       " <issn issntype=\"Print\">1948-5204</issn>\n",
       " <journalissue citedmedium=\"Print\">\n",
       " <volume>7</volume>\n",
       " <issue>9</issue>\n",
       " <pubdate>\n",
       " <year>2015</year>\n",
       " <month>Sep</month>\n",
       " <day>15</day>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>World journal of gastrointestinal oncology</title>\n",
       " <isoabbreviation>World J Gastrointest Oncol</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Hereditary diffuse gastric cancer: What the clinician should know.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>153-60</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.4251/wjgo.v7.i9.153</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Hereditary diffuse gastric cancer (HDGC) is an inherited autosomal dominant syndrome with a penetrance of up to 80% affecting diverse geographic populations. While it has been shown to be caused mainly by germline alterations in the E-cadherin gene (CDH1), problematically, the genetic diagnosis remains unknown in up to 60% of patients. Given the important knowledge gaps regarding the syndrome, asymptomatic carriers of CDH1 mutations are advised for a prophylactic total gastrectomy. Intensive annual endoscopic surveillance is the alternative for carriers who decline gastrectomy. As HDGCs have a prolonged indolent phase, this provides a window of opportunity for surveillance and treatment. Recent findings of other gene defects in CTNNA1 and MAP3K6, as well as further characterization of CDH1 mutations and their pathogenicity will change the way HDGC patients are counselled for screening, surveillance and treatment. This review will bring the reader up to date with these changes and discuss future directions for research; namely more accurate risk stratification and surveillance methods to improve clinical care of HDGC patients. </abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Tan</lastname>\n",
       " <forename>Ryan Ying Cong</forename>\n",
       " <initials>RY</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Ryan Ying Cong Tan, Joanne Ngeow, Cancer Genetics Service, Division of Medical Oncology, National Cancer Centre, Singapore 169610, Singapore.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ngeow</lastname>\n",
       " <forename>Joanne</forename>\n",
       " <initials>J</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Ryan Ying Cong Tan, Joanne Ngeow, Cancer Genetics Service, Division of Medical Oncology, National Cancer Centre, Singapore 169610, Singapore.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D016454\">Review</publicationtype>\n",
       " </publicationtypelist>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>China</country>\n",
       " <medlineta>World J Gastrointest Oncol</medlineta>\n",
       " <nlmuniqueid>101532470</nlmuniqueid>\n",
       " </medlinejournalinfo>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Cancer. 2004 Dec;4(12):988-93</refsource>\n",
       " <pmid version=\"1\">15573120</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Surg Oncol. 2009 Jul;16(7):1890-5</refsource>\n",
       " <pmid version=\"1\">19408054</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Oncogene. 2004 Mar 18;23(12):2236-40</refsource>\n",
       " <pmid version=\"1\">14661064</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Hum Genet. 2013 Aug;21(8):null</refsource>\n",
       " <pmid version=\"1\">23443028</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Sci. 2006 Jan;97(1):64-71</refsource>\n",
       " <pmid version=\"1\">16367923</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2013 Nov 25;8(11):e81579</refsource>\n",
       " <pmid version=\"1\">24282611</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1998 Mar 26;392(6674):402-5</refsource>\n",
       " <pmid version=\"1\">9537325</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Carcinogenesis. 2006 Nov;27(11):2180-9</refsource>\n",
       " <pmid version=\"1\">16675472</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>N Engl J Med. 2001 Jun 21;344(25):1904-9</refsource>\n",
       " <pmid version=\"1\">11419427</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2012 Sep;15 Suppl 1:S153-63</refsource>\n",
       " <pmid version=\"1\">22160243</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2010 Jul;47(7):436-44</refsource>\n",
       " <pmid version=\"1\">20591882</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mutat. 2002 May;19(5):510-7</refsource>\n",
       " <pmid version=\"1\">11968083</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 1999 Dec;36(12):873-80</refsource>\n",
       " <pmid version=\"1\">10593993</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Arch Pathol Lab Med. 2004 Jul;128(7):765-70</refsource>\n",
       " <pmid version=\"1\">15214826</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2000 Sep;26(1):16-7</refsource>\n",
       " <pmid version=\"1\">10973239</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Acta Pathol Microbiol Scand. 1965;64:31-49</refsource>\n",
       " <pmid version=\"1\">14320675</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2009 May 1;18(9):1545-55</refsource>\n",
       " <pmid version=\"1\">19168852</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Gastroenterol Hepatol. 2014 Mar;26(3):353-6</refsource>\n",
       " <pmid version=\"1\">24240619</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2003 Mar 1;12(5):575-82</refsource>\n",
       " <pmid version=\"1\">12588804</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Curr Gastroenterol Rep. 2012 Oct;14(5):428-38</refsource>\n",
       " <pmid version=\"1\">22864806</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2012;7(3):e33783</refsource>\n",
       " <pmid version=\"1\">22470475</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Cancer. 2004 Feb;4(2):118-32</refsource>\n",
       " <pmid version=\"1\">14964308</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2009 Jun;136(7):2137-48</refsource>\n",
       " <pmid version=\"1\">19269290</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Surg. 2014 Jul;260(1):87-93</refsource>\n",
       " <pmid version=\"1\">24424140</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Gastroenterol. 2010 Jun;105(6):1258-64; author reply 1265</refsource>\n",
       " <pmid version=\"1\">20051941</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2004 Jul;41(7):508-17</refsource>\n",
       " <pmid version=\"1\">15235021</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Lett. 2006 Jul 18;238(2):260-70</refsource>\n",
       " <pmid version=\"1\">16157446</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Cancer. 2008 Dec;8(12):915-28</refsource>\n",
       " <pmid version=\"1\">19029956</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2014 Oct;17(4):745-9</refsource>\n",
       " <pmid version=\"1\">23812922</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2012 Jul 15;131(2):367-76</refsource>\n",
       " <pmid version=\"1\">22020549</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Hum Genet. 2013 Mar;21(3):301-9</refsource>\n",
       " <pmid version=\"1\">22850631</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Mol Med (Berl). 2006 Dec;84(12):1023-31</refsource>\n",
       " <pmid version=\"1\">16924464</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 1995 Jul 19;87(14):1082-4</refsource>\n",
       " <pmid version=\"1\">7616601</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2004 Jun;203(2):681-7</refsource>\n",
       " <pmid version=\"1\">15141383</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2004 Jun;53(6):775-8</refsource>\n",
       " <pmid version=\"1\">15138199</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2004 Jul;41(7):484-91</refsource>\n",
       " <pmid version=\"1\">15235019</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 2000 Dec 6;92(23):1881-8</refsource>\n",
       " <pmid version=\"1\">11106679</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Pathobiology. 2011;78(6):302-10</refsource>\n",
       " <pmid version=\"1\">22104201</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fam Cancer. 2015 Jun;14(2):241-6</refsource>\n",
       " <pmid version=\"1\">25576241</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Biol. 2014 Aug 27;15(8):428</refsource>\n",
       " <pmid version=\"1\">25315765</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2007 Nov;44(11):726-31</refsource>\n",
       " <pmid version=\"1\">17660459</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Surg Oncol. 2011 Sep;18(9):2594-8</refsource>\n",
       " <pmid version=\"1\">21424370</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Cancer. 2012 Jan 06;12:8</refsource>\n",
       " <pmid version=\"1\">22225527</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2008 Nov;216(3):295-306</refsource>\n",
       " <pmid version=\"1\">18788075</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Aliment Pharmacol Ther. 2004 Jul;20 Suppl 1:131-42</refsource>\n",
       " <pmid version=\"1\">15298619</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2006 Feb;43(2):138-42</refsource>\n",
       " <pmid version=\"1\">15831593</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2010 Jun;138(6):2044-58</refsource>\n",
       " <pmid version=\"1\">20420945</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Exp Metastasis. 2004;21(1):49-56</refsource>\n",
       " <pmid version=\"1\">15065602</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2010 Feb;138(2):487-92</refsource>\n",
       " <pmid version=\"1\">19900449</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Expert Rev Mol Diagn. 2003 Mar;3(2):201-15</refsource>\n",
       " <pmid version=\"1\">12647996</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Epidemiol Biomarkers Prev. 2012 Dec;21(12):2272-4</refsource>\n",
       " <pmid version=\"1\">23071139</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Cancer. 2009 May;45(7):1282-93</refsource>\n",
       " <pmid version=\"1\">19195878</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Cancer. 2006 Apr;42(6):708-16</refsource>\n",
       " <pmid version=\"1\">16556496</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genet Med. 2011 Jul;13(7):651-7</refsource>\n",
       " <pmid version=\"1\">21552135</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fam Cancer. 2015 Mar;14(1):89-94</refsource>\n",
       " <pmid version=\"1\">25467645</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer. 2008 Oct 1;113(7 Suppl):1850-6</refsource>\n",
       " <pmid version=\"1\">18798546</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2013 Mar;229(4):621-9</refsource>\n",
       " <pmid version=\"1\">23208944</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>N Engl J Med. 2002 Oct 10;347(15):1175-86</refsource>\n",
       " <pmid version=\"1\">12374879</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2001 Dec;121(6):1348-53</refsource>\n",
       " <pmid version=\"1\">11729114</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>CA Cancer J Clin. 2011 Mar-Apr;61(2):69-90</refsource>\n",
       " <pmid version=\"1\">21296855</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Nucl Med Mol Imaging. 2002 Apr;29(4):525-9</refsource>\n",
       " <pmid version=\"1\">11914891</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Histopathology. 2014 Sep;65(3):353-62</refsource>\n",
       " <pmid version=\"1\">24548295</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fam Cancer. 2010 Sep;9(3):331-4</refsource>\n",
       " <pmid version=\"1\">20063069</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS Genet. 2014 Oct 23;10(10):e1004669</refsource>\n",
       " <pmid version=\"1\">25340522</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Genet. 2012 Sep;82(3):283-7</refsource>\n",
       " <pmid version=\"1\">21696387</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Best Pract Res Clin Gastroenterol. 2014 Dec;28(6):1055-68</refsource>\n",
       " <pmid version=\"1\">25439071</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <keywordlist owner=\"NOTNLM\">\n",
       " <keyword majortopicyn=\"N\">CDH1</keyword>\n",
       " <keyword majortopicyn=\"N\">CTNNA1</keyword>\n",
       " <keyword majortopicyn=\"N\">Gastrectomy</keyword>\n",
       " <keyword majortopicyn=\"N\">Hereditary diffuse gastric cancer</keyword>\n",
       " <keyword majortopicyn=\"N\">MAP3K6</keyword>\n",
       " </keywordlist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2015</year>\n",
       " <month>04</month>\n",
       " <day>11</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"revised\">\n",
       " <year>2015</year>\n",
       " <month>06</month>\n",
       " <day>02</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2015</year>\n",
       " <month>07</month>\n",
       " <day>08</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2015</year>\n",
       " <month>9</month>\n",
       " <day>18</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2015</year>\n",
       " <month>9</month>\n",
       " <day>18</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2015</year>\n",
       " <month>9</month>\n",
       " <day>18</day>\n",
       " <hour>6</hour>\n",
       " <minute>1</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">26380059</articleid>\n",
       " <articleid idtype=\"doi\">10.4251/wjgo.v7.i9.153</articleid>\n",
       " <articleid idtype=\"pmc\">PMC4569593</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">26049741</pmid>\n",
       " <datecompleted>\n",
       " <year>2016</year>\n",
       " <month>05</month>\n",
       " <day>02</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2017</year>\n",
       " <month>11</month>\n",
       " <day>21</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1435-5922</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>50</volume>\n",
       " <issue>8</issue>\n",
       " <pubdate>\n",
       " <year>2015</year>\n",
       " <month>Aug</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Journal of gastroenterology</title>\n",
       " <isoabbreviation>J. Gastroenterol.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Recognition of and recent issues in hereditary diffuse gastric cancer.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>831-43</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1007/s00535-015-1093-9</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>In East Asian countries, gastric cancer incidence is high, but detection rates for germline CDH1 mutations that cause hereditary diffuse gastric cancers (HDGCs) are low. Consequently, screens and genetic testing for HDGC are often considered unimportant. Since the first germline truncating CDH1 mutations in Japanese patients were reported, some HDGC cases have been reported, and some of these involve large germline rearrangements and de novo mutation of CDH1. New methods for mutation detection--such as multiplex ligation-dependent probe amplification, array comparative genomic hybridization, and exome sequencing--have become available, as have new experimental models, including novel gene-knockout mice and gastric organoids. Because of these advances, searches for candidate genes (e.g., CTNNA1, MAP3K6) and our understanding of HDGC pathogenesis have improved in recent years; moreover, there have been substantial changes in the field since the current HDGC consensus guidelines were released. This review focuses on recent issues and advances in the study of HDGC. For example, lobular breast cancer cases and de novo occurrences of DGC are unlikely to meet the existing criteria for genetic testing, but current evidence indicates that some such cases may be good candidates for genetic testing. It is important to recognize that HDGC is a syndrome and that lobular breast cancer can be the first manifestation of this syndrome. CDH1 testing, including analyses of large genomic rearrangements, should be recommended even in countries where few HDGC cases have been reported.</abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Sugimoto</lastname>\n",
       " <forename>Shinya</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Division of Gastroenterology and Hepatology, Department of Internal Medicine, Keio University School of Medicine, 35 Shinanomachi, Shinjuku-ku, Tokyo, 160-8582, Japan, sugimoto-kei@umin.ac.jp.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Komatsu</lastname>\n",
       " <forename>Hirokazu</forename>\n",
       " <initials>H</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Morohoshi</lastname>\n",
       " <forename>Yuichi</forename>\n",
       " <initials>Y</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Kanai</lastname>\n",
       " <forename>Takanori</forename>\n",
       " <initials>T</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D016454\">Review</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2015</year>\n",
       " <month>06</month>\n",
       " <day>07</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>Japan</country>\n",
       " <medlineta>J Gastroenterol</medlineta>\n",
       " <nlmuniqueid>9430794</nlmuniqueid>\n",
       " <issnlinking>0944-1174</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"C513384\">CDH1 protein, human</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D015820\">Cadherins</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2015 Mar 1;136(5):E359-86</refsource>\n",
       " <pmid version=\"1\">25220842</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Surg Oncol. 2009 Jul;16(7):1890-5</refsource>\n",
       " <pmid version=\"1\">19408054</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 2007 Mar 15;67(6):2480-9</refsource>\n",
       " <pmid version=\"1\">17363565</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>JAMA Oncol. 2015 Apr;1(1):23-32</refsource>\n",
       " <pmid version=\"1\">26182300</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2009 Apr 8;28(7):843-53</refsource>\n",
       " <pmid version=\"1\">19214184</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2002 Oct 10;101(5):434-41</refsource>\n",
       " <pmid version=\"1\">12216071</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 2014 Feb 18;110(4):1053-7</refsource>\n",
       " <pmid version=\"1\">24366306</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Pathol. 2002 Nov;161(5):1735-42</refsource>\n",
       " <pmid version=\"1\">12414520</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Biol. 2014 Apr 01;15(4):R55</refsource>\n",
       " <pmid version=\"1\">24690483</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1998 Mar 26;392(6674):402-5</refsource>\n",
       " <pmid version=\"1\">9537325</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer. 2002 Jan 1;94(1):78-83</refsource>\n",
       " <pmid version=\"1\">11815962</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genes Chromosomes Cancer. 2014 May;53(5):432-45</refsource>\n",
       " <pmid version=\"1\">24493355</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Intern Med. 2008;47(12):1077-83</refsource>\n",
       " <pmid version=\"1\">18552463</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fam Cancer. 2013 Sep;12(3):503-7</refsource>\n",
       " <pmid version=\"1\">23264079</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Endoscopy. 2012 Feb;44(2):122-7</refsource>\n",
       " <pmid version=\"1\">22271022</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>N Engl J Med. 2001 Jun 21;344(25):1904-9</refsource>\n",
       " <pmid version=\"1\">11419427</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Clin Oncol. 2013 Mar 1;31(7):868-75</refsource>\n",
       " <pmid version=\"1\">23341533</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2006 Dec 1;119(11):2592-6</refsource>\n",
       " <pmid version=\"1\">16929514</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Genet. 2005 Jan;67(1):81-6</refsource>\n",
       " <pmid version=\"1\">15617552</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2014 Oct;17(4):750-6</refsource>\n",
       " <pmid version=\"1\">24037103</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2011 Jun;14(2):113-23</refsource>\n",
       " <pmid version=\"1\">21573742</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Epidemiol Biomarkers Prev. 1994 Jan-Feb;3(1):15-8</refsource>\n",
       " <pmid version=\"1\">8118379</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2010 Dec;139(6):1927-33</refsource>\n",
       " <pmid version=\"1\">20600018</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2010 Jul;47(7):436-44</refsource>\n",
       " <pmid version=\"1\">20591882</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 2008 Sep 5;321(5894):1361-5</refsource>\n",
       " <pmid version=\"1\">18703712</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 1999 Dec;36(12):873-80</refsource>\n",
       " <pmid version=\"1\">10593993</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cancer. 2009 Aug 12;8:63</refsource>\n",
       " <pmid version=\"1\">19671196</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>JAMA. 2007 Jun 6;297(21):2360-72</refsource>\n",
       " <pmid version=\"1\">17545690</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Surg Pathol. 2008 Jun;32(6):799-809</refsource>\n",
       " <pmid version=\"1\">18391748</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Arch Pathol Lab Med. 2004 Jul;128(7):765-70</refsource>\n",
       " <pmid version=\"1\">15214826</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 1999 Aug 4;91(15):1310-6</refsource>\n",
       " <pmid version=\"1\">10433620</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2001 Sep;49(3):347-53</refsource>\n",
       " <pmid version=\"1\">11511555</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2000 Sep;26(1):16-7</refsource>\n",
       " <pmid version=\"1\">10973239</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 1999 Apr;8(4):607-10</refsource>\n",
       " <pmid version=\"1\">10072428</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Acta Pathol Microbiol Scand. 1965;64:31-49</refsource>\n",
       " <pmid version=\"1\">14320675</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2011 Jan;48(1):64-8</refsource>\n",
       " <pmid version=\"1\">20921021</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Endoscopy. 2010 Dec;42(12):1037-44</refsource>\n",
       " <pmid version=\"1\">20972955</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2009 May 1;18(9):1545-55</refsource>\n",
       " <pmid version=\"1\">19168852</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Endoscopy. 2013 Sep;45(9):703-7</refsource>\n",
       " <pmid version=\"1\">23990481</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2009;12(3):148-52</refsource>\n",
       " <pmid version=\"1\">19890694</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Sci. 2005 Jan;96(1):1-6</refsource>\n",
       " <pmid version=\"1\">15649247</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2009 Aug 1;125(3):666-73</refsource>\n",
       " <pmid version=\"1\">19382179</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>World J Gastroenterol. 2014 Nov 28;20(44):16433-42</refsource>\n",
       " <pmid version=\"1\">25469011</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2012 Mar;61(3):344-53</refsource>\n",
       " <pmid version=\"1\">21865403</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2008 May 1;17(9):1336-48</refsource>\n",
       " <pmid version=\"1\">18204050</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Surg Today. 2009;39(5):430-3</refsource>\n",
       " <pmid version=\"1\">19408083</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 1990 Nov 30;250(4985):1233-8</refsource>\n",
       " <pmid version=\"1\">1978757</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2000 Dec;3(4):219-225</refsource>\n",
       " <pmid version=\"1\">11984739</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 1998 May 15;280(5366):1086-8</refsource>\n",
       " <pmid version=\"1\">9582123</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2006 Apr;55(4):463-8</refsource>\n",
       " <pmid version=\"1\">16428266</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2009 Jun;136(7):2137-48</refsource>\n",
       " <pmid version=\"1\">19269290</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>World J Gastroenterol. 2006 Jan 21;12(3):354-62</refsource>\n",
       " <pmid version=\"1\">16489633</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2014 Aug;147(2):502-26</refsource>\n",
       " <pmid version=\"1\">25043945</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastrointest Endosc. 2014 Jul;80(1):78-87</refsource>\n",
       " <pmid version=\"1\">24472763</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Carcinogenesis. 2007 Sep;28(9):2013-8</refsource>\n",
       " <pmid version=\"1\">17690113</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 2002 Oct 7;87(8):888-91</refsource>\n",
       " <pmid version=\"1\">12373604</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Med. 2015 Mar;21(3):256-62</refsource>\n",
       " <pmid version=\"1\">25706875</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fam Cancer. 2014 Sep;13(3):415-21</refsource>\n",
       " <pmid version=\"1\">24838934</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Med. 2014 Jul;20(7):769-77</refsource>\n",
       " <pmid version=\"1\">24859528</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mutat. 1999;14(3):249-55</refsource>\n",
       " <pmid version=\"1\">10477433</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Carcinogenesis. 1999 Jun;20(6):1127-31</refsource>\n",
       " <pmid version=\"1\">10357799</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2014 Sep 11;513(7517):202-9</refsource>\n",
       " <pmid version=\"1\">25079317</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Commun. 2014 Nov 19;5:5477</refsource>\n",
       " <pmid version=\"1\">25407104</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2013 Mar 1;22(5):919-26</refsource>\n",
       " <pmid version=\"1\">23197654</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2014 Oct;17(4):745-9</refsource>\n",
       " <pmid version=\"1\">23812922</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 1997 Aug;41(2):142-50</refsource>\n",
       " <pmid version=\"1\">9301490</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2011 Nov;141(5):1762-72</refsource>\n",
       " <pmid version=\"1\">21889923</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2014 Jun;46(6):573-82</refsource>\n",
       " <pmid version=\"1\">24816253</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Gastroenterol. 2014 Jan;49(1):1-8</refsource>\n",
       " <pmid version=\"1\">24162382</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2012 Jul 15;131(2):367-76</refsource>\n",
       " <pmid version=\"1\">22020549</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Pathol. 2000 Sep;157(3):747-54</refsource>\n",
       " <pmid version=\"1\">10980114</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 1999 Dec;81(7):1103-10</refsource>\n",
       " <pmid version=\"1\">10584868</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Mol Med (Berl). 2006 Dec;84(12):1023-31</refsource>\n",
       " <pmid version=\"1\">16924464</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2010 Mar 1;19(5):943-52</refsource>\n",
       " <pmid version=\"1\">19965908</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2005 Apr;54(4):461-8</refsource>\n",
       " <pmid version=\"1\">15753528</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2004 Jun;203(2):681-7</refsource>\n",
       " <pmid version=\"1\">15141383</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Surg. 2007 Jun;245(6):873-9</refsource>\n",
       " <pmid version=\"1\">17522512</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2004 Jun;53(6):775-8</refsource>\n",
       " <pmid version=\"1\">15138199</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genomics. 1995 Mar 20;26(2):281-9</refsource>\n",
       " <pmid version=\"1\">7601454</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Cancer. 2011 Mar;47(4):631-9</refsource>\n",
       " <pmid version=\"1\">21106365</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2004 Jun;53(6):814-20</refsource>\n",
       " <pmid version=\"1\">15138207</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2006 May 15;15(10):1704-12</refsource>\n",
       " <pmid version=\"1\">16600987</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Genet. 2009 Jul;76(1):1-18</refsource>\n",
       " <pmid version=\"1\">19659756</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hered Cancer Clin Pract. 2014 Aug 13;12(1):18</refsource>\n",
       " <pmid version=\"1\">25180051</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2008 Nov;216(3):286-94</refsource>\n",
       " <pmid version=\"1\">18825658</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2014 Jun;46(6):583-7</refsource>\n",
       " <pmid version=\"1\">24816255</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Biol. 2014 Aug 27;15(8):428</refsource>\n",
       " <pmid version=\"1\">25315765</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell. 2014 Jun 5;157(6):1262-78</refsource>\n",
       " <pmid version=\"1\">24906146</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>JAMA Oncol. 2015 Apr;1(1):16-8</refsource>\n",
       " <pmid version=\"1\">26182297</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Mol Cell Biol. 2004 Aug;5(8):614-25</refsource>\n",
       " <pmid version=\"1\">15366705</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell Stem Cell. 2010 Jan 8;6(1):25-36</refsource>\n",
       " <pmid version=\"1\">20085740</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell Stem Cell. 2013 Dec 5;13(6):653-8</refsource>\n",
       " <pmid version=\"1\">24315439</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell Mol Life Sci. 2008 Nov;65(23):3756-88</refsource>\n",
       " <pmid version=\"1\">18726070</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2006 Feb;43(2):138-42</refsource>\n",
       " <pmid version=\"1\">15831593</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>N Engl J Med. 1999 Jan 14;340(2):77-84</refsource>\n",
       " <pmid version=\"1\">9887158</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Clin Oncol. 2011 Aug;16(4):328-34</refsource>\n",
       " <pmid version=\"1\">21301918</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2016 Feb;65(2):202-13</refsource>\n",
       " <pmid version=\"1\">25539675</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2007 Jul;56(7):926-33</refsource>\n",
       " <pmid version=\"1\">17267619</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Surg Oncol. 2005 Jun 1;90(3):114-33; discussion 133</refsource>\n",
       " <pmid version=\"1\">15895459</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2004 Dec 20;112(6):1049-55</refsource>\n",
       " <pmid version=\"1\">15386347</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Epidemiol Biomarkers Prev. 2012 Dec;21(12):2272-4</refsource>\n",
       " <pmid version=\"1\">23071139</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Sci. 2011 Oct;102(10):1782-8</refsource>\n",
       " <pmid version=\"1\">21777349</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genet Med. 2011 Jul;13(7):651-7</refsource>\n",
       " <pmid version=\"1\">21552135</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 1991 May 1;83(9):640-3</refsource>\n",
       " <pmid version=\"1\">2023282</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2012 May;61(5):774-9</refsource>\n",
       " <pmid version=\"1\">21813476</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Pathol. 2013 Mar;229(4):621-9</refsource>\n",
       " <pmid version=\"1\">23208944</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2015 Jan;148(1):126-136.e6</refsource>\n",
       " <pmid version=\"1\">25307862</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 1999 Apr;79(11-12):1935-7</refsource>\n",
       " <pmid version=\"1\">10206317</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2009 May 14;459(7244):262-5</refsource>\n",
       " <pmid version=\"1\">19329995</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2001 Dec;121(6):1348-53</refsource>\n",
       " <pmid version=\"1\">11729114</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>CA Cancer J Clin. 2011 Mar-Apr;61(2):69-90</refsource>\n",
       " <pmid version=\"1\">21296855</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2002 Jan;30(1):25-6</refsource>\n",
       " <pmid version=\"1\">11743581</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS Genet. 2014 Oct 23;10(10):e1004669</refsource>\n",
       " <pmid version=\"1\">25340522</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gut. 2012 Jul;61(7):1103-4</refsource>\n",
       " <pmid version=\"1\">22027476</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2000 Dec;119(6):1447-53</refsource>\n",
       " <pmid version=\"1\">11113065</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2013 Jul;50(7):486-9</refsource>\n",
       " <pmid version=\"1\">23709761</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2006 Jun 22;441(7096):1015-9</refsource>\n",
       " <pmid version=\"1\">16791201</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Surg Endosc. 2014 Feb;28(2):639-47</refsource>\n",
       " <pmid version=\"1\">24114514</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2016 Jan;19(1):198-205</refsource>\n",
       " <pmid version=\"1\">25616808</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Jpn J Clin Oncol. 2003 Jan;33(1):17-20</refsource>\n",
       " <pmid version=\"1\">12604719</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Genet. 2012 Sep;82(3):283-7</refsource>\n",
       " <pmid version=\"1\">21696387</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 1981 Jun;66(6):1037-52</refsource>\n",
       " <pmid version=\"1\">6941039</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015820\">Cadherins</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D004195\">Disease Models, Animal</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020022\">Genetic Predisposition to Disease</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005820\">Genetic Testing</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018095\">Germ-Line Mutation</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" type=\"Geographic\" ui=\"D007564\">Japan</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000453\">epidemiology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009386\">Neoplastic Syndromes, Hereditary</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000175\">diagnosis</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000453\">epidemiology</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013274\">Stomach Neoplasms</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000175\">diagnosis</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000453\">epidemiology</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2015</year>\n",
       " <month>02</month>\n",
       " <day>26</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2015</year>\n",
       " <month>05</month>\n",
       " <day>21</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2015</year>\n",
       " <month>6</month>\n",
       " <day>8</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2015</year>\n",
       " <month>6</month>\n",
       " <day>8</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2016</year>\n",
       " <month>5</month>\n",
       " <day>3</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">26049741</articleid>\n",
       " <articleid idtype=\"doi\">10.1007/s00535-015-1093-9</articleid>\n",
       " <articleid idtype=\"pii\">10.1007/s00535-015-1093-9</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">25714620</pmid>\n",
       " <datecompleted>\n",
       " <year>2016</year>\n",
       " <month>03</month>\n",
       " <day>07</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2016</year>\n",
       " <month>02</month>\n",
       " <day>04</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1554-8635</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>11</volume>\n",
       " <issue>3</issue>\n",
       " <pubdate>\n",
       " <year>2015</year>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Autophagy</title>\n",
       " <isoabbreviation>Autophagy</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>A rapid and high content assay that measures cyto-ID-stained autophagic compartments and estimates autophagy flux with potential clinical applications.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>560-72</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1080/15548627.2015.1017181</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>The lack of a rapid and quantitative autophagy assay has substantially hindered the development and implementation of autophagy-targeting therapies for a variety of human diseases. To address this critical issue, we developed a novel autophagy assay using the newly developed Cyto-ID fluorescence dye. We first verified that the Cyto-ID dye specifically labels autophagic compartments with minimal staining of lysosomes and endosomes. We then developed a new Cyto-ID fluorescence spectrophotometric assay that makes it possible to estimate autophagy flux based on measurements of the Cyto-ID-stained autophagic compartments. By comparing to traditional autophagy approaches, we found that this assay yielded a more sensitive, yet less variable, quantification of the stained autophagic compartments and the estimate of autophagy flux. Furthermore, we tested the potential application of this autophagy assay in high throughput research by integrating it into an RNA interference (RNAi) screen and a small molecule screen. The RNAi screen revealed WNK2 and MAP3K6 as autophagy-modulating genes, both of which inhibited the MTOR pathway. Similarly, the small molecule screen identified sanguinarine and actinomycin D as potent autophagy inducers in leukemic cells. Moreover, we successfully detected autophagy responses to kinase inhibitors and chloroquine in normal or leukemic mice using this assay. Collectively, this new Cyto-ID fluorescence spectrophotometric assay provides a rapid, reliable quantification of autophagic compartments and estimation of autophagy flux with potential applications in developing autophagy-related therapies and as a test to monitor autophagy responses in patients being treated with autophagy-modulating drugs. </abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Guo</lastname>\n",
       " <forename>Sujuan</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>a Virginia Tech Carilion Research Institute ; Roanoke , VA USA.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Liang</lastname>\n",
       " <forename>Yanping</forename>\n",
       " <initials>Y</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Murphy</lastname>\n",
       " <forename>Susan F</forename>\n",
       " <initials>SF</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Huang</lastname>\n",
       " <forename>Angela</forename>\n",
       " <initials>A</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Shen</lastname>\n",
       " <forename>Haihong</forename>\n",
       " <initials>H</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Kelly</lastname>\n",
       " <forename>Deborah F</forename>\n",
       " <initials>DF</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Sobrado</lastname>\n",
       " <forename>Pablo</forename>\n",
       " <initials>P</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Sheng</lastname>\n",
       " <forename>Zhi</forename>\n",
       " <initials>Z</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Autophagy</medlineta>\n",
       " <nlmuniqueid>101265188</nlmuniqueid>\n",
       " <issnlinking>1554-8627</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D005456\">Fluorescent Dyes</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>1CC1JFE158</registrynumber>\n",
       " <nameofsubstance ui=\"D003609\">Dactinomycin</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>886U3H6UFF</registrynumber>\n",
       " <nameofsubstance ui=\"D002738\">Chloroquine</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>8A1O1M485B</registrynumber>\n",
       " <nameofsubstance ui=\"D000068877\">Imatinib Mesylate</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.1.-</registrynumber>\n",
       " <nameofsubstance ui=\"C523482\">WNK2 protein, human</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.1</registrynumber>\n",
       " <nameofsubstance ui=\"D017346\">Protein-Serine-Threonine Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"D020930\">MAP Kinase Kinase Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"C116394\">MAP kinase kinase kinase 6</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2000 Nov 1;19(21):5720-8</refsource>\n",
       " <pmid version=\"1\">11060023</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Traffic. 2003 Nov;4(11):785-801</refsource>\n",
       " <pmid version=\"1\">14617360</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Acta Biol Acad Sci Hung. 1972;23(2):181-93</refsource>\n",
       " <pmid version=\"1\">4359852</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Tissue Cell. 1981;13(2):393-401</refsource>\n",
       " <pmid version=\"1\">7314075</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 1982 Mar;79(6):1889-92</refsource>\n",
       " <pmid version=\"1\">6952238</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Cell Biol. 1995 Jan;66(1):3-14</refsource>\n",
       " <pmid version=\"1\">7750517</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS Biol. 2005 Jul;3(7):e233</refsource>\n",
       " <pmid version=\"1\">15954801</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Ann Intern Med. 2006 Mar 7;144(5):337-43</refsource>\n",
       " <pmid version=\"1\">16520474</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Autophagy. 2005 Apr;1(1):23-36</refsource>\n",
       " <pmid version=\"1\">16874023</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Autophagy. 2007 Jul-Aug;3(4):323-8</refsource>\n",
       " <pmid version=\"1\">17387262</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2007 Aug 31;282(35):25464-74</refsource>\n",
       " <pmid version=\"1\">17595159</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2007 Nov 27;104(48):19023-8</refsource>\n",
       " <pmid version=\"1\">18024584</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Chem Biol. 2008 May;4(5):295-305</refsource>\n",
       " <pmid version=\"1\">18391949</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Chem Biol. 2008 Nov;4(11):691-9</refsource>\n",
       " <pmid version=\"1\">18849971</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2009 Apr 8;28(7):866-76</refsource>\n",
       " <pmid version=\"1\">19229297</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell. 2010 Feb 5;140(3):313-26</refsource>\n",
       " <pmid version=\"1\">20144757</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Dev Cell. 2010 Jun 15;18(6):1041-52</refsource>\n",
       " <pmid version=\"1\">20627085</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Autophagy. 2011 Aug;7(8):892-903</refsource>\n",
       " <pmid version=\"1\">21508686</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Blood. 2011 Sep 8;118(10):2840-8</refsource>\n",
       " <pmid version=\"1\">21715304</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2011 Dec 1;480(7375):113-7</refsource>\n",
       " <pmid version=\"1\">22020285</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2012 Apr 18;31(8):1931-46</refsource>\n",
       " <pmid version=\"1\">22354037</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Drug Discov. 2012 Sep;11(9):709-30</refsource>\n",
       " <pmid version=\"1\">22935804</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Autophagy. 2012 Sep;8(9):1371-82</refsource>\n",
       " <pmid version=\"1\">22895056</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Autophagy. 2012 Apr;8(4):445-544</refsource>\n",
       " <pmid version=\"1\">22966490</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Lancet. 2013 Jan 12;381(9861):125-32</refsource>\n",
       " <pmid version=\"1\">23158522</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>N Engl J Med. 2013 Feb 14;368(7):651-62</refsource>\n",
       " <pmid version=\"1\">23406030</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Histochem Cell Biol. 2013 May;139(5):659-70</refsource>\n",
       " <pmid version=\"1\">23203316</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2013 Dec 12;504(7479):291-5</refsource>\n",
       " <pmid version=\"1\">24270810</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Antioxid Redox Signal. 2014 Jan 20;20(3):460-73</refsource>\n",
       " <pmid version=\"1\">23725295</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D001343\">Autophagy</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D002470\">Cell Survival</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D002738\">Chloroquine</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D003609\">Dactinomycin</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D011992\">Endosomes</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005456\">Fluorescent Dyes</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015973\">Gene Expression Regulation, Leukemic</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D057809\">HEK293 Cells</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006367\">HeLa Cells</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000068877\">Imatinib Mesylate</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020014\">K562 Cells</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D007938\">Leukemia</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008247\">Lysosomes</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020930\">MAP Kinase Kinase Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051379\">Mice</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008810\">Mice, Inbred C57BL</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D016513\">Mice, SCID</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008856\">Microscopy, Fluorescence</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009368\">Neoplasm Transplantation</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D017346\">Protein-Serine-Threonine Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D034622\">RNA Interference</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013050\">Spectrometry, Fluorescence</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " <keywordlist owner=\"NOTNLM\">\n",
       " <keyword majortopicyn=\"N\">3-MA, 3-methyladenine</keyword>\n",
       " <keyword majortopicyn=\"N\">Cyto-ID</keyword>\n",
       " <keyword majortopicyn=\"N\">FBS, fetal bovine serum</keyword>\n",
       " <keyword majortopicyn=\"N\">GFP, green fluorescent protein</keyword>\n",
       " <keyword majortopicyn=\"N\">LAMP1, lysosomal-associated membrane protein 1</keyword>\n",
       " <keyword majortopicyn=\"N\">MAP1LC3B/LC3B, microtubule-associated protein 1 light chain 3 beta</keyword>\n",
       " <keyword majortopicyn=\"N\">MAP3K6, mitogen-activated protein kinase kinase kinase 6</keyword>\n",
       " <keyword majortopicyn=\"N\">MDC, monodansylcadaverine</keyword>\n",
       " <keyword majortopicyn=\"N\">MTOR, mechanistic target of rapamycin</keyword>\n",
       " <keyword majortopicyn=\"N\">NS, nonsilencing</keyword>\n",
       " <keyword majortopicyn=\"N\">RAB5A, member RAS oncogene family</keyword>\n",
       " <keyword majortopicyn=\"N\">RNA interference screen</keyword>\n",
       " <keyword majortopicyn=\"N\">RNAi, RNA interference</keyword>\n",
       " <keyword majortopicyn=\"N\">SQSTM1, sequestosome 1</keyword>\n",
       " <keyword majortopicyn=\"N\">WNK2, WNK lysine deficient protein kinase 2</keyword>\n",
       " <keyword majortopicyn=\"N\">autophagy</keyword>\n",
       " <keyword majortopicyn=\"N\">autophagy flux</keyword>\n",
       " <keyword majortopicyn=\"N\">autophagy response</keyword>\n",
       " <keyword majortopicyn=\"N\">mRFP, monomeric red fluorescent protein</keyword>\n",
       " <keyword majortopicyn=\"N\">shRNA, short-hairpin RNA</keyword>\n",
       " <keyword majortopicyn=\"N\">small molecule screen</keyword>\n",
       " <keyword majortopicyn=\"N\">spectrophotometric assay</keyword>\n",
       " </keywordlist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2015</year>\n",
       " <month>2</month>\n",
       " <day>26</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2015</year>\n",
       " <month>2</month>\n",
       " <day>26</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2016</year>\n",
       " <month>3</month>\n",
       " <day>8</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">25714620</articleid>\n",
       " <articleid idtype=\"doi\">10.1080/15548627.2015.1017181</articleid>\n",
       " <articleid idtype=\"pmc\">PMC4502761</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">25340522</pmid>\n",
       " <datecompleted>\n",
       " <year>2015</year>\n",
       " <month>07</month>\n",
       " <day>02</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2015</year>\n",
       " <month>10</month>\n",
       " <day>29</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Electronic-eCollection\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1553-7404</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>10</volume>\n",
       " <issue>10</issue>\n",
       " <pubdate>\n",
       " <year>2014</year>\n",
       " <month>Oct</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>PLoS genetics</title>\n",
       " <isoabbreviation>PLoS Genet.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Germline mutations in MAP3K6 are associated with familial gastric cancer.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>e1004669</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1371/journal.pgen.1004669</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Gastric cancer is among the leading causes of cancer-related deaths worldwide. While heritable forms of gastric cancer are relatively rare, identifying the genes responsible for such cases can inform diagnosis and treatment for both hereditary and sporadic cases of gastric cancer. Mutations in the E-cadherin gene, CDH1, account for 40% of the most common form of familial gastric cancer (FGC), hereditary diffuse gastric cancer (HDGC). The genes responsible for the remaining forms of FGC are currently unknown. Here we examined a large family from Maritime Canada with FGC without CDH1 mutations, and identified a germline coding variant (p.P946L) in mitogen-activated protein kinase kinase kinase 6 (MAP3K6). Based on conservation, predicted pathogenicity and a known role of the gene in cancer predisposition, MAP3K6 was considered a strong candidate and was investigated further. Screening of an additional 115 unrelated individuals with non-CDH1 FGC identified the p.P946L MAP3K6 variant, as well as four additional coding variants in MAP3K6 (p.F849Sfs*142, p.P958T, p.D200Y and p.V207G). A somatic second-hit variant (p.H506Y) was present in DNA obtained from one of the tumor specimens, and evidence of DNA hypermethylation within the MAP3K6 gene was observed in DNA from the tumor of another affected individual. These findings, together with previous evidence from mouse models that MAP3K6 acts as a tumor suppressor, and studies showing the presence of somatic mutations in MAP3K6 in non-hereditary gastric cancers and gastric cancer cell lines, point towards MAP3K6 variants as a predisposing factor for FGC. </abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Gaston</lastname>\n",
       " <forename>Daniel</forename>\n",
       " <initials>D</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hansford</lastname>\n",
       " <forename>Samantha</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, British Columbia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Oliveira</lastname>\n",
       " <forename>Carla</forename>\n",
       " <initials>C</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Expression Regulation in Cancer Group, IPATIMUP, Institute of Molecular Pathology and Immunology of the University of Porto &amp; Medical Faculty of the University of Porto, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Nightingale</lastname>\n",
       " <forename>Mathew</forename>\n",
       " <initials>M</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Pinheiro</lastname>\n",
       " <forename>Hugo</forename>\n",
       " <initials>H</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Expression Regulation in Cancer Group, IPATIMUP, Institute of Molecular Pathology and Immunology of the University of Porto &amp; Medical Faculty of the University of Porto, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Macgillivray</lastname>\n",
       " <forename>Christine</forename>\n",
       " <initials>C</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Ophthalmology and Visual Sciences, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Kaurah</lastname>\n",
       " <forename>Pardeep</forename>\n",
       " <initials>P</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, British Columbia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Rideout</lastname>\n",
       " <forename>Andrea L</forename>\n",
       " <initials>AL</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medical Genetics, IWK Health Centre, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Steele</lastname>\n",
       " <forename>Patricia</forename>\n",
       " <initials>P</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medical Genetics, IWK Health Centre, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Soares</lastname>\n",
       " <forename>Gabriela</forename>\n",
       " <initials>G</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Center of Medical Genetics Jacinto de Magalhães, Porto Hospital Center, Porto, Portugal.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Huang</lastname>\n",
       " <forename>Weei-Yuarn</forename>\n",
       " <initials>WY</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Division of Anatomical Pathology, Department of Pathology, Queen Elizabeth II Health Science Center and Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Whitehouse</lastname>\n",
       " <forename>Scott</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Blowers</lastname>\n",
       " <forename>Sarah</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Queen's Family Health Team, Kingston, Ontario, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>LeBlanc</lastname>\n",
       " <forename>Marissa A</forename>\n",
       " <initials>MA</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Jiang</lastname>\n",
       " <forename>Haiyan</forename>\n",
       " <initials>H</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Biostatistics, Princess Margaret Cancer Centre, Toronto, Ontario, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Greer</lastname>\n",
       " <forename>Wenda</forename>\n",
       " <initials>W</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Samuels</lastname>\n",
       " <forename>Mark E</forename>\n",
       " <initials>ME</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada; Centre de Recherche du CHU Ste-Justine and Department of Medicine, University of Montreal, Montreal, Quebec, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Orr</lastname>\n",
       " <forename>Andrew</forename>\n",
       " <initials>A</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada; Department of Ophthalmology and Visual Sciences, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Fernandez</lastname>\n",
       " <forename>Conrad V</forename>\n",
       " <initials>CV</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pediatrics, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Majewski</lastname>\n",
       " <forename>Jacek</forename>\n",
       " <initials>J</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Human Genetics, McGill University, Montreal, Québec, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ludman</lastname>\n",
       " <forename>Mark</forename>\n",
       " <initials>M</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medical Genetics, IWK Health Centre, Halifax, Nova Scotia, Canada; Oncogenetics Service, Institute of Medical Genetics, Meir Medical Center, Kfar Saba, Israel.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Dyack</lastname>\n",
       " <forename>Sarah</forename>\n",
       " <initials>S</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medical Genetics, IWK Health Centre, Halifax, Nova Scotia, Canada; Department of Pediatrics, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Penney</lastname>\n",
       " <forename>Lynette S</forename>\n",
       " <initials>LS</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Medical Genetics, IWK Health Centre, Halifax, Nova Scotia, Canada; Department of Pediatrics, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>McMaster</lastname>\n",
       " <forename>Christopher R</forename>\n",
       " <initials>CR</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pharmacology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Huntsman</lastname>\n",
       " <forename>David</forename>\n",
       " <initials>D</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology and Laboratory Medicine, University of British Columbia, Vancouver, British Columbia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Bedard</lastname>\n",
       " <forename>Karen</forename>\n",
       " <initials>K</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Department of Pathology, Dalhousie University, Halifax, Nova Scotia, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2014</year>\n",
       " <month>10</month>\n",
       " <day>23</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>PLoS Genet</medlineta>\n",
       " <nlmuniqueid>101239074</nlmuniqueid>\n",
       " <issnlinking>1553-7390</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"C513384\">CDH1 protein, human</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D015820\">Cadherins</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"D020930\">MAP Kinase Kinase Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"C116394\">MAP kinase kinase kinase 6</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Best Pract Res Clin Gastroenterol. 2009;23(2):147-57</refsource>\n",
       " <pmid version=\"1\">19414142</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2009 May 1;18(9):1545-55</refsource>\n",
       " <pmid version=\"1\">19168852</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Clin Oncol. 2009 Aug 20;27(24):3975-80</refsource>\n",
       " <pmid version=\"1\">19620482</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2010 Apr;7(4):248-9</refsource>\n",
       " <pmid version=\"1\">20354512</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2010 Jul;47(7):436-44</refsource>\n",
       " <pmid version=\"1\">20591882</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2010 Aug;7(8):575-6</refsource>\n",
       " <pmid version=\"1\">20676075</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Res. 2010 Sep;20(9):1297-303</refsource>\n",
       " <pmid version=\"1\">20644199</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2010 Sep 2;467(7311):52-8</refsource>\n",
       " <pmid version=\"1\">20811451</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2010 Oct 28;467(7319):1061-73</refsource>\n",
       " <pmid version=\"1\">20981092</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 2010 Dec 7;103(12):1875-84</refsource>\n",
       " <pmid version=\"1\">21063410</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2011 Jan;39(Database issue):D945-50</refsource>\n",
       " <pmid version=\"1\">20952405</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 2011 Jan 1;71(1):29-39</refsource>\n",
       " <pmid version=\"1\">21097718</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2011 Jan 11;108(2):780-5</refsource>\n",
       " <pmid version=\"1\">21187402</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2010 Dec 15;127(12):2893-917</refsource>\n",
       " <pmid version=\"1\">21351269</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Med Genet. 2011 Sep;48(9):602-5</refsource>\n",
       " <pmid version=\"1\">21785126</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2012 Jun 1;21(11):2572-87</refsource>\n",
       " <pmid version=\"1\">22378147</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Fly (Austin). 2012 Apr-Jun;6(2):80-92</refsource>\n",
       " <pmid version=\"1\">22728672</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Exp Mol Med. 2012 Aug 31;44(8):503-12</refsource>\n",
       " <pmid version=\"1\">22684242</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2012 Sep;9(9):855-6</refsource>\n",
       " <pmid version=\"1\">22936163</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2012;7(10):e46688</refsource>\n",
       " <pmid version=\"1\">23056405</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Jpn Acad Ser B Phys Biol Sci. 2012;88(8):434-53</refsource>\n",
       " <pmid version=\"1\">23060232</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Arch Ophthalmol. 2012 Nov;130(11):1425-32</refsource>\n",
       " <pmid version=\"1\">23143442</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mutat. 2013 Jan;34(1):57-65</refsource>\n",
       " <pmid version=\"1\">23033316</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Curr Protoc Hum Genet. 2013 Jan;Chapter 7:Unit7.20</refsource>\n",
       " <pmid version=\"1\">23315928</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cell Biochem. 2013 Jul;379(1-2):161-9</refsource>\n",
       " <pmid version=\"1\">23580093</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Bioinformatics. 2013 Jun 15;29(12):1504-10</refsource>\n",
       " <pmid version=\"1\">23620363</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cell Biol. 2013 Jul;33(13):2574-85</refsource>\n",
       " <pmid version=\"1\">23608533</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS Comput Biol. 2013;9(7):e1003153</refsource>\n",
       " <pmid version=\"1\">23874191</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Protein Sci. 2013 Aug;22(8):1071-7</refsource>\n",
       " <pmid version=\"1\">23776076</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Clin Oncol. 2013 Nov;10(11):643-55</refsource>\n",
       " <pmid version=\"1\">24061039</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biochim Biophys Acta. 2013 Nov;1829(11):1161-74</refsource>\n",
       " <pmid version=\"1\">23938249</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>JAMA Ophthalmol. 2014 Aug;132(8):1002-4</refsource>\n",
       " <pmid version=\"1\">24830548</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Natl Cancer Inst. 2000 Nov 1;92(21):1781-2</refsource>\n",
       " <pmid version=\"1\">11058622</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2001 Jan 1;29(1):308-11</refsource>\n",
       " <pmid version=\"1\">11125122</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mol Genet. 2001 Apr;10(7):687-92</refsource>\n",
       " <pmid version=\"1\">11257100</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2002 Jan;30(1):97-101</refsource>\n",
       " <pmid version=\"1\">11731797</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Hum Genet. 2003 May;72(5):1187-99</refsource>\n",
       " <pmid version=\"1\">12677556</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2003 Jul 1;31(13):3812-4</refsource>\n",
       " <pmid version=\"1\">12824425</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>In Silico Biol. 2003;3(3):235-40</refsource>\n",
       " <pmid version=\"1\">12954087</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Hum Genet. 2004 Dec;75(6):1106-12</refsource>\n",
       " <pmid version=\"1\">15492927</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Infect Dis. 1994 Feb;169(2):434-7</refsource>\n",
       " <pmid version=\"1\">8106778</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biometrics. 1994 Mar;50(1):118-27</refsource>\n",
       " <pmid version=\"1\">8086596</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Hum Genet. 1997 Nov;61(5):1179-88</refsource>\n",
       " <pmid version=\"1\">9345087</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1998 Mar 26;392(6674):402-5</refsource>\n",
       " <pmid version=\"1\">9537325</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Acta Pathol Microbiol Scand. 1965;64:31-49</refsource>\n",
       " <pmid version=\"1\">14320675</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Bioinformatics. 2005 Jul 15;21(14):3176-8</refsource>\n",
       " <pmid version=\"1\">15879453</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Cancer. 2006 Jun 15;118(12):3030-44</refsource>\n",
       " <pmid version=\"1\">16404738</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 2006 Oct 13;314(5797):268-74</refsource>\n",
       " <pmid version=\"1\">16959974</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2007 Mar 9;282(10):7522-31</refsource>\n",
       " <pmid version=\"1\">17210579</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2007 Mar 8;446(7132):153-8</refsource>\n",
       " <pmid version=\"1\">17344846</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Hum Genet. 2007 Sep;81(3):559-75</refsource>\n",
       " <pmid version=\"1\">17701901</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biochem Biophys Res Commun. 2007 Oct 19;362(2):454-9</refsource>\n",
       " <pmid version=\"1\">17714688</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Can J Gastroenterol. 2008 Mar;22(3):289-95</refsource>\n",
       " <pmid version=\"1\">18354758</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastroenterology. 2009 Apr;136(4):1251-60</refsource>\n",
       " <pmid version=\"1\">19245865</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Pathol. 2009 Apr;174(4):1553-63</refsource>\n",
       " <pmid version=\"1\">19246638</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2009 Apr 8;28(7):843-53</refsource>\n",
       " <pmid version=\"1\">19214184</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Bioinformatics. 2009 Jul 15;25(14):1754-60</refsource>\n",
       " <pmid version=\"1\">19451168</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015820\">Cadherins</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D004252\">DNA Mutational Analysis</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005260\">Female</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008040\">Genetic Linkage</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D020022\">Genetic Predisposition to Disease</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005838\">Genotype</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018095\">Germ-Line Mutation</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020930\">MAP Kinase Kinase Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008297\">Male</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D010375\">Pedigree</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020641\">Polymorphism, Single Nucleotide</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013274\">Stomach Neoplasms</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000473\">pathology</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2014</year>\n",
       " <month>03</month>\n",
       " <day>03</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2014</year>\n",
       " <month>08</month>\n",
       " <day>14</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2014</year>\n",
       " <month>10</month>\n",
       " <day>24</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2014</year>\n",
       " <month>10</month>\n",
       " <day>24</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2015</year>\n",
       " <month>7</month>\n",
       " <day>3</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>epublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">25340522</articleid>\n",
       " <articleid idtype=\"doi\">10.1371/journal.pgen.1004669</articleid>\n",
       " <articleid idtype=\"pii\">PGENETICS-D-14-00617</articleid>\n",
       " <articleid idtype=\"pmc\">PMC4207611</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">23800963</pmid>\n",
       " <datecompleted>\n",
       " <year>2014</year>\n",
       " <month>03</month>\n",
       " <day>05</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2017</year>\n",
       " <month>02</month>\n",
       " <day>20</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1532-298X</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>25</volume>\n",
       " <issue>6</issue>\n",
       " <pubdate>\n",
       " <year>2013</year>\n",
       " <month>Jun</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>The Plant cell</title>\n",
       " <isoabbreviation>Plant Cell</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Salt-responsive ERF1 regulates reactive oxygen species-dependent signaling during the initial response to salt stress in rice.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>2115-31</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1105/tpc.113.113068</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Early detection of salt stress is vital for plant survival and growth. Still, the molecular processes controlling early salt stress perception and signaling are not fully understood. Here, we identified salt-responsive ERF1 (SERF1), a rice (Oryza sativa) transcription factor (TF) gene that shows a root-specific induction upon salt and hydrogen peroxide (H2O2) treatment. Loss of SERF1 impairs the salt-inducible expression of genes encoding members of a mitogen-activated protein kinase (MAPK) cascade and salt tolerance-mediating TFs. Furthermore, we show that SERF1-dependent genes are H2O2 responsive and demonstrate that SERF1 binds to the promoters of MAPK kinase kinase6 (MAP3K6), MAPK5, dehydration-responsive element bindinG2A (DREB2A), and zinc finger protein179 (ZFP179) in vitro and in vivo. SERF1 also directly induces its own gene expression. In addition, SERF1 is a phosphorylation target of MAPK5, resulting in enhanced transcriptional activity of SERF1 toward its direct target genes. In agreement, plants deficient for SERF1 are more sensitive to salt stress compared with the wild type, while constitutive overexpression of SERF1 improves salinity tolerance. We propose that SERF1 amplifies the reactive oxygen species-activated MAPK cascade signal during the initial phase of salt stress and translates the salt-induced signal into an appropriate expressional response resulting in salt tolerance.</abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Schmidt</lastname>\n",
       " <forename>Romy</forename>\n",
       " <initials>R</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Institute of Biochemistry and Biology, University of Potsdam, 14476 Potsdam, Germany.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Mieulet</lastname>\n",
       " <forename>Delphine</forename>\n",
       " <initials>D</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hubberten</lastname>\n",
       " <forename>Hans-Michael</forename>\n",
       " <initials>HM</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Obata</lastname>\n",
       " <forename>Toshihiro</forename>\n",
       " <initials>T</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Hoefgen</lastname>\n",
       " <forename>Rainer</forename>\n",
       " <initials>R</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Fernie</lastname>\n",
       " <forename>Alisdair R</forename>\n",
       " <initials>AR</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Fisahn</lastname>\n",
       " <forename>Joachim</forename>\n",
       " <initials>J</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>San Segundo</lastname>\n",
       " <forename>Blanca</forename>\n",
       " <initials>B</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Guiderdoni</lastname>\n",
       " <forename>Emmanuel</forename>\n",
       " <initials>E</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Schippers</lastname>\n",
       " <forename>Jos H M</forename>\n",
       " <initials>JH</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Mueller-Roeber</lastname>\n",
       " <forename>Bernd</forename>\n",
       " <initials>B</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <databanklist completeyn=\"Y\">\n",
       " <databank>\n",
       " <databankname>GENBANK</databankname>\n",
       " <accessionnumberlist>\n",
       " <accessionnumber>JQ246414</accessionnumber>\n",
       " </accessionnumberlist>\n",
       " </databank>\n",
       " </databanklist>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2013</year>\n",
       " <month>06</month>\n",
       " <day>25</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Plant Cell</medlineta>\n",
       " <nlmuniqueid>9208688</nlmuniqueid>\n",
       " <issnlinking>1040-4651</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D008164\">Luminescent Proteins</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D016877\">Oxidants</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D010940\">Plant Proteins</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D017382\">Reactive Oxygen Species</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D014157\">Transcription Factors</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>451W47IQ8X</registrynumber>\n",
       " <nameofsubstance ui=\"D012965\">Sodium Chloride</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>BBX060AN9V</registrynumber>\n",
       " <nameofsubstance ui=\"D006861\">Hydrogen Peroxide</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"D020930\">MAP Kinase Kinase Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biosci. 2011 Mar;36(1):139-51</refsource>\n",
       " <pmid version=\"1\">21451255</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2000 Dec;24(5):655-65</refsource>\n",
       " <pmid version=\"1\">11123804</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2005 Oct;139(2):847-56</refsource>\n",
       " <pmid version=\"1\">16183833</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biochim Biophys Acta. 2006 Aug;1758(8):994-1003</refsource>\n",
       " <pmid version=\"1\">16566894</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Annu Rev Plant Biol. 2008;59:651-81</refsource>\n",
       " <pmid version=\"1\">18444910</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Genet Genomics. 2010 Sep;284(3):173-83</refsource>\n",
       " <pmid version=\"1\">20632034</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Phytochemistry. 2011 Jul;72(10):1081-91</refsource>\n",
       " <pmid version=\"1\">21146842</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Cell Biol. 1997 Sep;7(9):353-61</refsource>\n",
       " <pmid version=\"1\">17708980</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>New Phytol. 2005 Sep;167(3):645-63</refsource>\n",
       " <pmid version=\"1\">16101905</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2004 Aug;39(3):450-64</refsource>\n",
       " <pmid version=\"1\">15255873</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Plant Sci. 2011 Jun;16(6):300-9</refsource>\n",
       " <pmid version=\"1\">21482172</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Methods. 2007 Jun 08;3:7</refsource>\n",
       " <pmid version=\"1\">17559651</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Mol Biol. 2004 Aug;55(6):853-67</refsource>\n",
       " <pmid version=\"1\">15604721</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Curr Opin Plant Biol. 2003 Oct;6(5):441-5</refsource>\n",
       " <pmid version=\"1\">12972044</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Planta. 1981 Dec;153(4):376-87</refsource>\n",
       " <pmid version=\"1\">24276943</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Exp Bot. 2012 Jan;63(1):305-17</refsource>\n",
       " <pmid version=\"1\">21984648</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Methods. 2001 Dec;25(4):402-8</refsource>\n",
       " <pmid version=\"1\">11846609</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2006 Nov;48(4):535-47</refsource>\n",
       " <pmid version=\"1\">17059409</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 1996 Jan 23;93(2):765-9</refsource>\n",
       " <pmid version=\"1\">8570631</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2005 Jul;138(3):1185-94</refsource>\n",
       " <pmid version=\"1\">16009994</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2004 Nov;40(4):512-22</refsource>\n",
       " <pmid version=\"1\">15500467</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2002 Jun;30(5):601-9</refsource>\n",
       " <pmid version=\"1\">12047634</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Adv Bioinformatics. 2008;2008:420747</refsource>\n",
       " <pmid version=\"1\">19956698</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Transgenic Res. 2008 Oct;17(5):985-9</refsource>\n",
       " <pmid version=\"1\">18594998</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Plant Physiol. 2007 Nov;164(11):1429-35</refsource>\n",
       " <pmid version=\"1\">17223222</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell. 2003 Mar;15(3):745-59</refsource>\n",
       " <pmid version=\"1\">12615946</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell. 2011 Jan;23(1):412-27</refsource>\n",
       " <pmid version=\"1\">21239643</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell. 1999 Dec;11(12):2393-405</refsource>\n",
       " <pmid version=\"1\">10590166</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Genomics. 2007;8:175</refsource>\n",
       " <pmid version=\"1\">17577400</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2008 Dec;148(4):1938-52</refsource>\n",
       " <pmid version=\"1\">18931143</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2003 Aug;132(4):1961-72</refsource>\n",
       " <pmid version=\"1\">12913152</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Agric Food Chem. 2006 Aug 23;54(17):6159-65</refsource>\n",
       " <pmid version=\"1\">16910702</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2003 Apr;131(4):1843-54</refsource>\n",
       " <pmid version=\"1\">12692344</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Bioinformatics. 2008;9:465</refsource>\n",
       " <pmid version=\"1\">18976492</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Signal Behav. 2009 Aug;4(8):718-26</refsource>\n",
       " <pmid version=\"1\">19820346</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell. 2006 Jul;18(7):1542-57</refsource>\n",
       " <pmid version=\"1\">16809814</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Plant Sci. 2002 May;7(5):193-5</refsource>\n",
       " <pmid version=\"1\">11992820</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2012 Jan;40(Database issue):D1178-86</refsource>\n",
       " <pmid version=\"1\">22110026</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2003 Oct;133(2):462-9</refsource>\n",
       " <pmid version=\"1\">14555774</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>New Phytol. 2010 Nov;188(3):762-73</refsource>\n",
       " <pmid version=\"1\">20796215</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Mol Biol. 2012 Oct;80(3):241-53</refsource>\n",
       " <pmid version=\"1\">22875749</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell Death Dis. 2010;1:e89</refsource>\n",
       " <pmid version=\"1\">21368862</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>New Phytol. 2011 Apr;190(2):442-56</refsource>\n",
       " <pmid version=\"1\">20840511</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biotechnol Lett. 2011 Aug;33(8):1689-97</refsource>\n",
       " <pmid version=\"1\">21528404</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biochim Biophys Acta. 2007 Apr;1769(4):220-7</refsource>\n",
       " <pmid version=\"1\">17434609</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2008;3(3):e1829</refsource>\n",
       " <pmid version=\"1\">18350165</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Methods. 2011 Sep 30;7(1):30</refsource>\n",
       " <pmid version=\"1\">21961694</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Plant Sci. 2005 Jul;10(7):339-46</refsource>\n",
       " <pmid version=\"1\">15953753</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Mol Biol. 2009 Jan;69(1-2):133-53</refsource>\n",
       " <pmid version=\"1\">18931976</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell Rep. 2011 Nov;30(11):2097-104</refsource>\n",
       " <pmid version=\"1\">21735232</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2010 Aug;63(4):599-612</refsource>\n",
       " <pmid version=\"1\">20525005</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2010 Feb;152(2):876-90</refsource>\n",
       " <pmid version=\"1\">20007444</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Methods Mol Biol. 2000;132:365-86</refsource>\n",
       " <pmid version=\"1\">10547847</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2012 Nov 14;31(22):4359-70</refsource>\n",
       " <pmid version=\"1\">23064146</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genes Dev. 2009 Jan 1;23(1):80-92</refsource>\n",
       " <pmid version=\"1\">19095804</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Planta. 1999 Jul;209(1):104-11</refsource>\n",
       " <pmid version=\"1\">10467036</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Plant Sci. 2004 Oct;9(10):490-8</refsource>\n",
       " <pmid version=\"1\">15465684</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Plant Sci. 2010 Nov;15(11):595-9</refsource>\n",
       " <pmid version=\"1\">20832349</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2009 Jul;150(3):1368-79</refsource>\n",
       " <pmid version=\"1\">19429605</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell Physiol. 2011 Sep;52(9):1583-602</refsource>\n",
       " <pmid version=\"1\">21828102</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2007 Dec;145(4):1192-200</refsource>\n",
       " <pmid version=\"1\">17981986</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Theor Appl Genet. 2003 May;106(8):1396-408</refsource>\n",
       " <pmid version=\"1\">12677401</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cell. 2004 Jul 2;15(1):141-52</refsource>\n",
       " <pmid version=\"1\">15225555</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2006 Aug 29;103(35):12987-92</refsource>\n",
       " <pmid version=\"1\">16924117</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell Physiol. 2010 Apr;51(4):497-523</refsource>\n",
       " <pmid version=\"1\">20208064</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2012 Oct 16;109(42):17129-34</refsource>\n",
       " <pmid version=\"1\">23027948</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2010 Apr;152(4):1863-73</refsource>\n",
       " <pmid version=\"1\">20154096</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cells. 2012 Sep;34(3):271-8</refsource>\n",
       " <pmid version=\"1\">22886763</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell Mol Life Sci. 2012 Oct;69(19):3245-57</refsource>\n",
       " <pmid version=\"1\">22842779</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2007 Apr;50(2):347-63</refsource>\n",
       " <pmid version=\"1\">17376166</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2011;6(11):e26982</refsource>\n",
       " <pmid version=\"1\">22073231</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2007 Jan;35(Database issue):D883-7</refsource>\n",
       " <pmid version=\"1\">17145706</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Integr Plant Biol. 2012 Jul;54(7):500-10</refsource>\n",
       " <pmid version=\"1\">22693960</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Phytochemistry. 2011 Jul;72(10):1136-44</refsource>\n",
       " <pmid version=\"1\">21477822</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2003 Feb;33(4):751-63</refsource>\n",
       " <pmid version=\"1\">12609047</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Physiol. 2006 Feb;140(2):411-32</refsource>\n",
       " <pmid version=\"1\">16407444</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Plant Biol. 2010;10:16</refsource>\n",
       " <pmid version=\"1\">20100339</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant J. 2007 Oct;52(1):185-95</refsource>\n",
       " <pmid version=\"1\">17662028</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>AoB Plants. 2012;2012:pls011</refsource>\n",
       " <pmid version=\"1\">22616023</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 2003 Jun 2;22(11):2623-33</refsource>\n",
       " <pmid version=\"1\">12773379</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Cell. 2006 Oct;18(10):2767-81</refsource>\n",
       " <pmid version=\"1\">17041150</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Plant Mol Biol. 2008 May;67(1-2):169-81</refsource>\n",
       " <pmid version=\"1\">18273684</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>FEBS Lett. 2008 Apr 2;582(7):1037-43</refsource>\n",
       " <pmid version=\"1\">18325341</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Exp Bot. 2010 Jun;61(10):2807-18</refsource>\n",
       " <pmid version=\"1\">20460361</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018507\">Gene Expression Regulation, Developmental</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018506\">Gene Expression Regulation, Plant</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D055785\">Gene Knockdown Techniques</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006861\">Hydrogen Peroxide</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008164\">Luminescent Proteins</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020930\">MAP Kinase Kinase Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018613\">Microscopy, Confocal</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008957\">Models, Genetic</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008969\">Molecular Sequence Data</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009154\">Mutation</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D012275\">Oryza</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000254\">growth &amp; development</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D016877\">Oxidants</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D010940\">Plant Proteins</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D030821\">Plants, Genetically Modified</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D011401\">Promoter Regions, Genetic</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D011485\">Protein Binding</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D017382\">Reactive Oxygen Species</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020133\">Reverse Transcriptase Polymerase Chain Reaction</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D055049\">Salt-Tolerance</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015398\">Signal Transduction</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D012965\">Sodium Chloride</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013312\">Stress, Physiological</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D014157\">Transcription Factors</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2013</year>\n",
       " <month>6</month>\n",
       " <day>27</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2013</year>\n",
       " <month>6</month>\n",
       " <day>27</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2014</year>\n",
       " <month>3</month>\n",
       " <day>7</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">23800963</articleid>\n",
       " <articleid idtype=\"pii\">tpc.113.113068</articleid>\n",
       " <articleid idtype=\"doi\">10.1105/tpc.113.113068</articleid>\n",
       " <articleid idtype=\"pmc\">PMC3723616</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">21097718</pmid>\n",
       " <datecompleted>\n",
       " <year>2011</year>\n",
       " <month>02</month>\n",
       " <day>01</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2018</year>\n",
       " <month>04</month>\n",
       " <day>25</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1538-7445</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>71</volume>\n",
       " <issue>1</issue>\n",
       " <pubdate>\n",
       " <year>2011</year>\n",
       " <month>Jan</month>\n",
       " <day>01</day>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Cancer research</title>\n",
       " <isoabbreviation>Cancer Res.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Genetic and structural variation in the gastric cancer kinome revealed through targeted deep sequencing.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>29-39</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1158/0008-5472.CAN-10-1749</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Genetic alterations in kinases have been linked to multiple human pathologies. To explore the landscape of kinase genetic variation in gastric cancer (GC), we used targeted, paired-end deep sequencing to analyze 532 protein and phosphoinositide kinases in 14 GC cell lines. We identified 10,604 single-nucleotide variants (SNV) in kinase exons including greater than 300 novel nonsynonymous SNVs. Family-wise analysis of the nonsynonymous SNVs revealed a significant enrichment in mitogen-activated protein kinase (MAPK)-related genes (P &lt; 0.01), suggesting a preferential involvement of this kinase family in GC. A potential antioncogenic role for MAP2K4, a gene exhibiting recurrent alterations in 2 lines, was functionally supported by siRNA knockdown and overexpression studies in wild-type and MAP2K4 variant lines. The deep sequencing data also revealed novel, large-scale structural rearrangement events involving kinases including gene fusions involving CDK12 and the ERBB2 receptor tyrosine kinase in MKN7 cells. Integrating SNVs and copy number alterations, we identified Hs746T as a cell line exhibiting both splice-site mutations and genomic amplification of MET, resulting in MET protein overexpression. When applied to primary GCs, we identified somatic mutations in 8 kinases, 4 of which were recurrently altered in both primary tumors and cell lines (MAP3K6, STK31, FER, and CDKL5). These results demonstrate that how targeted deep sequencing approaches can deliver unprecedented multilevel characterization of a medically and pharmacologically relevant gene family. The catalog of kinome genetic variants assembled here may broaden our knowledge on kinases and provide useful information on genetic alterations in GC.</abstracttext>\n",
       " <copyrightinformation>© 2011 AACR.</copyrightinformation>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zang</lastname>\n",
       " <forename>Zhi Jiang</forename>\n",
       " <initials>ZJ</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Cellular and Molecular Research, National Cancer Centre, Singapore.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ong</lastname>\n",
       " <forename>Choon Kiat</forename>\n",
       " <initials>CK</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Cutcutache</lastname>\n",
       " <forename>Ioana</forename>\n",
       " <initials>I</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Yu</lastname>\n",
       " <forename>Willie</forename>\n",
       " <initials>W</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhang</lastname>\n",
       " <forename>Shen Li</forename>\n",
       " <initials>SL</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Huang</lastname>\n",
       " <forename>Dachuan</forename>\n",
       " <initials>D</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ler</lastname>\n",
       " <forename>Lian Dee</forename>\n",
       " <initials>LD</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Dykema</lastname>\n",
       " <forename>Karl</forename>\n",
       " <initials>K</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Gan</lastname>\n",
       " <forename>Anna</forename>\n",
       " <initials>A</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Tao</lastname>\n",
       " <forename>Jiong</forename>\n",
       " <initials>J</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Lim</lastname>\n",
       " <forename>Siyu</forename>\n",
       " <initials>S</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Liu</lastname>\n",
       " <forename>Yujing</forename>\n",
       " <initials>Y</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Futreal</lastname>\n",
       " <forename>P Andrew</forename>\n",
       " <initials>PA</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Grabsch</lastname>\n",
       " <forename>Heike</forename>\n",
       " <initials>H</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Furge</lastname>\n",
       " <forename>Kyle A</forename>\n",
       " <initials>KA</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Goh</lastname>\n",
       " <forename>Liang Kee</forename>\n",
       " <initials>LK</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Rozen</lastname>\n",
       " <forename>Steve</forename>\n",
       " <initials>S</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Teh</lastname>\n",
       " <forename>Bin Tean</forename>\n",
       " <initials>BT</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Tan</lastname>\n",
       " <forename>Patrick</forename>\n",
       " <initials>P</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <grantlist completeyn=\"Y\">\n",
       " <grant>\n",
       " <grantid>077012</grantid>\n",
       " <acronym>NULL</acronym>\n",
       " <agency>Wellcome Trust</agency>\n",
       " <country>United Kingdom</country>\n",
       " </grant>\n",
       " </grantlist>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2010</year>\n",
       " <month>11</month>\n",
       " <day>19</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Cancer Res</medlineta>\n",
       " <nlmuniqueid>2984705R</nlmuniqueid>\n",
       " <issnlinking>0008-5472</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.-</registrynumber>\n",
       " <nameofsubstance ui=\"D011494\">Protein Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2005 Nov 15;102(46):16842-7</refsource>\n",
       " <pmid version=\"1\">16269541</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Gastric Cancer. 2005;8(4):249-52</refsource>\n",
       " <pmid version=\"1\">16328600</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Neuron. 2006 Jan 5;49(1):3-4</refsource>\n",
       " <pmid version=\"1\">16387632</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Bioinformatics. 2006 May 1;22(9):1036-46</refsource>\n",
       " <pmid version=\"1\">16500937</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mutat. 2006 Oct;27(10):1060-1</refsource>\n",
       " <pmid version=\"1\">16941478</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2007 Mar 8;446(7132):153-8</refsource>\n",
       " <pmid version=\"1\">17344846</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Oncogene. 2007 May 14;26(22):3279-90</refsource>\n",
       " <pmid version=\"1\">17496922</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2007 Nov;4(11):903-5</refsource>\n",
       " <pmid version=\"1\">17934467</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2007 Nov;4(11):931-6</refsource>\n",
       " <pmid version=\"1\">17934468</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Methods. 2007 Nov;4(11):907-9</refsource>\n",
       " <pmid version=\"1\">17934469</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2007 Dec;39(12):1522-7</refsource>\n",
       " <pmid version=\"1\">17982454</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Breast Cancer Res. 2008;10(2):R36</refsource>\n",
       " <pmid version=\"1\">18435854</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2008 Jul 8;105(27):9296-301</refsource>\n",
       " <pmid version=\"1\">18599465</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Biotechnol. 2008 Oct;26(10):1135-45</refsource>\n",
       " <pmid version=\"1\">18846087</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Res. 2008 Nov;18(11):1851-8</refsource>\n",
       " <pmid version=\"1\">18714091</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Eur J Cancer. 2000 May;36(7):925-31</refsource>\n",
       " <pmid version=\"1\">10785599</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2000 May 26;275(21):15728-32</refsource>\n",
       " <pmid version=\"1\">10748105</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Pharmacol Rev. 2000 Sep;52(3):375-414</refsource>\n",
       " <pmid version=\"1\">10977868</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2001 Jan 1;29(1):308-11</refsource>\n",
       " <pmid version=\"1\">11125122</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Res. 2001 May;11(5):863-74</refsource>\n",
       " <pmid version=\"1\">11337480</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2002 Sep 1;30(17):3894-900</refsource>\n",
       " <pmid version=\"1\">12202775</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 2002 Dec 6;298(5600):1912-34</refsource>\n",
       " <pmid version=\"1\">12471243</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Cell. 2003 Jun;3(6):519-23</refsource>\n",
       " <pmid version=\"1\">12842080</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Lancet. 2003 Jul 26;362(9380):305-15</refsource>\n",
       " <pmid version=\"1\">12892963</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mutat Res. 2004 Jan 10;557(1):41-51</refsource>\n",
       " <pmid version=\"1\">14706517</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2004 Sep;36(9):949-51</refsource>\n",
       " <pmid version=\"1\">15286789</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Biol. 2004;5(10):R80</refsource>\n",
       " <pmid version=\"1\">15461798</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1986 Jan 16-22;319(6050):230-4</refsource>\n",
       " <pmid version=\"1\">3003577</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Lancet. 1986 Apr 5;1(8484):765-7</refsource>\n",
       " <pmid version=\"1\">2870269</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cell. 1987 Sep 11;50(6):823-9</refsource>\n",
       " <pmid version=\"1\">3113737</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Science. 1989 Jul 7;245(4913):66-8</refsource>\n",
       " <pmid version=\"1\">2544998</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 1996 Aug;13(4):485-8</refsource>\n",
       " <pmid version=\"1\">8696348</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 1997 Oct 1;57(19):4177-82</refsource>\n",
       " <pmid version=\"1\">9331070</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1998 Jan 8;391(6663):184-7</refsource>\n",
       " <pmid version=\"1\">9428765</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Oncol. 1998 Sep;13(3):601-4</refsource>\n",
       " <pmid version=\"1\">9683800</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genes Chromosomes Cancer. 1999 Apr;24(4):299-305</refsource>\n",
       " <pmid version=\"1\">10092127</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Clin Genet. 2005 Jan;67(1):81-6</refsource>\n",
       " <pmid version=\"1\">15617552</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Gastroenterol. 2004 Dec;39(12):1210-4</refsource>\n",
       " <pmid version=\"1\">15622488</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Thorac Oncol. 2009 Jan;4(1):5-11</refsource>\n",
       " <pmid version=\"1\">19096300</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Rev Cancer. 2009 Jan;9(1):28-39</refsource>\n",
       " <pmid version=\"1\">19104514</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Biotechnol. 2009 Feb;27(2):182-9</refsource>\n",
       " <pmid version=\"1\">19182786</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Genomics. 2009;10:22</refsource>\n",
       " <pmid version=\"1\">19144180</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Bioinformatics. 2009 May 15;25(10):1231-5</refsource>\n",
       " <pmid version=\"1\">19297348</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>BMC Med Genet. 2009;10:69</refsource>\n",
       " <pmid version=\"1\">19615099</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>PLoS One. 2009;4(10):e7464</refsource>\n",
       " <pmid version=\"1\">19826477</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Genome Biol. 2009;10(10):R116</refsource>\n",
       " <pmid version=\"1\">19835619</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nucleic Acids Res. 2010 Jan;38(Database issue):D652-7</refsource>\n",
       " <pmid version=\"1\">19906727</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Hum Genet. 2010 Jan;86(1):72-6</refsource>\n",
       " <pmid version=\"1\">20004881</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Hum Mutat. 2010 Apr;31(4):494-9</refsource>\n",
       " <pmid version=\"1\">20151403</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2010 Aug 12;466(7308):869-73</refsource>\n",
       " <pmid version=\"1\">20668451</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000595\">Amino Acid Sequence</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D045744\">Cell Line, Tumor</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018628\">Gene Dosage</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D014644\">Genetic Variation</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020935\">MAP Kinase Signaling System</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008969\">Molecular Sequence Data</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D011494\">Protein Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000737\">chemistry</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D017386\">Sequence Homology, Amino Acid</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015398\">Signal Transduction</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013274\">Stomach Neoplasms</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000201\">enzymology</qualifiername>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000473\">pathology</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " <otherid source=\"NLM\">EMS52698</otherid>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2010</year>\n",
       " <month>11</month>\n",
       " <day>25</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2010</year>\n",
       " <month>11</month>\n",
       " <day>26</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2011</year>\n",
       " <month>2</month>\n",
       " <day>2</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">21097718</articleid>\n",
       " <articleid idtype=\"pii\">0008-5472.CAN-10-1749</articleid>\n",
       " <articleid idtype=\"doi\">10.1158/0008-5472.CAN-10-1749</articleid>\n",
       " <articleid idtype=\"pmc\">PMC3719377</articleid>\n",
       " <articleid idtype=\"mid\">EMS52698</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">21040762</pmid>\n",
       " <datecompleted>\n",
       " <year>2011</year>\n",
       " <month>01</month>\n",
       " <day>18</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2013</year>\n",
       " <month>11</month>\n",
       " <day>21</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1879-3169</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>200</volume>\n",
       " <issue>1-2</issue>\n",
       " <pubdate>\n",
       " <year>2011</year>\n",
       " <month>Jan</month>\n",
       " <day>15</day>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Toxicology letters</title>\n",
       " <isoabbreviation>Toxicol. Lett.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Global gene expression profiles of MT knockout and wild-type mice in the condition of doxorubicin-induced cardiomyopathy.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>77-87</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1016/j.toxlet.2010.10.017</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Increasing evidence from in vivo and in vitro studies has indicated that MT exerts protective effects against DOX-induced cardiotoxicity; however the underlying precise mechanisms still remain an enigma. Therefore, the present study was designed using MT knockout mice in concert with genomic approaches to explore the possible molecular and cellular mechanisms in terms of the genetic network changes. MT-I/II null (MT⁻/⁻) mice and corresponding wild-type mice (MT+/+) were administrated with a single dose of DOX (15 mg/kg, i.p.) or equal volume of saline. Animals were sacrificed on the 4th day after DOX administration and samples were collected for further analyses. Global gene expression profiles of cardiac mRNA from two genotype mice revealed that 381 characteristically MT-responsive genes were identified between MT+/+ mice and MT⁻/⁻ mice in response to DOX, including fos, ucp3, car3, atf3, map3k6, etc. Functional analysis implied MAPK signaling pathway, p53 signaling pathway, Jak-STAT signaling pathway, PPAR signaling pathway, Wnt signaling pathway, etc. might be involved to mediate the protection of DOX cardiomyopathy by MT. Results from the present study not only validated the previously reported possible mechanisms of MT protection against DOX toxicity, but also provided new clues into the molecular mechanisms involved in this process.</abstracttext>\n",
       " <copyrightinformation>Copyright © 2010 Elsevier Ireland Ltd. All rights reserved.</copyrightinformation>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Shuai</lastname>\n",
       " <forename>Yi</forename>\n",
       " <initials>Y</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Evaluation and Research Center for Toxicology, Institute of Disease Control and Prevention, Academy of Military Medical Science, 20 Dongdajie Street, Fengtai District, Beijing 100071, PR China.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Guo</lastname>\n",
       " <forename>Jun</forename>\n",
       " <initials>J</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Dong</lastname>\n",
       " <forename>Yansheng</forename>\n",
       " <initials>Y</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhong</lastname>\n",
       " <forename>Weijian</forename>\n",
       " <initials>W</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Xiao</lastname>\n",
       " <forename>Ping</forename>\n",
       " <initials>P</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhou</lastname>\n",
       " <forename>Tong</forename>\n",
       " <initials>T</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Zhang</lastname>\n",
       " <forename>Lishi</forename>\n",
       " <initials>L</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Peng</lastname>\n",
       " <forename>Shuangqing</forename>\n",
       " <initials>S</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2010</year>\n",
       " <month>10</month>\n",
       " <day>30</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>Netherlands</country>\n",
       " <medlineta>Toxicol Lett</medlineta>\n",
       " <nlmuniqueid>7709027</nlmuniqueid>\n",
       " <issnlinking>0378-4274</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>80168379AG</registrynumber>\n",
       " <nameofsubstance ui=\"D004317\">Doxorubicin</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>9038-94-2</registrynumber>\n",
       " <nameofsubstance ui=\"D008668\">Metallothionein</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 1.1.1.27</registrynumber>\n",
       " <nameofsubstance ui=\"D007770\">L-Lactate Dehydrogenase</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.3.2</registrynumber>\n",
       " <nameofsubstance ui=\"D003402\">Creatine Kinase</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009202\">Cardiomyopathies</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000139\">chemically induced</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D003402\">Creatine Kinase</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D004317\">Doxorubicin</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D020869\">Gene Expression Profiling</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005796\">Genes</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D007770\">L-Lactate Dehydrogenase</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008297\">Male</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008668\">Metallothionein</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051379\">Mice</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D018345\">Mice, Knockout</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020411\">Oligonucleotide Array Sequence Analysis</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D014158\">Transcription, Genetic</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2010</year>\n",
       " <month>08</month>\n",
       " <day>20</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"revised\">\n",
       " <year>2010</year>\n",
       " <month>10</month>\n",
       " <day>22</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2010</year>\n",
       " <month>10</month>\n",
       " <day>25</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2010</year>\n",
       " <month>11</month>\n",
       " <day>3</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2010</year>\n",
       " <month>11</month>\n",
       " <day>3</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2011</year>\n",
       " <month>1</month>\n",
       " <day>19</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">21040762</articleid>\n",
       " <articleid idtype=\"pii\">S0378-4274(10)01742-X</articleid>\n",
       " <articleid idtype=\"doi\">10.1016/j.toxlet.2010.10.017</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">19558237</pmid>\n",
       " <datecompleted>\n",
       " <year>2010</year>\n",
       " <month>02</month>\n",
       " <day>01</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2009</year>\n",
       " <month>06</month>\n",
       " <day>29</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1091-7691</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>21 Suppl 1</volume>\n",
       " <pubdate>\n",
       " <year>2009</year>\n",
       " <month>Jul</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Inhalation toxicology</title>\n",
       " <isoabbreviation>Inhal Toxicol</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Impact of nose-only exposure system on pulmonary gene expression.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>74-82</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.1080/08958370902962309</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Nose-only exposure is used to study the distribution and toxicity of airborne contaminants. Restraint of animals in nose-only tubes causes stress, but the impact on pulmonary mRNA levels is unknown. Since stress and xenobiotics activate common pathways, we assessed whether nose-only exposure would alter expression of toxicologically relevant genes in the lungs. To identify candidate genes for further analysis, we first interrogated microarray data to examine time-dependent changes in gene expression in air-control animals from a nose-only inhalation study involving male wild-type C57BL/6 mice and transgenic tumor necrosis factor (TNF)-alpha over-expressing littermates. Comparison of transcript levels immediately and 24 h after a single 4-h nose-only exposure to air revealed differential expression of 280 genes (false discovery rate-adjusted, p &lt; .05). Functional analysis revealed enrichment of immune response, apoptosis, and signalling terms, consistent with effects of restraint stress. We then selected a subset of target genes for comparison of naive animals and air-exposed animals from the inhalation study by real-time polymerase chain reaction (PCR). Expression of genes involved in stress (BNIP, sestrin-1, CDKN1A [p21], GADD45 gamma), glucocorticoid-response (GILZ, Sgk), and signal transduction (MAP3K6, C/EBP-delta) was increased as a result of nose-only exposure (p &lt; .05). In contrast, proinflammatory factors (lymphotoxin-beta, chemokine receptor CXCR5) were decreased (p &lt; .05). Immune gene responses observed in wild-type animals were reduced in animals with lung inflammation, indicating that pathological states can modify the response to nose-only exposure. Observed responses may warrant consideration in the evaluation of materials delivered by nose-only inhalation, and suggest that incorporation of naive animals into nose-only studies should be considered as a best practice.</abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Thomson</lastname>\n",
       " <forename>Errol M</forename>\n",
       " <initials>EM</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Hazard Identification Division, Environmental Health Studies and Research Bureau, Safe Environments Programme, Health Canada, Ottawa, Canada.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Williams</lastname>\n",
       " <forename>Andrew</forename>\n",
       " <initials>A</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Yauk</lastname>\n",
       " <forename>Carole L</forename>\n",
       " <initials>CL</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Vincent</lastname>\n",
       " <forename>Renaud</forename>\n",
       " <initials>R</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " </publicationtypelist>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>England</country>\n",
       " <medlineta>Inhal Toxicol</medlineta>\n",
       " <nlmuniqueid>8910739</nlmuniqueid>\n",
       " <issnlinking>0895-8378</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D000393\">Air Pollutants</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D012333\">RNA, Messenger</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D014409\">Tumor Necrosis Factor-alpha</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000393\">Air Pollutants</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000633\">toxicity</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020869\">Gene Expression Profiling</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005786\">Gene Expression Regulation</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D019570\">Inhalation Exposure</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008168\">Lung</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000276\">immunology</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008297\">Male</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051379\">Mice</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008810\">Mice, Inbred C57BL</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008822\">Mice, Transgenic</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020411\">Oligonucleotide Array Sequence Analysis</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D012333\">RNA, Messenger</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D012149\">Restraint, Physical</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000009\">adverse effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020133\">Reverse Transcriptase Polymerase Chain Reaction</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013315\">Stress, Psychological</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013997\">Time Factors</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"Y\" ui=\"D018675\">Toxicity Tests</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D014409\">Tumor Necrosis Factor-alpha</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015854\">Up-Regulation</descriptorname>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2009</year>\n",
       " <month>6</month>\n",
       " <day>30</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2009</year>\n",
       " <month>6</month>\n",
       " <day>30</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2010</year>\n",
       " <month>2</month>\n",
       " <day>2</day>\n",
       " <hour>6</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">19558237</articleid>\n",
       " <articleid idtype=\"doi\">10.1080/08958370902962309</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">19246638</pmid>\n",
       " <datecompleted>\n",
       " <year>2009</year>\n",
       " <month>04</month>\n",
       " <day>28</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2014</year>\n",
       " <month>12</month>\n",
       " <day>10</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Electronic\">1525-2191</issn>\n",
       " <journalissue citedmedium=\"Internet\">\n",
       " <volume>174</volume>\n",
       " <issue>4</issue>\n",
       " <pubdate>\n",
       " <year>2009</year>\n",
       " <month>Apr</month>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>The American journal of pathology</title>\n",
       " <isoabbreviation>Am. J. Pathol.</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Mitogen-activated protein 3 kinase 6 mediates angiogenic and tumorigenic effects via vascular endothelial growth factor expression.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>1553-63</medlinepgn>\n",
       " </pagination>\n",
       " <elocationid eidtype=\"doi\" validyn=\"Y\">10.2353/ajpath.2009.080190</elocationid>\n",
       " <abstract>\n",
       " <abstracttext>Genome-wide screening using a small interfering RNA (siRNA) library has revealed novel molecules that are involved in a wide range of physiological responses. The expression of vascular endothelial growth factor (VEGF) is increased under hypoxic conditions, and plays a crucial role in tumor angiogenesis and tissue responses to ischemia. Here, we used a siRNA expression vector library to elucidate molecules that modify VEGF expression. Screening using an siRNA library revealed that MAPKKK6 (MEKK6/MAP3K6) regulates VEGF expression under both normoxic and hypoxic conditions in vitro, although the biological function of MAP3K6 remains unknown. Attenuation of VEGF expression by MAP3K6 inhibition was demonstrated by transient transfection of double-stranded RNA as well as by stable transfection of short hairpin RNA-expressing vectors against MAP3K6. Conditioned medium of MAP3K6-knocked down cells attenuated both endothelial proliferation and capillary network formation in a VEGF-dependent manner in vitro. In addition, tumor cells with down-regulation of MAP3K6 expression showed significant suppression of tumor growth in vivo, which was accompanied by significant repression of vessel formation and VEGF expression in these tumors. The results of this study suggest that MAP3K6 regulates VEGF expression in both normoxia and hypoxia, and that regulation of VEGF by MAP3K6 may play a crucial role in both angiogenesis and tumorigenesis.</abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Eto</lastname>\n",
       " <forename>Nobuaki</forename>\n",
       " <initials>N</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Division of Nephrology and Endocrinology, University of Tokyo School of Medicine, 7-3-1, Hongo, Bunkyo-ku, Tokyo, 113-8655, Japan.</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Miyagishi</lastname>\n",
       " <forename>Makoto</forename>\n",
       " <initials>M</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Inagi</lastname>\n",
       " <forename>Reiko</forename>\n",
       " <initials>R</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Fujita</lastname>\n",
       " <forename>Toshiro</forename>\n",
       " <initials>T</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Nangaku</lastname>\n",
       " <forename>Masaomi</forename>\n",
       " <initials>M</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " <publicationtype ui=\"D013485\">Research Support, Non-U.S. Gov't</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2009</year>\n",
       " <month>02</month>\n",
       " <day>26</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Am J Pathol</medlineta>\n",
       " <nlmuniqueid>0370502</nlmuniqueid>\n",
       " <issnlinking>0002-9440</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D034741\">RNA, Small Interfering</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D042461\">Vascular Endothelial Growth Factor A</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"D020930\">MAP Kinase Kinase Kinases</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>EC 2.7.11.25</registrynumber>\n",
       " <nameofsubstance ui=\"C116394\">MAP kinase kinase kinase 6</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>AIM</citationsubset>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <commentscorrectionslist>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Anticancer Res. 2001 Nov-Dec;21(6B):4373-82</refsource>\n",
       " <pmid version=\"1\">11908693</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Metastasis Rev. 2000;19(1-2):139-45</refsource>\n",
       " <pmid version=\"1\">11191053</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Biotechnol. 2002 May;20(5):497-500</refsource>\n",
       " <pmid version=\"1\">11981564</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Biotechnol. 2002 May;20(5):500-5</refsource>\n",
       " <pmid version=\"1\">11981565</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Biotechnol. 2002 May;20(5):505-8</refsource>\n",
       " <pmid version=\"1\">11981566</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2002 Apr 30;99(9):6047-52</refsource>\n",
       " <pmid version=\"1\">11972060</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Trends Mol Med. 2001 Aug;7(8):345-50</refsource>\n",
       " <pmid version=\"1\">11516994</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Br J Cancer. 2002 May 20;86(10):1566-77</refsource>\n",
       " <pmid version=\"1\">12085206</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Cell. 2002 Mar;1(2):193-202</refsource>\n",
       " <pmid version=\"1\">12086877</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nat Genet. 2003 Jan;33(1):40-8</refsource>\n",
       " <pmid version=\"1\">12447374</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2003 Jan 16;421(6920):231-7</refsource>\n",
       " <pmid version=\"1\">12529635</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2005 Jul 8;280(27):25687-96</refsource>\n",
       " <pmid version=\"1\">15845550</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Lab Invest. 2005 Oct;85(10):1292-307</refsource>\n",
       " <pmid version=\"1\">16127428</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Am J Physiol Renal Physiol. 2005 Nov;289(5):F1123-33</refsource>\n",
       " <pmid version=\"1\">15956779</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biomed Sci. 2006 Jan;13(1):143-56</refsource>\n",
       " <pmid version=\"1\">16328781</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Oncogene. 2006 Mar 2;25(9):1340-8</refsource>\n",
       " <pmid version=\"1\">16247451</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2006 Mar 7;103(10):3775-80</refsource>\n",
       " <pmid version=\"1\">16537454</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 2006 Mar 15;66(6):3197-204</refsource>\n",
       " <pmid version=\"1\">16540671</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2006 May 19;281(20):13957-63</refsource>\n",
       " <pmid version=\"1\">16543245</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Curr Med Chem. 2006;13(16):1845-57</refsource>\n",
       " <pmid version=\"1\">16842197</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Int J Oncol. 2006 Oct;29(4):981-7</refsource>\n",
       " <pmid version=\"1\">16964394</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Leukemia. 2006 Oct;20(10):1759-66</refsource>\n",
       " <pmid version=\"1\">16932349</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2006 Nov 14;103(46):17260-5</refsource>\n",
       " <pmid version=\"1\">17088559</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Aging Cell. 2007 Aug;6(4):489-503</refsource>\n",
       " <pmid version=\"1\">17608836</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Free Radic Biol Med. 2007 Sep 1;43(5):781-8</refsource>\n",
       " <pmid version=\"1\">17664141</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Handb Exp Pharmacol. 2008;(181):131-50</refsource>\n",
       " <pmid version=\"1\">18071944</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Annu Rev Cell Dev Biol. 1999;15:551-78</refsource>\n",
       " <pmid version=\"1\">10611972</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Oncologist. 2000;5 Suppl 1:37-44</refsource>\n",
       " <pmid version=\"1\">10804090</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Neurooncol. 2000 Oct-Nov;50(1-2):1-15</refsource>\n",
       " <pmid version=\"1\">11245270</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2001 May 24;411(6836):494-8</refsource>\n",
       " <pmid version=\"1\">11373684</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>J Biol Chem. 2003 Feb 28;278(9):6885-95</refsource>\n",
       " <pmid version=\"1\">12482858</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 2004 Mar 1;64(5):1765-72</refsource>\n",
       " <pmid version=\"1\">14996738</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1992 Oct 29;359(6398):843-5</refsource>\n",
       " <pmid version=\"1\">1279431</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1992 Oct 29;359(6398):845-8</refsource>\n",
       " <pmid version=\"1\">1279432</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Kidney Int. 1993 Aug;44(2):278-84</refsource>\n",
       " <pmid version=\"1\">8377372</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cancer Res. 1993 Dec 1;53(23):5822-7</refsource>\n",
       " <pmid version=\"1\">7694795</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Lab Invest. 1994 Sep;71(3):374-9</refsource>\n",
       " <pmid version=\"1\">7933988</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 1995 Jan 31;92(3):768-72</refsource>\n",
       " <pmid version=\"1\">7531342</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 1996 Apr 4;380(6573):439-42</refsource>\n",
       " <pmid version=\"1\">8602242</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 1998 Mar 31;95(7):3609-14</refsource>\n",
       " <pmid version=\"1\">9520413</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>EMBO J. 1998 Aug 3;17(15):4426-41</refsource>\n",
       " <pmid version=\"1\">9687510</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 1998 Dec 22;95(26):15809-14</refsource>\n",
       " <pmid version=\"1\">9861052</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>FASEB J. 1999 Jan;13(1):9-22</refsource>\n",
       " <pmid version=\"1\">9872925</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Biochem Biophys Res Commun. 1998 Dec 9;253(1):33-7</refsource>\n",
       " <pmid version=\"1\">9875215</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Pharmacol Rev. 2004 Dec;56(4):549-80</refsource>\n",
       " <pmid version=\"1\">15602010</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Cytokine Growth Factor Rev. 2005 Feb;16(1):77-89</refsource>\n",
       " <pmid version=\"1\">15733833</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Proc Natl Acad Sci U S A. 2000 Aug 29;97(18):10242-7</refsource>\n",
       " <pmid version=\"1\">10963684</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Nature. 2000 Nov 16;408(6810):325-30</refsource>\n",
       " <pmid version=\"1\">11099033</pmid>\n",
       " </commentscorrections>\n",
       " <commentscorrections reftype=\"Cites\">\n",
       " <refsource>Mol Cell Biol. 2002 Apr;22(8):2871-81</refsource>\n",
       " <pmid version=\"1\">11909979</pmid>\n",
       " </commentscorrections>\n",
       " </commentscorrectionslist>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015153\">Blotting, Western</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D045744\">Cell Line, Tumor</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D042783\">Endothelial Cells</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D004797\">Enzyme-Linked Immunosorbent Assay</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005786\">Gene Expression Regulation</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000502\">physiology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015723\">Gene Library</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D017930\">Genes, Reporter</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D006801\">Humans</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020930\">MAP Kinase Kinase Kinases</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051379\">Mice</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009369\">Neoplasms</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D009389\">Neovascularization, Pathologic</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D016133\">Polymerase Chain Reaction</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D034741\">RNA, Small Interfering</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D014162\">Transfection</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D042461\">Vascular Endothelial Growth Factor A</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000235\">genetics</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D023041\">Xenograft Model Antitumor Assays</descriptorname>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2009</year>\n",
       " <month>2</month>\n",
       " <day>28</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2009</year>\n",
       " <month>2</month>\n",
       " <day>28</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2009</year>\n",
       " <month>4</month>\n",
       " <day>29</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">19246638</articleid>\n",
       " <articleid idtype=\"pii\">S0002-9440(10)61011-5</articleid>\n",
       " <articleid idtype=\"doi\">10.2353/ajpath.2009.080190</articleid>\n",
       " <articleid idtype=\"pmc\">PMC2671385</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>, <pubmedarticle>\n",
       " <medlinecitation owner=\"NLM\" status=\"MEDLINE\">\n",
       " <pmid version=\"1\">16216424</pmid>\n",
       " <datecompleted>\n",
       " <year>2006</year>\n",
       " <month>04</month>\n",
       " <day>12</day>\n",
       " </datecompleted>\n",
       " <daterevised>\n",
       " <year>2013</year>\n",
       " <month>11</month>\n",
       " <day>21</day>\n",
       " </daterevised>\n",
       " <article pubmodel=\"Print-Electronic\">\n",
       " <journal>\n",
       " <issn issntype=\"Print\">0306-4522</issn>\n",
       " <journalissue citedmedium=\"Print\">\n",
       " <volume>136</volume>\n",
       " <issue>4</issue>\n",
       " <pubdate>\n",
       " <year>2005</year>\n",
       " </pubdate>\n",
       " </journalissue>\n",
       " <title>Neuroscience</title>\n",
       " <isoabbreviation>Neuroscience</isoabbreviation>\n",
       " </journal>\n",
       " <articletitle>Acute phase gene expression in mice exposed to the marine neurotoxin domoic acid.</articletitle>\n",
       " <pagination>\n",
       " <medlinepgn>1121-32</medlinepgn>\n",
       " </pagination>\n",
       " <abstract>\n",
       " <abstracttext>Domoic acid is a rigid analog of the neurotransmitter glutamate and a potent agonist of kainate subtype glutamate receptors. Persistent activation of these receptor subtypes results in rapid excitotoxicity, calcium dependent cell death and neuronal lesions in areas of the brain where kainate pathways are concentrated. To better understand responses to domoic acid induced excitotoxicity, microarrays were used to profile gene expression in mouse brain following domoic acid exposure. Adult female mice were subjected intraperitoneally to domoic acid at the lethal dose 50, killed and dissected at 30, 60 and 240 min post-injection. Total brain RNA from treated mice was compared with time-matched controls on Agilent 22K feature microarrays. Real-time PCR was performed on selected genes. For the 30, 60 and 240 min time points, 3.96%, 3.94% and 4.36% of the genes interrogated were differentially expressed (P-value &lt; or = 0.01), respectively. Rigorous filtering of the data resulted in a set of 56 genes used for trending analysis and K-medians and agglomerative clustering. The earliest genes induced consisted primarily of early response gene families (Jun, Fos, Ier, Egr, growth arrest and DNA damage 45) and the inflammatory response element cyclooxygenase 2. Some later responding genes involved glucocorticoid responses (Gilz, Sgk), cold inducible proteins (Cirbp, Rbm3), Map kinases (Map3k6) and NF-kappaB inhibition. Real-time PCR in male mice from an additional study confirmed the expression of several of these genes across gender. The transcriptional profile induced by domoic acid shared similarity with expression profiles of brain ischemia and other excitotoxins, suggesting a common transcriptional response.</abstracttext>\n",
       " </abstract>\n",
       " <authorlist completeyn=\"Y\">\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ryan</lastname>\n",
       " <forename>J C</forename>\n",
       " <initials>JC</initials>\n",
       " <affiliationinfo>\n",
       " <affiliation>Marine Biotoxins Program, NOAA/National Ocean Service Center for Coastal Environmental Health and Biomolecular Research, 219 Fort Johnson Road, Charleston, SC 29412, USA. james.ryan@noaa.gov</affiliation>\n",
       " </affiliationinfo>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Morey</lastname>\n",
       " <forename>J S</forename>\n",
       " <initials>JS</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Ramsdell</lastname>\n",
       " <forename>J S</forename>\n",
       " <initials>JS</initials>\n",
       " </author>\n",
       " <author validyn=\"Y\">\n",
       " <lastname>Van Dolah</lastname>\n",
       " <forename>F M</forename>\n",
       " <initials>FM</initials>\n",
       " </author>\n",
       " </authorlist>\n",
       " <language>eng</language>\n",
       " <publicationtypelist>\n",
       " <publicationtype ui=\"D003160\">Comparative Study</publicationtype>\n",
       " <publicationtype ui=\"D016428\">Journal Article</publicationtype>\n",
       " </publicationtypelist>\n",
       " <articledate datetype=\"Electronic\">\n",
       " <year>2005</year>\n",
       " <month>10</month>\n",
       " <day>10</day>\n",
       " </articledate>\n",
       " </article>\n",
       " <medlinejournalinfo>\n",
       " <country>United States</country>\n",
       " <medlineta>Neuroscience</medlineta>\n",
       " <nlmuniqueid>7605074</nlmuniqueid>\n",
       " <issnlinking>0306-4522</issnlinking>\n",
       " </medlinejournalinfo>\n",
       " <chemicallist>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D008387\">Marine Toxins</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>0</registrynumber>\n",
       " <nameofsubstance ui=\"D012333\">RNA, Messenger</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>M02525818H</registrynumber>\n",
       " <nameofsubstance ui=\"C012301\">domoic acid</nameofsubstance>\n",
       " </chemical>\n",
       " <chemical>\n",
       " <registrynumber>SIV03811UC</registrynumber>\n",
       " <nameofsubstance ui=\"D007608\">Kainic Acid</nameofsubstance>\n",
       " </chemical>\n",
       " </chemicallist>\n",
       " <citationsubset>IM</citationsubset>\n",
       " <meshheadinglist>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000210\">Acute-Phase Reaction</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D000818\">Animals</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D005260\">Female</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D015870\">Gene Expression</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000187\">drug effects</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D007608\">Kainic Acid</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000031\">analogs &amp; derivatives</qualifiername>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008297\">Male</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D008387\">Marine Toxins</descriptorname>\n",
       " <qualifiername majortopicyn=\"Y\" ui=\"Q000494\">pharmacology</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D051379\">Mice</descriptorname>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D046228\">Microarray Analysis</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D012333\">RNA, Messenger</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000378\">metabolism</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D020133\">Reverse Transcriptase Polymerase Chain Reaction</descriptorname>\n",
       " <qualifiername majortopicyn=\"N\" ui=\"Q000379\">methods</qualifiername>\n",
       " </meshheading>\n",
       " <meshheading>\n",
       " <descriptorname majortopicyn=\"N\" ui=\"D013997\">Time Factors</descriptorname>\n",
       " </meshheading>\n",
       " </meshheadinglist>\n",
       " </medlinecitation>\n",
       " <pubmeddata>\n",
       " <history>\n",
       " <pubmedpubdate pubstatus=\"received\">\n",
       " <year>2005</year>\n",
       " <month>04</month>\n",
       " <day>25</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"revised\">\n",
       " <year>2005</year>\n",
       " <month>07</month>\n",
       " <day>11</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"accepted\">\n",
       " <year>2005</year>\n",
       " <month>08</month>\n",
       " <day>15</day>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"pubmed\">\n",
       " <year>2005</year>\n",
       " <month>10</month>\n",
       " <day>12</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"medline\">\n",
       " <year>2006</year>\n",
       " <month>4</month>\n",
       " <day>13</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " <pubmedpubdate pubstatus=\"entrez\">\n",
       " <year>2005</year>\n",
       " <month>10</month>\n",
       " <day>12</day>\n",
       " <hour>9</hour>\n",
       " <minute>0</minute>\n",
       " </pubmedpubdate>\n",
       " </history>\n",
       " <publicationstatus>ppublish</publicationstatus>\n",
       " <articleidlist>\n",
       " <articleid idtype=\"pubmed\">16216424</articleid>\n",
       " <articleid idtype=\"pii\">S0306-4522(05)00926-7</articleid>\n",
       " <articleid idtype=\"doi\">10.1016/j.neuroscience.2005.08.047</articleid>\n",
       " </articleidlist>\n",
       " </pubmeddata>\n",
       " </pubmedarticle>]"
      ]
     },
     "execution_count": 246,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soups_MAP3K6soup #raw xml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "MAP3K6_docstrings=[] #cleaning up the text! \n",
    "for soup in soups_MAP3K6soup: #for each soup, get the text you want. \n",
    "    articleTitle=soup.articletitle.getText()\n",
    "    abstract_text_list=soup.find_all('abstracttext')\n",
    "    string=\"\"\n",
    "    for abstract_text in abstract_text_list: #looks to see if more than one abstract tag\n",
    "        string=string + \" \" + abstract_text.getText()\n",
    "    MAP3K6_docstrings.append(\"TITLE: \" + articleTitle + \" ABSTRACT:\" + \"\".join(string))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['TITLE: Role of germline aberrations affecting CTNNA1, MAP3K6 and MYD88 in gastric cancer susceptibility. ABSTRACT: In approximately 10% of all gastric cancer (GC) cases, a heritable cause is suspected. A subset of these cases have a causative germline CDH1 mutation; however, in most cases the cause remains unknown. Our objective was to assess to what extent these remaining cases may be explained by germline mutations in the novel candidate GC predisposing genes CTNNA1, MAP3K6 or MYD88. We sequenced a large cohort of unexplained young and/or familial patients with GC (n=286) without a CDH1germline mutation for germline variants affecting CTNNA1, MAP3K6 and MYD88 using a targeted next-generation sequencing approach based on single-molecule molecular inversion probes. Predicted deleterious germline variants were not encountered in MYD88, but recurrently observed in CTNNA1 (n=2) and MAP3K6 (n=3) in our cohort of patients with GC. In contrast to deleterious variants in CTNNA1, deleterious variants in MAP3K6 also occur frequently in the general population. Based on our results MAP3K6 should no longer be considered a GC predisposition gene, whereas deleterious CTNNA1 variants are confirmed as an infrequent cause of GC susceptibility. Biallelic MYD88 germline mutations are at most a very rare cause of GC susceptibility as no additional cases were identified.',\n",
       " 'TITLE: Cyclin-dependent kinase 5 regulates MAPK/ERK signaling in the skin of mice. ABSTRACT: Cyclin-dependent kinase 5 (CDK5) is a proline-directed serine/threonine kinase that has been shown to play important roles in many tissues except the nervous system. We previously reported that CDK5 showed differential expression in the transcriptome profiles of the skin of alpacas with different hair colors. To understand the functional role of CDK5 in hair color determination, we constructed CDK5-knockdown mice and identified the effect on the mitogen-activated protein kinase (MAPK) pathway in the mouse skin. Quantitative real-time polymerase chain reaction, co-immunoprecipitation, and western blotting were performed to analyze the effects of CDK5-knockdown on the MAPK pathway in mice. The results showed that MAP3K6 was inhibited by phosphorylated CDK5 through its activator CDK7. The decrease in MAP3K6 levels caused down-regulation of MEK1 and ERK expression, leading to the up-regulation of miR-143-3p, which targets MAP3K6 via Dicer. Taken together, our findings indicate that CDK5 functions in regulating the MAPK pathway. Given that MAP3K6 was inhibited in two directions, this mechanism can provide insight into the contributions of the MAPK/ERK pathway to the inhibition of melanin production.',\n",
       " 'TITLE: Genetic predisposition to gastric cancer. ABSTRACT: Gastric cancer ranks as the third leading cause of cancer mortality worldwide and confers a 5-year survival of 20%. While most gastric cancers are sporadic, ~1%-3% can be attributed to inherited cancer predisposition syndromes. Germline E-cadherin/CDH1 mutations have been identified in families with an autosomal dominant inherited predisposition to diffuse gastric cancer. The cumulative risk of gastric cancer for CDH1 mutation carriers by age 80 years is reportedly 70% for men and 56% for women. Female mutation carriers also have an estimated 42% risk for developing lobular breast cancer by age 80 years. However, most individuals meeting clinical criteria for hereditary diffuse gastric cancer syndrome (HDGC) do not have a germline CDH1 mutation, and germline CDH1 mutation carriers do not all exhibit similar clinical outcomes in terms of age of diagnosis or cancer types. E-cadherin (CDH1) as the one known causative gene for HDGC accounts for only 40% of cases, leaving 60% with an unknown genetic diagnosis. In addition to HDGC, we will review other genetic syndromes with elevated gastric cancer risk, as well as newly implicated alterations in other genes (CTNNA1, DOT1L, FBXO24, PRSS1, MAP3K6, MSR1, and INSR) that may affect gastric cancer susceptibility and age-specific penetrance.',\n",
       " 'TITLE: Hereditary diffuse gastric cancer: What the clinician should know. ABSTRACT: Hereditary diffuse gastric cancer (HDGC) is an inherited autosomal dominant syndrome with a penetrance of up to 80% affecting diverse geographic populations. While it has been shown to be caused mainly by germline alterations in the E-cadherin gene (CDH1), problematically, the genetic diagnosis remains unknown in up to 60% of patients. Given the important knowledge gaps regarding the syndrome, asymptomatic carriers of CDH1 mutations are advised for a prophylactic total gastrectomy. Intensive annual endoscopic surveillance is the alternative for carriers who decline gastrectomy. As HDGCs have a prolonged indolent phase, this provides a window of opportunity for surveillance and treatment. Recent findings of other gene defects in CTNNA1 and MAP3K6, as well as further characterization of CDH1 mutations and their pathogenicity will change the way HDGC patients are counselled for screening, surveillance and treatment. This review will bring the reader up to date with these changes and discuss future directions for research; namely more accurate risk stratification and surveillance methods to improve clinical care of HDGC patients. ',\n",
       " 'TITLE: Recognition of and recent issues in hereditary diffuse gastric cancer. ABSTRACT: In East Asian countries, gastric cancer incidence is high, but detection rates for germline CDH1 mutations that cause hereditary diffuse gastric cancers (HDGCs) are low. Consequently, screens and genetic testing for HDGC are often considered unimportant. Since the first germline truncating CDH1 mutations in Japanese patients were reported, some HDGC cases have been reported, and some of these involve large germline rearrangements and de novo mutation of CDH1. New methods for mutation detection--such as multiplex ligation-dependent probe amplification, array comparative genomic hybridization, and exome sequencing--have become available, as have new experimental models, including novel gene-knockout mice and gastric organoids. Because of these advances, searches for candidate genes (e.g., CTNNA1, MAP3K6) and our understanding of HDGC pathogenesis have improved in recent years; moreover, there have been substantial changes in the field since the current HDGC consensus guidelines were released. This review focuses on recent issues and advances in the study of HDGC. For example, lobular breast cancer cases and de novo occurrences of DGC are unlikely to meet the existing criteria for genetic testing, but current evidence indicates that some such cases may be good candidates for genetic testing. It is important to recognize that HDGC is a syndrome and that lobular breast cancer can be the first manifestation of this syndrome. CDH1 testing, including analyses of large genomic rearrangements, should be recommended even in countries where few HDGC cases have been reported.',\n",
       " 'TITLE: A rapid and high content assay that measures cyto-ID-stained autophagic compartments and estimates autophagy flux with potential clinical applications. ABSTRACT: The lack of a rapid and quantitative autophagy assay has substantially hindered the development and implementation of autophagy-targeting therapies for a variety of human diseases. To address this critical issue, we developed a novel autophagy assay using the newly developed Cyto-ID fluorescence dye. We first verified that the Cyto-ID dye specifically labels autophagic compartments with minimal staining of lysosomes and endosomes. We then developed a new Cyto-ID fluorescence spectrophotometric assay that makes it possible to estimate autophagy flux based on measurements of the Cyto-ID-stained autophagic compartments. By comparing to traditional autophagy approaches, we found that this assay yielded a more sensitive, yet less variable, quantification of the stained autophagic compartments and the estimate of autophagy flux. Furthermore, we tested the potential application of this autophagy assay in high throughput research by integrating it into an RNA interference (RNAi) screen and a small molecule screen. The RNAi screen revealed WNK2 and MAP3K6 as autophagy-modulating genes, both of which inhibited the MTOR pathway. Similarly, the small molecule screen identified sanguinarine and actinomycin D as potent autophagy inducers in leukemic cells. Moreover, we successfully detected autophagy responses to kinase inhibitors and chloroquine in normal or leukemic mice using this assay. Collectively, this new Cyto-ID fluorescence spectrophotometric assay provides a rapid, reliable quantification of autophagic compartments and estimation of autophagy flux with potential applications in developing autophagy-related therapies and as a test to monitor autophagy responses in patients being treated with autophagy-modulating drugs. ',\n",
       " 'TITLE: Germline mutations in MAP3K6 are associated with familial gastric cancer. ABSTRACT: Gastric cancer is among the leading causes of cancer-related deaths worldwide. While heritable forms of gastric cancer are relatively rare, identifying the genes responsible for such cases can inform diagnosis and treatment for both hereditary and sporadic cases of gastric cancer. Mutations in the E-cadherin gene, CDH1, account for 40% of the most common form of familial gastric cancer (FGC), hereditary diffuse gastric cancer (HDGC). The genes responsible for the remaining forms of FGC are currently unknown. Here we examined a large family from Maritime Canada with FGC without CDH1 mutations, and identified a germline coding variant (p.P946L) in mitogen-activated protein kinase kinase kinase 6 (MAP3K6). Based on conservation, predicted pathogenicity and a known role of the gene in cancer predisposition, MAP3K6 was considered a strong candidate and was investigated further. Screening of an additional 115 unrelated individuals with non-CDH1 FGC identified the p.P946L MAP3K6 variant, as well as four additional coding variants in MAP3K6 (p.F849Sfs*142, p.P958T, p.D200Y and p.V207G). A somatic second-hit variant (p.H506Y) was present in DNA obtained from one of the tumor specimens, and evidence of DNA hypermethylation within the MAP3K6 gene was observed in DNA from the tumor of another affected individual. These findings, together with previous evidence from mouse models that MAP3K6 acts as a tumor suppressor, and studies showing the presence of somatic mutations in MAP3K6 in non-hereditary gastric cancers and gastric cancer cell lines, point towards MAP3K6 variants as a predisposing factor for FGC. ',\n",
       " 'TITLE: Salt-responsive ERF1 regulates reactive oxygen species-dependent signaling during the initial response to salt stress in rice. ABSTRACT: Early detection of salt stress is vital for plant survival and growth. Still, the molecular processes controlling early salt stress perception and signaling are not fully understood. Here, we identified salt-responsive ERF1 (SERF1), a rice (Oryza sativa) transcription factor (TF) gene that shows a root-specific induction upon salt and hydrogen peroxide (H2O2) treatment. Loss of SERF1 impairs the salt-inducible expression of genes encoding members of a mitogen-activated protein kinase (MAPK) cascade and salt tolerance-mediating TFs. Furthermore, we show that SERF1-dependent genes are H2O2 responsive and demonstrate that SERF1 binds to the promoters of MAPK kinase kinase6 (MAP3K6), MAPK5, dehydration-responsive element bindinG2A (DREB2A), and zinc finger protein179 (ZFP179) in vitro and in vivo. SERF1 also directly induces its own gene expression. In addition, SERF1 is a phosphorylation target of MAPK5, resulting in enhanced transcriptional activity of SERF1 toward its direct target genes. In agreement, plants deficient for SERF1 are more sensitive to salt stress compared with the wild type, while constitutive overexpression of SERF1 improves salinity tolerance. We propose that SERF1 amplifies the reactive oxygen species-activated MAPK cascade signal during the initial phase of salt stress and translates the salt-induced signal into an appropriate expressional response resulting in salt tolerance.',\n",
       " 'TITLE: Genetic and structural variation in the gastric cancer kinome revealed through targeted deep sequencing. ABSTRACT: Genetic alterations in kinases have been linked to multiple human pathologies. To explore the landscape of kinase genetic variation in gastric cancer (GC), we used targeted, paired-end deep sequencing to analyze 532 protein and phosphoinositide kinases in 14 GC cell lines. We identified 10,604 single-nucleotide variants (SNV) in kinase exons including greater than 300 novel nonsynonymous SNVs. Family-wise analysis of the nonsynonymous SNVs revealed a significant enrichment in mitogen-activated protein kinase (MAPK)-related genes (P < 0.01), suggesting a preferential involvement of this kinase family in GC. A potential antioncogenic role for MAP2K4, a gene exhibiting recurrent alterations in 2 lines, was functionally supported by siRNA knockdown and overexpression studies in wild-type and MAP2K4 variant lines. The deep sequencing data also revealed novel, large-scale structural rearrangement events involving kinases including gene fusions involving CDK12 and the ERBB2 receptor tyrosine kinase in MKN7 cells. Integrating SNVs and copy number alterations, we identified Hs746T as a cell line exhibiting both splice-site mutations and genomic amplification of MET, resulting in MET protein overexpression. When applied to primary GCs, we identified somatic mutations in 8 kinases, 4 of which were recurrently altered in both primary tumors and cell lines (MAP3K6, STK31, FER, and CDKL5). These results demonstrate that how targeted deep sequencing approaches can deliver unprecedented multilevel characterization of a medically and pharmacologically relevant gene family. The catalog of kinome genetic variants assembled here may broaden our knowledge on kinases and provide useful information on genetic alterations in GC.',\n",
       " 'TITLE: Global gene expression profiles of MT knockout and wild-type mice in the condition of doxorubicin-induced cardiomyopathy. ABSTRACT: Increasing evidence from in vivo and in vitro studies has indicated that MT exerts protective effects against DOX-induced cardiotoxicity; however the underlying precise mechanisms still remain an enigma. Therefore, the present study was designed using MT knockout mice in concert with genomic approaches to explore the possible molecular and cellular mechanisms in terms of the genetic network changes. MT-I/II null (MT⁻/⁻) mice and corresponding wild-type mice (MT+/+) were administrated with a single dose of DOX (15 mg/kg, i.p.) or equal volume of saline. Animals were sacrificed on the 4th day after DOX administration and samples were collected for further analyses. Global gene expression profiles of cardiac mRNA from two genotype mice revealed that 381 characteristically MT-responsive genes were identified between MT+/+ mice and MT⁻/⁻ mice in response to DOX, including fos, ucp3, car3, atf3, map3k6, etc. Functional analysis implied MAPK signaling pathway, p53 signaling pathway, Jak-STAT signaling pathway, PPAR signaling pathway, Wnt signaling pathway, etc. might be involved to mediate the protection of DOX cardiomyopathy by MT. Results from the present study not only validated the previously reported possible mechanisms of MT protection against DOX toxicity, but also provided new clues into the molecular mechanisms involved in this process.',\n",
       " 'TITLE: Impact of nose-only exposure system on pulmonary gene expression. ABSTRACT: Nose-only exposure is used to study the distribution and toxicity of airborne contaminants. Restraint of animals in nose-only tubes causes stress, but the impact on pulmonary mRNA levels is unknown. Since stress and xenobiotics activate common pathways, we assessed whether nose-only exposure would alter expression of toxicologically relevant genes in the lungs. To identify candidate genes for further analysis, we first interrogated microarray data to examine time-dependent changes in gene expression in air-control animals from a nose-only inhalation study involving male wild-type C57BL/6 mice and transgenic tumor necrosis factor (TNF)-alpha over-expressing littermates. Comparison of transcript levels immediately and 24 h after a single 4-h nose-only exposure to air revealed differential expression of 280 genes (false discovery rate-adjusted, p < .05). Functional analysis revealed enrichment of immune response, apoptosis, and signalling terms, consistent with effects of restraint stress. We then selected a subset of target genes for comparison of naive animals and air-exposed animals from the inhalation study by real-time polymerase chain reaction (PCR). Expression of genes involved in stress (BNIP, sestrin-1, CDKN1A [p21], GADD45 gamma), glucocorticoid-response (GILZ, Sgk), and signal transduction (MAP3K6, C/EBP-delta) was increased as a result of nose-only exposure (p < .05). In contrast, proinflammatory factors (lymphotoxin-beta, chemokine receptor CXCR5) were decreased (p < .05). Immune gene responses observed in wild-type animals were reduced in animals with lung inflammation, indicating that pathological states can modify the response to nose-only exposure. Observed responses may warrant consideration in the evaluation of materials delivered by nose-only inhalation, and suggest that incorporation of naive animals into nose-only studies should be considered as a best practice.',\n",
       " 'TITLE: Mitogen-activated protein 3 kinase 6 mediates angiogenic and tumorigenic effects via vascular endothelial growth factor expression. ABSTRACT: Genome-wide screening using a small interfering RNA (siRNA) library has revealed novel molecules that are involved in a wide range of physiological responses. The expression of vascular endothelial growth factor (VEGF) is increased under hypoxic conditions, and plays a crucial role in tumor angiogenesis and tissue responses to ischemia. Here, we used a siRNA expression vector library to elucidate molecules that modify VEGF expression. Screening using an siRNA library revealed that MAPKKK6 (MEKK6/MAP3K6) regulates VEGF expression under both normoxic and hypoxic conditions in vitro, although the biological function of MAP3K6 remains unknown. Attenuation of VEGF expression by MAP3K6 inhibition was demonstrated by transient transfection of double-stranded RNA as well as by stable transfection of short hairpin RNA-expressing vectors against MAP3K6. Conditioned medium of MAP3K6-knocked down cells attenuated both endothelial proliferation and capillary network formation in a VEGF-dependent manner in vitro. In addition, tumor cells with down-regulation of MAP3K6 expression showed significant suppression of tumor growth in vivo, which was accompanied by significant repression of vessel formation and VEGF expression in these tumors. The results of this study suggest that MAP3K6 regulates VEGF expression in both normoxia and hypoxia, and that regulation of VEGF by MAP3K6 may play a crucial role in both angiogenesis and tumorigenesis.',\n",
       " 'TITLE: Acute phase gene expression in mice exposed to the marine neurotoxin domoic acid. ABSTRACT: Domoic acid is a rigid analog of the neurotransmitter glutamate and a potent agonist of kainate subtype glutamate receptors. Persistent activation of these receptor subtypes results in rapid excitotoxicity, calcium dependent cell death and neuronal lesions in areas of the brain where kainate pathways are concentrated. To better understand responses to domoic acid induced excitotoxicity, microarrays were used to profile gene expression in mouse brain following domoic acid exposure. Adult female mice were subjected intraperitoneally to domoic acid at the lethal dose 50, killed and dissected at 30, 60 and 240 min post-injection. Total brain RNA from treated mice was compared with time-matched controls on Agilent 22K feature microarrays. Real-time PCR was performed on selected genes. For the 30, 60 and 240 min time points, 3.96%, 3.94% and 4.36% of the genes interrogated were differentially expressed (P-value < or = 0.01), respectively. Rigorous filtering of the data resulted in a set of 56 genes used for trending analysis and K-medians and agglomerative clustering. The earliest genes induced consisted primarily of early response gene families (Jun, Fos, Ier, Egr, growth arrest and DNA damage 45) and the inflammatory response element cyclooxygenase 2. Some later responding genes involved glucocorticoid responses (Gilz, Sgk), cold inducible proteins (Cirbp, Rbm3), Map kinases (Map3k6) and NF-kappaB inhibition. Real-time PCR in male mice from an additional study confirmed the expression of several of these genes across gender. The transcriptional profile induced by domoic acid shared similarity with expression profiles of brain ischemia and other excitotoxins, suggesting a common transcriptional response.']"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "MAP3K6_docstrings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "#getting docstrings for everything else\n",
    "#create a list of soups and a list of docstrings and append them... \n",
    "MAP3K2_docstrings=[]\n",
    "MAP3K3_docstrings=[]\n",
    "MAP3K5_docstrings=[]\n",
    "MAP3K6_docstrings=[]\n",
    "MAP3K15_docstrings=[]\n",
    "docstring_list=[MAP3K2_docstrings,MAP3K3_docstrings,MAP3K5_docstrings,MAP3K6_docstrings,MAP3K15_docstrings]\n",
    "soupsList=[soups_MAP3K2soup, soups_MAP3K3soup, soups_MAP3K5soup, soups_MAP3K6soup, soups_MAP3K15soup]\n",
    "\n",
    "#iterate through souplist and append each docstring of the docstring list\n",
    "\n",
    "for soups,docstrings in zip(soupsList,docstring_list): #goes through each map3k soup\n",
    "    for soup in soups: #goes through each soup in the map3ksoup\n",
    "        articleTitle = soup.articletitle.getText()\n",
    "        abstract_text_list=soup.find_all('abstracttext')\n",
    "        string=\"\"\n",
    "        for abstract_text in abstract_text_list:\n",
    "            string=string + \" \" + abstract_text.getText()\n",
    "        docstrings.append(\"TITLE: \" + articleTitle + \" ABSTRACT:\" + \"\".join(string))\n",
    "        \n",
    "        \n",
    "        \n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'titl role germlin aberr affect ctnna1 map3k6 myd88 gastric cancer suscept abstract approxim gastric cancer gc case herit caus suspect subset case causat germlin cdh1 mutat howev case caus remain unknown object assess extent remain case may explain germlin mutat novel candid gc predispos gene ctnna1 map3k6 myd88 sequenc larg cohort unexplain young famili patient gc without cdh1germlin mutat germlin variant affect ctnna1 map3k6 myd88 use target next generat sequenc approach base singl molecul molecular invers probe predict deleteri germlin variant encount myd88 recurr observ ctnna1 map3k6 cohort patient gc contrast deleteri variant ctnna1 deleteri variant map3k6 occur frequent general popul base result map3k6 longer consid gc predisposit gene wherea deleteri ctnna1 variant confirm infrequ caus gc suscept biallel myd88 germlin mutat rare caus gc suscept addit case identifi'"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from nltk.tokenize import RegexpTokenizer\n",
    "import nltk\n",
    "import string\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "#processing a single article, (first article of MAP3K6 docstrings)\n",
    "\n",
    "#let's tokenizes and get rid of punctuation\n",
    "tokenized_article =nltk.tokenize.regexp_tokenize(MAP3K6_docstrings[0],'\\w{2,}') \n",
    "\n",
    "#let's get rid of stopwords and make everything lower case. and get rid of numbers too. \n",
    "sw = stopwords.words()\n",
    "tokenized_article = [word.lower() for word in tokenized_article if word.lower() not in sw and word[0] in string.ascii_letters]\n",
    "\n",
    "#finally, let's stem each word\n",
    "sb = nltk.stem.SnowballStemmer('english')\n",
    "stemmed_tokenized_article= [sb.stem(word) for word in tokenized_article]\n",
    "\n",
    "#join the stemmed words\n",
    "\" \".join(stemmed_tokenized_article) \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'TITLE: Role of germline aberrations affecting CTNNA1, MAP3K6 and MYD88 in gastric cancer susceptibility. ABSTRACT: In approximately 10% of all gastric cancer (GC) cases, a heritable cause is suspected. A subset of these cases have a causative germline CDH1 mutation; however, in most cases the cause remains unknown. Our objective was to assess to what extent these remaining cases may be explained by germline mutations in the novel candidate GC predisposing genes CTNNA1, MAP3K6 or MYD88. We sequenced a large cohort of unexplained young and/or familial patients with GC (n=286) without a CDH1germline mutation for germline variants affecting CTNNA1, MAP3K6 and MYD88 using a targeted next-generation sequencing approach based on single-molecule molecular inversion probes. Predicted deleterious germline variants were not encountered in MYD88, but recurrently observed in CTNNA1 (n=2) and MAP3K6 (n=3) in our cohort of patients with GC. In contrast to deleterious variants in CTNNA1, deleterious variants in MAP3K6 also occur frequently in the general population. Based on our results MAP3K6 should no longer be considered a GC predisposition gene, whereas deleterious CTNNA1 variants are confirmed as an infrequent cause of GC susceptibility. Biallelic MYD88 germline mutations are at most a very rare cause of GC susceptibility as no additional cases were identified.'"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "MAP3K6_docstrings[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['titl role germlin aberr affect ctnna1 map3k6 myd88 gastric cancer suscept abstract approxim gastric cancer gc case herit caus suspect subset case causat germlin cdh1 mutat howev case caus remain unknown object assess extent remain case may explain germlin mutat novel candid gc predispos gene ctnna1 map3k6 myd88 sequenc larg cohort unexplain young famili patient gc without cdh1germlin mutat germlin variant affect ctnna1 map3k6 myd88 use target next generat sequenc approach base singl molecul molecular invers probe predict deleteri germlin variant encount myd88 recurr observ ctnna1 map3k6 cohort patient gc contrast deleteri variant ctnna1 deleteri variant map3k6 occur frequent general popul base result map3k6 longer consid gc predisposit gene wherea deleteri ctnna1 variant confirm infrequ caus gc suscept biallel myd88 germlin mutat rare caus gc suscept addit case identifi',\n",
       " 'titl cyclin depend kinas regul mapk erk signal skin mice abstract cyclin depend kinas cdk5 prolin direct serin threonin kinas shown play import role mani tissu except nervous system previous report cdk5 show differenti express transcriptom profil skin alpaca differ hair color understand function role cdk5 hair color determin construct cdk5 knockdown mice identifi effect mitogen activ protein kinas mapk pathway mous skin quantit real time polymeras chain reaction co immunoprecipit western blot perform analyz effect cdk5 knockdown mapk pathway mice result show map3k6 inhibit phosphoryl cdk5 activ cdk7 decreas map3k6 level caus regul mek1 erk express lead regul target map3k6 via dicer taken togeth find indic cdk5 function regul mapk pathway given map3k6 inhibit two direct mechan provid insight contribut mapk erk pathway inhibit melanin product',\n",
       " 'titl genet predisposit gastric cancer abstract gastric cancer rank third lead caus cancer mortal worldwid confer year surviv gastric cancer sporad attribut inherit cancer predisposit syndrom germlin cadherin cdh1 mutat identifi famili autosom domin inherit predisposit diffus gastric cancer cumul risk gastric cancer cdh1 mutat carrier age year report women femal mutat carrier estim risk develop lobular breast cancer age year howev individu meet clinic criteria hereditari diffus gastric cancer syndrom hdgc germlin cdh1 mutat germlin cdh1 mutat carrier exhibit similar clinic outcom term age diagnosi cancer type cadherin cdh1 one known causat gene hdgc account case leav unknown genet diagnosi addit hdgc review genet syndrom elev gastric cancer risk well newli implic alter gene ctnna1 dot1l fbxo24 prss1 map3k6 msr1 insr may affect gastric cancer suscept age specif penetr',\n",
       " 'titl hereditari diffus gastric cancer clinician know abstract hereditari diffus gastric cancer hdgc inherit autosom domin syndrom penetr affect divers geograph popul shown caus main germlin alter cadherin gene cdh1 problemat genet diagnosi remain unknown patient given import knowledg gap regard syndrom asymptomat carrier cdh1 mutat advis prophylact total gastrectomi intens annual endoscop surveil altern carrier declin gastrectomi hdgcs prolong indol phase provid window opportun surveil treatment recent find gene defect ctnna1 map3k6 well character cdh1 mutat pathogen chang way hdgc patient counsel screen surveil treatment review bring reader date chang discuss futur direct research name accur risk stratif surveil method improv clinic hdgc patient',\n",
       " 'titl recognit recent issu hereditari diffus gastric cancer abstract east asian countri gastric cancer incid high detect rate germlin cdh1 mutat caus hereditari diffus gastric cancer hdgcs low consequ screen genet test hdgc often consid unimport sinc first germlin truncat cdh1 mutat japanes patient report hdgc case report involv larg germlin rearrang novo mutat cdh1 new method mutat detect multiplex ligat depend probe amplif array compar genom hybrid exom sequenc becom avail new experiment model includ novel gene knockout mice gastric organoid advanc search candid gene ctnna1 map3k6 understand hdgc pathogenesi improv recent year moreov substanti chang field sinc current hdgc consensus guidelin releas review focus recent issu advanc studi hdgc exampl lobular breast cancer case novo occurr dgc unlik meet exist criteria genet test current evid indic case may good candid genet test import recogn hdgc syndrom lobular breast cancer first manifest syndrom cdh1 test includ analys larg genom rearrang recommend even countri hdgc case report',\n",
       " 'titl rapid high content assay measur cyto id stain autophag compart estim autophagi flux potenti clinic applic abstract lack rapid quantit autophagi assay substanti hinder develop implement autophagi target therapi varieti human diseas address critic issu develop novel autophagi assay use newli develop cyto id fluoresc dye first verifi cyto id dye specif label autophag compart minim stain lysosom endosom develop new cyto id fluoresc spectrophotometr assay make possibl estim autophagi flux base measur cyto id stain autophag compart compar tradit autophagi approach found assay yield sensit yet less variabl quantif stain autophag compart estim autophagi flux furthermor test potenti applic autophagi assay high throughput research integr rna interfer rnai screen small molecul screen rnai screen reveal wnk2 map3k6 autophagi modul gene inhibit mtor pathway similar small molecul screen identifi sanguinarin actinomycin potent autophagi induc leukem cell moreov success detect autophagi respons kinas inhibitor chloroquin normal leukem mice use assay collect new cyto id fluoresc spectrophotometr assay provid rapid reliabl quantif autophag compart estim autophagi flux potenti applic develop autophagi relat therapi test monitor autophagi respons patient treat autophagi modul drug',\n",
       " 'titl germlin mutat map3k6 associ famili gastric cancer abstract gastric cancer among lead caus cancer relat death worldwid herit form gastric cancer relat rare identifi gene respons case inform diagnosi treatment hereditari sporad case gastric cancer mutat cadherin gene cdh1 account common form famili gastric cancer fgc hereditari diffus gastric cancer hdgc gene respons remain form fgc current unknown examin larg famili maritim canada fgc without cdh1 mutat identifi germlin code variant p946l mitogen activ protein kinas kinas kinas map3k6 base conserv predict pathogen known role gene cancer predisposit map3k6 consid strong candid investig screen addit unrel individu cdh1 fgc identifi p946l map3k6 variant well four addit code variant map3k6 f849sfs p958t d200i v207g somat second hit variant h506i present dna obtain one tumor specimen evid dna hypermethyl within map3k6 gene observ dna tumor anoth affect individu find togeth previous evid mous model map3k6 act tumor suppressor studi show presenc somat mutat map3k6 hereditari gastric cancer gastric cancer cell line point toward map3k6 variant predispos factor fgc',\n",
       " 'titl salt respons erf1 regul reactiv oxygen speci depend signal initi respons salt stress rice abstract earli detect salt stress vital plant surviv growth still molecular process control earli salt stress percept signal fulli understood identifi salt respons erf1 serf1 rice oryza sativa transcript factor tf gene show root specif induct upon salt hydrogen peroxid h2o2 treatment loss serf1 impair salt induc express gene encod member mitogen activ protein kinas mapk cascad salt toler mediat tfs furthermor show serf1 depend gene h2o2 respons demonstr serf1 bind promot mapk kinas kinase6 map3k6 mapk5 dehydr respons element binding2a dreb2a zinc finger protein179 zfp179 vitro vivo serf1 direct induc gene express addit serf1 phosphoryl target mapk5 result enhanc transcript activ serf1 toward direct target gene agreement plant defici serf1 sensit salt stress compar wild type constitut overexpress serf1 improv salin toler propos serf1 amplifi reactiv oxygen speci activ mapk cascad signal initi phase salt stress translat salt induc signal appropri expression respons result salt toler',\n",
       " 'titl genet structur variat gastric cancer kinom reveal target deep sequenc abstract genet alter kinas link multipl human patholog explor landscap kinas genet variat gastric cancer gc use target pair deep sequenc analyz protein phosphoinositid kinas gc cell line identifi singl nucleotid variant snv kinas exon includ greater novel nonsynonym snvs famili wise analysi nonsynonym snvs reveal signific enrich mitogen activ protein kinas mapk relat gene suggest preferenti involv kinas famili gc potenti antioncogen role map2k4 gene exhibit recurr alter line function support sirna knockdown overexpress studi wild type map2k4 variant line deep sequenc data reveal novel larg scale structur rearrang event involv kinas includ gene fusion involv cdk12 erbb2 receptor tyrosin kinas mkn7 cell integr snvs copi number alter identifi hs746t cell line exhibit splice site mutat genom amplif result protein overexpress appli primari gcs identifi somat mutat kinas recurr alter primari tumor cell line map3k6 stk31 fer cdkl5 result demonstr target deep sequenc approach deliv unpreced multilevel character medic pharmacolog relev gene famili catalog kinom genet variant assembl may broaden knowledg kinas provid use inform genet alter gc',\n",
       " 'titl global gene express profil mt knockout wild type mice condit doxorubicin induc cardiomyopathi abstract increas evid vivo vitro studi indic mt exert protect effect dox induc cardiotox howev under precis mechan still remain enigma therefor present studi design use mt knockout mice concert genom approach explor possibl molecular cellular mechan term genet network chang mt null mt mice correspond wild type mice mt administr singl dose dox mg kg equal volum salin anim sacrif day dox administr sampl collect analys global gene express profil cardiac mrna two genotyp mice reveal characterist mt respons gene identifi mt mice mt mice respons dox includ fos ucp3 car3 atf3 map3k6 etc function analysi impli mapk signal pathway p53 signal pathway jak stat signal pathway ppar signal pathway wnt signal pathway etc might involv mediat protect dox cardiomyopathi mt result present studi valid previous report possibl mechan mt protect dox toxic provid new clue molecular mechan involv process',\n",
       " 'titl impact nose exposur system pulmonari gene express abstract nose exposur use studi distribut toxic airborn contamin restraint anim nose tube caus stress impact pulmonari mrna level unknown sinc stress xenobiot activ common pathway assess whether nose exposur would alter express toxicolog relev gene lung identifi candid gene analysi first interrog microarray data examin time depend chang gene express air control anim nose inhal studi involv male wild type c57bl mice transgen tumor necrosi factor tnf alpha express litterm comparison transcript level immedi singl nose exposur air reveal differenti express gene fals discoveri rate adjust function analysi reveal enrich immun respons apoptosi signal term consist effect restraint stress select subset target gene comparison naiv anim air expos anim inhal studi real time polymeras chain reaction pcr express gene involv stress bnip sestrin cdkn1a p21 gadd45 gamma glucocorticoid respons gilz sgk signal transduct map3k6 ebp delta increas result nose exposur contrast proinflammatori factor lymphotoxin beta chemokin receptor cxcr5 decreas immun gene respons observ wild type anim reduc anim lung inflamm indic patholog state modifi respons nose exposur observ respons may warrant consider evalu materi deliv nose inhal suggest incorpor naiv anim nose studi consid best practic',\n",
       " 'titl mitogen activ protein kinas mediat angiogen tumorigen effect via vascular endotheli growth factor express abstract genom wide screen use small interf rna sirna librari reveal novel molecul involv wide rang physiolog respons express vascular endotheli growth factor vegf increas hypox condit play crucial role tumor angiogenesi tissu respons ischemia use sirna express vector librari elucid molecul modifi vegf express screen use sirna librari reveal mapkkk6 mekk6 map3k6 regul vegf express normox hypox condit vitro although biolog function map3k6 remain unknown attenu vegf express map3k6 inhibit demonstr transient transfect doubl strand rna well stabl transfect short hairpin rna express vector map3k6 condit medium map3k6 knock cell attenu endotheli prolifer capillari network format vegf depend manner vitro addit tumor cell regul map3k6 express show signific suppress tumor growth vivo accompani signific repress vessel format vegf express tumor result studi suggest map3k6 regul vegf express normoxia hypoxia regul vegf map3k6 may play crucial role angiogenesi tumorigenesi',\n",
       " 'titl acut phase gene express mice expos marin neurotoxin domoic acid abstract domoic acid rigid analog neurotransmitt glutam potent agonist kainat subtyp glutam receptor persist activ receptor subtyp result rapid excitotox calcium depend cell death neuron lesion area brain kainat pathway concentr better understand respons domoic acid induc excitotox microarray use profil gene express mous brain follow domoic acid exposur adult femal mice subject intraperiton domoic acid lethal dose kill dissect post inject total brain rna treat mice compar time match control agil featur microarray real time pcr perform select gene time point gene interrog differenti express valu respect rigor filter data result set gene use trend analysi median agglom cluster earliest gene induc consist primarili earli respons gene famili jun fos ier egr growth arrest dna damag inflammatori respons element cyclooxygenas later respond gene involv glucocorticoid respons gilz sgk cold induc protein cirbp rbm3 map kinas map3k6 nf kappab inhibit real time pcr male mice addit studi confirm express sever gene across gender transcript profil induc domoic acid share similar express profil brain ischemia excitotoxin suggest common transcript respons']"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#let's try stemming each article in a list of articles now.\n",
    "\n",
    "sw = stopwords.words()\n",
    "sb = nltk.stem.SnowballStemmer('english')\n",
    "tokenized_MAP3K6_docstrings=[]\n",
    "for docstring in MAP3K6_docstrings:\n",
    "    tokenized_article = nltk.tokenize.regexp_tokenize(docstring,'\\w{2,}') \n",
    "    tokenized_article = [word.lower() for word in tokenized_article if word.lower() not in sw and word[0] in string.ascii_letters]\n",
    "    stemmed_tokenized_article= [sb.stem(word) for word in tokenized_article]\n",
    "    tokenized_MAP3K6_docstrings.append(\" \".join(stemmed_tokenized_article))\n",
    "\n",
    "\n",
    "tokenized_MAP3K6_docstrings\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "#let's stem each list of articles now!\n",
    "\n",
    "from nltk.tokenize import RegexpTokenizer\n",
    "import nltk\n",
    "import string\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "tokenized_MAP3K2_docstrings=[]\n",
    "tokenized_MAP3K3_docstrings=[]\n",
    "tokenized_MAP3K5_docstrings=[]\n",
    "tokenized_MAP3K6_docstrings=[]\n",
    "tokenized_MAP3K15_docstrings=[]\n",
    "tokenized_docstring_list=[tokenized_MAP3K2_docstrings,tokenized_MAP3K3_docstrings,\n",
    "                             tokenized_MAP3K5_docstrings,tokenized_MAP3K6_docstrings,\n",
    "                             tokenized_MAP3K15_docstrings]\n",
    "#recall previous list of docstrings named docstring_list, which contains ordered MAP3K \n",
    "#docstrings\n",
    "#we're going to make a list called \"tokenized_docstring_list\". With the following\n",
    "#for-loop, we will write in all of the processed articles. \n",
    "\n",
    "for docstrings,tokenized_list in zip(docstring_list,tokenized_docstring_list):\n",
    "    for docstring in docstrings:\n",
    "        tokenized_article = nltk.tokenize.regexp_tokenize(docstring,'\\w{2,}') \n",
    "        tokenized_article = [word.lower() for word in tokenized_article if word.lower() not in sw and word[0] in string.ascii_letters]\n",
    "        stemmed_tokenized_article= [sb.stem(word) for word in tokenized_article]\n",
    "        tokenized_list.append(\" \".join(stemmed_tokenized_article))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "88"
      ]
     },
     "execution_count": 250,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(tokenized_MAP3K2_docstrings)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['titl mekk2 mekk3 suppress hedgehog pathway depend medulloblastoma inhibit gli1 function abstract hedgehog hh pathway play pivot role divers aspect develop postnat physiolog perturb hh signal activ gli1 glioma associ oncogen dedic transcript factor hh pathway high associ sever cancer medulloblastoma basal cell carcinoma dynam precis control gli1 activ thus import ensur proper homeostasi tumorigenesi show mekk2 map3k2 mekk3 map3k3 inhibit gli1 transcript activ oncogen function phosphoryl multipl ser thr site gli1 reduc gli1 protein stabil dna bind abil increas associ gli1 sufu interest mekk2 mekk3 respons fgf2 mediat inhibit hh signal moreov express mekk2 mekk3 inhibit medulloblastoma cell prolifer negat correl hh pathway activ medulloblastoma clinic sampl togeth find reveal novel noncanon gli1 regul provid potenti therapeut target treatment cancer aberr hh pathway activ medulloblastoma',\n",
       " 'titl optim method use peptid array identif vitro substrat lysin methyltransferas enzym abstract number post translat modif ptm phosphoryl ubiquitin extens studi lysin methyl emerg import ptm implic grow number divers cellular process date approxim identifi methyl site histon protein methyllysin proteom expand becom import identifi lysin methyltransferas enzym respons methyl event use peptid spot methyl assay proven use identif valid novel substrat lysin methyltransferas enzym use weak beta emitt coupl fluorographi detect methyl event method describ paper provid improv typic protocol assay high sensit tritium assay develop less radioact previous describ protocol provid inexpens altern weak beta signal enhanc spray wash use lysin methyl peptid spot array simpl open sourc method array quantif',\n",
       " 'titl compar evalu aqueous humor proteom primari angl closur primari open angl glaucoma age relat cataract eye abstract analyz compar total proteom aqueous humor ah patient primari angl closur glaucoma pacg primari open angl glaucoma poag age relat cataract aqueous humor collect age match pacg poag cataract patient underw surgeri immedi store analysi sampl total protein subject trypsin digest subsequ lc ms ms analysi perform deep proteom analysi data acquir lc ms ms analysi analyz use proteom discover identifi peptid match valid use percol less fals discoveri rate total protein identifi pacg poag cataract group respect group inter group comparison among group show protein identifi three group averag protein ah sampl pacg poag cataract respect identifi protein common found pacg ah protein poag ah protein cataract ah sampl individu analysi protein found sampl analyz repres constitut ah proteom spectral count analysi protein identifi three group type show signific differ protein abund protein uniqu pacg ah protein viz arhgef12 apc2 pik3cg itgb1 msn pfn1 found regul actin cytoskeleton pathway wherea poag protein viz adcy2 itpr1 mapk3 map3k2 tubb1 found gap junction pathway qualit well quantit comparison proteom ah pacg poag age relat cataract eye show signific differ thus provid clue diseas pathophysiolog',\n",
       " 'titl discoveri character iminocoumarin scaffold inhibitor mekk2 map3k2 abstract kinas mekk2 map3k2 activ mek5 erk5 cell signal pathway may play import role tumor growth metastasi thus mekk2 may repres novel kinas target cancer order identifi inhibitor mekk2 screen librari compound use high throughput mekk2 intrins atpas enzym assay identifi two hit valid structur confirm activ primari assay ic50 valu nm two orthogon mekk2 biochem assay compound potent hit subject investig limit structur activ relationship sar studi perform iminocoumarin hit result fold potent analog nm ic50 two analog improv select member kinas profil panel compar hit studi suggest substitut around phenoxi ring scaffold impart improv potenc select mekk2 analog compound nm ic50 verifi extern test inhibit mekk2 mekk3 similar potenc compound display activ cell base assay inhibit erk5 pathway activ cell inhibit cell migrat scratch assay thus identifi scaffold promis potenti develop high select potent inhibitor mekk2 inform sar studi provid specif guidanc futur design mekk2 inhibitor probe',\n",
       " 'titl promot chicken cell prolifer least part upregul myc via map3k2 target abstract cluster well studi mammal less extens studi bird demonstr cluster overexpress promot prolifer df1 cell immort chicken preadipocyt icpa member cluster target map3k2 analysi show map3k2 overexpress reduc prolifer df1 icpa cell attenu promot effect cluster cell prolifer downstream gene express analysi mapk signal pathway show map3k2 overexpress decreas myc express contrast map3k2 knockdown use rna interfer cluster overexpress increas myc express furthermor myc overexpress promot cluster express df1 cell prolifer taken togeth data indic promot chicken cell prolifer least part upregul myc via target map3k2 cluster myc e2f1 form complex regulatori network chicken cell prolifer',\n",
       " 'titl analysi mirna express profil human umbil vein endotheli cell affect heat stress abstract investig regul endotheli cell ec microrna mirna alter heat stress mirna microarray bioinformat method use determin chang mirna profil pathophysiolog characterist differenti express mirna total differenti express mirna identifi includ downregul upregul mirna gene ontolog go enrich analysi reveal valid target differenti express mirna signific enrich gene transcript regul pathway signific enrich kyoto encyclopedia gene genom analysi cancer relat includ mitogen activ protein kinas signal pathway pathway involv cancer wnt signal pathway hippo signal pathway proteoglycan involv cancer axon guidanc mirna gene mirna go network analys reveal sever hub mirna gene function notabl play domin role network map3k2 mgat4a tgfbr1 ube2r2 smad4 like control alter mirna mirna gene network mirna go network analysi reveal signific complic associ mirna differ function signific enrich function target differenti express mirna most involv regul gene transcript present studi demonstr mirna involv pathophysiolog heat treat ec understand function mirna may provid novel insight molecular mechan under heat induc pathophysiolog ec',\n",
       " 'titl differenti express co express gene network reveal candid biomark boar taint castrat pig abstract boar taint bt offens odour tast observ pork proport castrat male pig surgic castrat effect avoid bt anim welfar issu creat incent altern genom select order find candid biomark gene express profil analys tissu castrat pig group genet merit bt differenti express analysi reveal substanti chang log transform fold chang liver testi respect co express network analysi reveal one modul correl liver three modul correl testi differenti express co express analysi reveal candid biomark vari biolog function phase coq3 cox6c cyp2j2 cyp2b6 acox2 phase metabol gsto1 gsr fmo3 skatol androstenon liver steroidgenesi hsd17b7 hsd17b8 cyp27a1 regul steroidgenesi stard10 cyb5r3 gnrh signal mapk3 map2k2 map3k2 testi overrepres pathway includ ribosom protein export oxid phosphoryl liver steroid hormon biosynthesi gap junction testi futur work evalu biomark larg popul ensur use genom select program',\n",
       " 'titl regul cancer progress epitheli mesenchym transit via modul map3k2 signal tripl negat breast cancer cell abstract emerg limit data evidenc essenti involv microrna mirna develop progress tripl negat breast cancer tnbc empow small regul innov therapeut approach especi uniqu tumor subgroup still lack effici specif therapeut target herein report regul level tnbc tissu express close associ estrogen receptor alpha erα receptor well character breast cancer bca cell function ectop express inhibit migrat invas epitheli mesenchym transit emt tnbc cell mechanist standpoint bioinformat coupl luciferas gain function loss function assay show may regul tnbc progress direct target untransl region utr mitogen activ protein kinas kinas kinas map3k2 consist map3k2 overexpress could effect rescu mimic induc suppress cell invas emt light find may function tumor suppressor regul tnbc invas emt negat modul map3k2 pathway futur endeavor field may help identifi novel biomark predict prognosi respons therapi tnbc',\n",
       " 'titl therapeut potenti deregul lysin methyltransferas smyd3 safe target novel anticanc agent abstract set mynd domain contain smyd3 member lysin methyltransferas famili protein play import role methyl various histon histon target proper function smyd3 import target molecul determin differ role chromatin remodel signal transduct cell cycl control due abnorm express smyd3 tumor project prognost marker various solid cancer area cover elabor general inform structur patholog role smyd3 protein summar role smyd3 mediat protein interact oncolog pathway mutat effect regul smyd3 specif type cancer efficaci mechan action current avail smyd3 small molecul inhibitor address expert opinion find analyz herein demonstr aberr level smyd3 protein exert tumorigen effect alter epigenet regul target gene partial involv smyd3 distinct pathway provid vital opportun target cancer effect fewer side effect identif co target synergist oncogen pathway suggest could provid much benefici effect treatment solid cancer',\n",
       " 'titl transcriptom effect adenosin receptor delet healthi endotoxem murin myocardium abstract influenc adenosin receptor a2ar activ cardiac transcriptom genesi endotoxem myocard unclear appli transcriptom profil affymetrix array identifi a2ar sensit molecul reveal receptor knockout ko healthi endotoxem heart baselin cardiac function unalt a2ar sensit gene modifi a2ar ko fold chang fdr five induc mtr ppbp chac1 ctsk cnpy2 five repress hp yipf4 acta1 cidec map3k2 canon path impact alter gnb1 prkar2b pde3b map3k2 among other implic modifi protein camp pka cgmp signal lipopolysaccharid lps mg kg challeng modifi transcript wild type wt myocardium fold chang fdr induc lcn2 saa3 serpina3n cxcl9 cxcl1 repress car3 adipoq atgrl1 aplnr h19 itga8 canon respons centr inflamm immun cell death remodel pronounc amplif toll like receptor tlr under jak stat nfκb mapk pathway cardio depress profil encompass suppress adrenerg pka ca2 signal electromechan mitochondri function major shift transcript impact function injuri includ lcn2 s100a8 s100a9 icam1 vcam nox2 induct adipoq igf1 aplnr repress endotoxem respons select modifi a2ar ko support inflammatori suppress via a2ar sensit shift regul nfκb jak stat signal iκbζ iκbα stat1 cdkn1a rras2 without impact cardio depress gene profil data indic a2ar exert minor effect stress myocardium select suppress nfκb jak stat signal cardiac injuri without influenc cardiac depress endotoxemia',\n",
       " 'titl smyd3 associ regulatori pathway cancer abstract smyd3 member set mynd domain famili methyl transferas increas express correl poor prognosi various type cancer liver colon tumor smyd3 local nucleus interact rna pol h3k4me3 function select transcript amplifi oncogen gene control cell prolifer metastat spread smyd3 express high discrimin power character liver tumor posit correl poor prognosi lung pancreat cancer smyd3 act cytoplasm potenti oncogen ras erk signal methyl map3k2 kinas subsequ releas inhibitor clinico patholog analysi lung cancer patient uncov prognost signific smyd3 first progress surviv howev stratif patient accord smoke histori signific expand prognost valu smyd3 overal surviv featur suggest smoke relat effect satur clinic analysi mask function smyd3 oncogen potenti',\n",
       " 'titl microrna suppress cell prolifer metastasi target map3k2 small cell lung cancer abstract microrna class small endogen code rnas play crucial role initi progress human cancer found decreas various human malign function tumor suppressor howev regul mechan growth metastasi human small cell lung cancer nsclc still poor understood investig role growth metastasi human nsclc present studi found signific decreas lung cancer tissu cell furthermor overexpress suppress lung cancer cell prolifer migrat invas induc cell apoptosi moreov found confirm mitogen activ protein kinas kinas kinas map3k2 protein increas lung cancer tissu confirm map3k2 target gene addit knockdown map3k2 rna interfer inhibit lung cancer cell prolifer migrat invas promot cell apoptosi vitro furthermor observ tthat overexpress map3k2 partial revers inhibitori effect prolifer metastasi a549 hcc827 cell line taken togeth data indic regul lung cancer growth metastasi suppress map3k2 express least part therefor map3k2 may repres new use potenti clinic treatment diagnosi target nsclc',\n",
       " 'titl cross studi biomark signatur human bronchial epitheli cell infect respiratori syncyti virus abstract respiratori syncyti virus rsv major caus lower respiratori tract infect children elder immunocompromis individu despit advanc diagnosi treatment biomark rsv infect still unclear understand host respons propos signatur rsv infect previous studi evalu transcript profil human bronchial epitheli cell line bea infect differ strain virus howev evolut statist method function analysi togeth larg amount express data provid opportun uncov novel biomark inflamm infect view fact public avail microarray dataset rsv infect bea cell analyz linear model base statist platform function analysi innatedb result analys argu reevalu previous report transcript pattern biolog pathway bea cell line infect rsv import studi reveal biosignatur constitut gene abcc4 armc8 bclaf1 ezh1 fam118a fam208b hsph1 kazn map3k2 n6amt1 prmt2 s100pbp serpina1 tlk2 znf322 znf337 consid develop new molecular diagnosi tool',\n",
       " 'titl transcript factor foxf1 promot prostat cancer stimul mitogen activ protein kinas erk5 abstract forkhead box f1 foxf1 stromal transcript factor express epitheli cell normal prostat tissu role foxf1 cancer conflict loss cancer suggest tumor suppress function abund other associ protumorigen metastat trait extracellular signal regul kinas erk5 associ advanc stage prostat adenocarcinoma pca patient detect popul foxf1 posit tumor cell aggress mous human pca use two murin orthotop model pca found overexpress foxf1 myc cap tramp prostat tumor cell induc tumor growth prostat progress periton metastasi increas growth foxf1 posit prostat tumor associ increas phosphoryl erk5 member mitogen activ protein kinas mapk famili foxf1 transcript induc direct bound promot region gene encod kinas map3k2 wnk1 promot phosphoryl activ erk5 knockdown erk5 map3k2 wnk1 foxf1 overexpress pca cell reduc cell prolifer cultur suppress tumor growth tumor metastasi implant mice human tumor foxf1 express correl posit map3k2 wnk1 thus contrast tumor foxf1 may function tumor suppressor foxf1 promot prostat tumor growth progress activ erk5 signal result indic erk5 may new therapeut target patient foxf1 posit pca',\n",
       " 'titl structur basi substrat prefer smyd3 set domain contain protein lysin methyltransferas abstract smyd3 set domain contain lysin methyltransferas associ multipl cancer report substrat includ histon h3k4 h4k5 vascular endotheli growth factor receptor vegfr1 lys map3 kinas kinas map3k2 lys reveal structur basi substrat prefer catalyt mechan smyd3 solv co crystal structur vegfr1 map3k2 peptid structur biochem analys show map3k2 serv robust substrat smyd3 presenc phenylalanin residu posit shallow hydrophob pocket smyd3 accommod bind phenylalanin promot effici catalyt activ smyd3 contrast smyd3 display weak activ toward vegfr1 peptid locat acceptor lysin fold kinas domain vegfr1 requir drastic conform rearrang juxtaposit acceptor lysin enzymat activ site result clear reveal structur determin substrat prefer smyd3 provid mechanist insight lysin methyl map3k2 knowledg use develop smyd3 inhibitor fight map3k2 ras driven cancer',\n",
       " 'titl mekk2 mediat altern catenin pathway promot bone format abstract proper tune catenin activ osteoblast requir bone homeostasi increas decreas catenin activ patholog consequ classic pathway catenin activ stimul wnt ligand suppress constitut phosphoryl catenin glycogen synthas kinas prevent catenin ubiquitin proteasom degrad found mitogen activ protein kinas kinas kinas map3k2 mekk2 mediat altern pathway catenin activ osteoblast distinct canon wnt pathway fgf2 activ mekk2 phosphoryl catenin serin promot recruit deubiquitin enzym ubiquitin specif peptidas usp15 usp15 turn prevent basal turnov catenin inhibit ubiquitin depend proteasom degrad therebi enhanc wnt signal analysi mekk2 defici mice genet interact studi mekk2 catenin null allel confirm pathway import physiolog regul bone mass vivo thus fgf2 mekk2 pathway mediat altern nonclass pathway catenin activ pathway key regul bone format osteoblast',\n",
       " 'titl activ mek1 erk1 signal nnk induc lung carcinogenesi femal mice abstract extracellular signal regul kinas erk1 signal pathway activ sever growth factor mitogen upregul note mani human cancer includ exampl lung studi studi associ erk1 activ mutat kras encod upstream activ erk1 lung premalign lesion immunohistochem examin express phosphoryl form erk1 perk1 map erk kinas pmek1 protein correl erk activ mutat kras encod upstream activ erk1 mous lung carcinogenesi model femal week old mice administ singl dose methylnitrosamino pyridyl butanon nnk maintain without addit treatment sacrific week histopatholog adenocarcinoma adenoma hyperplasia observ lung pmek1 express most cell cytoplasm three contrast perk1 posit cell relat rare histolog type compar level pmek1 express howev perk1 posit cell adenocarcinoma still mark common hyperplasia adenoma fold fold activ mutat kras gene codon detect major adenoma adenocarcinoma without signific relat perk1 express result suggest activ erk1 play key role malign transform lung carcinogenesi featur kras mutaion activ erk1 lung premalign lesion littl regardless mutat kras erk1 activ nnk induc mous lung carcinogenesi may regul kras mutat signal pathway regulatori factor',\n",
       " 'titl role cutan squamous cell carcinoma repress mapk pathway abstract dysregul mirna common featur human cancer lack studi role mirna cutan squamous cell carcinoma cscc member famili found downregul differ type cancer role cscc remain unknown studi aim investig express cellular function cscc found underexpress cscc tissu cell line map3k2 map3k4 map3k9 predict target gene latter two gene verifi target gene cscc cell import demonstr overexpress signific inhibit cscc cell prolifer metastasi via regul map3k9 map3k4 express map3k4 map3k9 negat associ express cscc tissu result suggest indic act tumor suppressor cscc via inhibit mapk pathway result may provid promis alter strategi cscc therapi',\n",
       " 'titl identif ponatinib known kinas inhibitor potent mekk2 inhibitori activ abstract kinas mekk2 map3k2 may play import role tumor growth metastasi sever cancer type thus target mekk2 may repres novel strategi develop effect therapi cancer order identifi small molecul mekk2 inhibitori activ screen collect known kinas inhibitor use high throughput mekk2 intrins atpas enzym assay confirm activ potent hit primari assay confirm activ known kinas inhibitor mekk2 transphosphoryl slot blot assay use mkk6 substrat observ good correl potenc two orthogon mekk2 kinas activ assay format set inhibitor report ponatinib at9283 azd7762 jnj pp121 hesperadin potent mekk2 enzym inhibitori activ rang nm ic50 ponatinib fda approv drug potent inhibit mekk2 enzym activ ic50 valu nm at9283 current clinic trial produc mekk2 ic50 valu nm set known kinas inhibitor repres potent vitro mekk2 inhibitor report date may use research tool although compound select mekk2 structur compound give insight pharmacophor potent inhibit mekk2 could use initi lead design high select inhibitor mekk2',\n",
       " 'titl stk38 protein kinas preferenti inhibit tlr9 activ inflammatori respons promot mekk2 ubiquitin macrophag abstract ndr lat kinas famili high conserv yeast human remain unknown whether member famili function innat immun respons demonstr stk38 negat regul tlr9 mediat immun respons macrophag stk38 constitut associ ubiquitin e3 ligas smurf1 facilit smurf1 mediat mekk2 ubiquitin degrad mekk2 requir cpg induc erk1 activ tnf product requir lps induc tnf product accord stk38 defici increas cpg induc erk1 activ tnf product without signific affect lps induc tnf product stk38 defici mice produc tnf display increas lethal control wild type mice upon coli infect stk38 defici mice suscept clp induc sepsi control mice thus stk38 import limit inflammatori cytokin product necessari protect host inflammatori injuri infect possibl negat regul tlr9 signal',\n",
       " 'titl microrna inhibit prolifer apoptosi metastasi target map3k2 small cell lung cancer abstract grow evid indic involv complement attack migrat tumor cell nonetheless role small cell lung cancer nsclc clear mitogen activ protein kinas kinas kinas map3k2 kinas belong serin threonin protein kinas famili develop potenti therapi target map3k2 studi role prolifer apoptosi metastasi nsclc express level quantifi tumor tissu nsclc qrt pcr mimic inhibitor use verifi function cell prolifer evalu mtt assay migrat invas evalu transwel assay athym mice subcutan inject use research nsclc cell tumor format bioinformat tool luciferas assay appli detect relationship target protein level target determin western blot analysi express decreas nsclc tissu compar normal counterpart lower express nsclc tissu associ higher clinic stage nsclc metastasi poor prognosi inhibit express reduc vitro nsclc cell migrat invas well vivo metastasi map3k2 mrna contain bind site utr map3k2 one target togeth data demonstr inhibit prolifer apoptosi metastasi nsclc target map3k2 may use prognosi marker nsclc clinic research',\n",
       " 'titl mapk gene interact diet lifestyl factor alter risk breast cancer breast cancer health dispar studi abstract mitogen activ protein kinas mapk integr point multipl biochem signal evalu mapk gene breast cancer risk determin diet lifestyl factor mediat risk data popul base case control studi conduct southwestern unit state california mexico includ control case percent indigen american ancestri determin ancestri inform marker adapt rank truncat product artp use determin signific gene pathway breast cancer risk menopaus status genet ancestri level estrogen receptor progesteron receptor pr strata map3k9 associ breast cancer overal artp strongest associ among women highest ancestri artp sever snps map3k9 associ pr tumor interact dietari oxid balanc score dob dietari folat bodi mass index bmi alcohol consumpt cigarett smoke histori diabet dusp4 mapk8 interact calori alter breast cancer risk mapk1 interact dob dietari fiber folat bmi map3k2 interact dietari fat mapk14 interact dietari folat bmi pattern associ across diet lifestyl factor similar biolog properti snps within gene provid support associ',\n",
       " 'titl lysin methyl cancer smyd3 map3k2 teach us new lesson ras erk pathway abstract lysin methyl tradit associ histon epigenet recent lysin methyltransferas demethylas involv methyl histon substrat frequent found deregul human tumour realm new discoveri unveil methyltransferas smyd3 enhanc ras driven cancer smyd3 regul differ type tumour smyd3 mediat methyl map3k2 increas mutant ras induc activ erk1 methyl map3k2 prevent bind phosphatas pp2a therebi imped impact negat regul ras erk1 signal lead format lung pancreat adenocarcinoma furthermor deplet smyd3 synergis mek inhibitor current clinic trial block ras driven pancreat neoplasia result underscor import lysin methyl regul signal pathway relev tumourigenesi endors develop drug target unregul lysin methyl therapeut agent struggl cancer',\n",
       " 'titl oncoprotein hbxip enhanc migrat breast cancer cell increas filopodia format involv mekk2 erk1 capn4 signal abstract report oncoprotein hepat interact protein hbxip play crucial role promot migrat breast cancer cell lamellipodia filopodia protrus play fundament role involv dynam cytoskeleton reorgan metastasi cancer observ express level hbxip calpain small subunit capn4 high clinic metastat lymph node breast tumor found hbxip abl regul capn4 level promot mrna protein breast cancer cell activ erk1 moreov show hbxip activ erk1 regul mekk2 function reveal hbxip increas filopodia format capn4 result cell migrat thus conclud oncoprotein hbxip enhanc migrat breast cancer increas filopodia format involv mekk2 erk1 capn4 signal therapeut hbxip may serv novel target breast cancer',\n",
       " 'titl mekk2 regul paxillin ubiquityl local mda mb breast cancer cell abstract intracellular kinas mekk2 mitogen activ protein kinas extracellular signal regul kinas kinas kinas upstream regul jnk jun termin kinas addit function mekk2 well defin silenc mekk2 express invas breast tumour cell mark inhibit xenograft metastasi indic mekk2 control tumour cell function requir tumour progress previous investig mekk2 function discov tumour cell attach fibronectin recruit mekk2 focal adhes complex mekk2 knockdown associ stabil focal adhes signific inhibit tumour cell migrat present studi investig mekk2 function focal adhes report mekk2 physic associ ld1 motif focal adhes protein paxillin reveal mekk2 induc paxillin ubiquityl function requir paxillin ld1 motif mekk2 kinas activ final demonstr mekk2 promot paxillin redistribut focal adhes cytoplasm promot paxillin degrad taken togeth result reveal novel function mekk2 regul ubiquityl depend paxillin redistribut breast tumour cell',\n",
       " 'titl studi gene involv adipocyt differenti abstract use microarray techniqu gene express late phase adipocyt differenti investig gene play import role stimul adipocyt growth lipid droplet format therefor contribut great deal onset obes use sw872 adipocyt microarray techniqu gene relat adipocyt differenti test compar undifferenti preadipocyt day induct real time revers transcript polymeras chain reaction rt pcr use confirm transcriptor express determin increas decreas undifferenti adipocyt among high express fold rt pcr confirm includ apelin cidec pid1 lyrm1 add1 pparγ2 angptl4 adipoq acox1 fip1l1 map3k2 pex14 furthermor gene involv lipid metabol signal transduct dna replic redox status transcript factor determin well novel gene involv adipogenesi apelin detect varieti gene discov valid rt pcr late phase adipocyt differenti may help us better understand onset obes potenti role adipocyt organ',\n",
       " 'titl erk5 signal get xiap role ubiquitin disassembl mapk cascad abstract',\n",
       " 'titl ebv microrna bart target map3k2 facilit persist vivo inhibit viral replic cell abstract ebv oncogen human herpesvirus abil infect transform cell latent vitro howev virus establish lifetim benign persist latent infect rest memori cell vivo virus quiescent express none known latent protein virus encod micro rnas mirna transcrib bamh1 fragment rightward transcript bart region virus shown previous subset mirna present high copi number latent infect memori cell vivo suggest role maintain latenc describ role one mirna bart show target utr mrna encod import cellular signal molecul map kinas kinas kinas map3k2 exact site oncogen cellular mirna knowledg first report virus encod mirna suppress target map kinas signal cascad central signal transduct pathway govern broad spectrum biolog process show map3k2 intermediari signal pathway initi lytic viral replic thus express latent infect cell effect block viral replic propos role maintain latenc reduc risk fortuit reactiv virus latent infect memori cell',\n",
       " 'titl diet lifestyl factor interact mapk gene influenc surviv breast cancer health dispar studi abstract mapk gene activ varieti factor relat growth factor hormon environment stress evalu associ mapk gene surviv among nonhispan white hispan nativ american women diagnos breast cancer assess influenc diet lifestyl genet ancestri associ percent ancestri determin ancestri inform marker adapt rank truncat product artp use determin gene pathway signific associ predomin observ among women lower ancestri specif mitogen activ protein kinas mapk pathway associ caus mortal artp breast cancer specif mortal artp howev map2k1 map3k9 associ breast cancer specif caus mortal mapk12 artp associ breast cancer specif mortal map3k1 artp mapk1 artp associ caus mortal among women higher ancestri map3k2 signific associ caus mortal artp sever diet lifestyl factor includ alcohol consumpt calor intak dietari folat cigarett smoke signific modifi associ mapk gene caus mortal studi support associ mapk gene surviv diagnosi breast cancer especi among women low ancestri interact genet variat mapk pathway diet lifestyl factor women support import role factor breast cancer survivorship',\n",
       " 'titl ubiquitin depend regul mekk2 mek5 erk5 signal modul xiap ciap1 abstract mitogen activ protein kinas mapk high conserv protein kinas modul control fundament cellular process activ mapk well studi littl known mechan drive inactiv uncov role ubiquitin inactiv mapk modul extracellular signal regul kinas erk5 uniqu conserv member mapk famili activ respons various stimuli three tier cascad constitut mek5 mekk2 reveal unexpect role inhibitor apoptosi protein iap inactiv erk5 pathway bimod manner involv direct interact ubiquitin xiap direct interact mekk2 compet pb1 domain mediat bind mek5 xiap ciap1 conjug predomin k63 link ubiquitin chain mekk2 mekk3 direct imped mek5 erk5 interact trimer complex lead erk5 inactiv consist loss xiap ciap1 various strategi lead hyperactiv erk5 normal tumorigen cell loss xiap promot differenti human primari skelet myoblast myocyt mekk2 erk5 depend manner result reveal novel obligatori role iap ubiquitin physic function disassembl erk5 mapk modul human muscl cell differenti',\n",
       " 'titl smyd3 link lysin methyl map3k2 ras driven cancer abstract deregul lysin methyl signal emerg common aetiolog factor cancer pathogenesi inhibitor sever histon lysin methyltransferas kmts develop chemotherapeut larg cytoplasm kmt smyd3 set mynd domain contain protein overexpress numer human tumour howev molecular mechan smyd3 regul cancer pathway relationship tumorigenesi vivo larg unknown show methyl map3k2 smyd3 increas map kinas signal promot format ras driven carcinoma use mous model pancreat ductal adenocarcinoma lung adenocarcinoma found abrog smyd3 catalyt activ inhibit tumour develop respons oncogen ras use protein array technolog identifi map3k2 kinas target smyd3 cancer cell line smyd3 mediat methyl map3k2 lysin potenti activ ras raf mek erk signal modul smyd3 deplet synerg mek inhibitor block ras driven tumorigenesi final pp2a phosphatas complex key negat regul map kinas pathway bind map3k2 interact block methyl togeth result elucid new role lysin methyl integr cytoplasm kinas signal cascad establish pivot role smyd3 regul oncogen ras signal',\n",
       " 'titl cancer biolog enzym meet surpris target abstract',\n",
       " 'titl dna microarray analysi gene differenti express adipocyt differenti abstract present studi human liposarcoma cell line sw872 use identifi global chang gene express profil occur adipogenesi explor gene express late phase adipocyt differenti gene may play major role promot excess prolifer accumul lipid droplet contribut develop obes use microarray base technolog examin differenti gene express earli differenti adipocyt late differenti adipocyt valid gene exhibit greater equal fold increas late phase adipocyt differenti polymeras chain reaction rt pcr compar undifferenti preadipocyt found gene increas earli differenti adipocyt gene increas later differenti adipocyt furthermor gene found express fold higher late phase adipocyt differenti result accord rtpcr test valid gene name cidec pid1 lyrm1 add1 ppar angptl4 adipoq acox1 fip1l1 map3k2 pex14 gene found express later phase adipocyt differenti involv obes relat diseas find may help better understand mechan obes relat diseas',\n",
       " 'titl restor mir17 solid tumor cell enhanc natur killer cell antitumor activ target mekk2 abstract aberr microrna mirna express identifi various human solid cancer howev whether level mirna express tumor cell effect tumor progress determin proof concept studi restor high level express mir17 cluster mirna reveal function tumor suppressor murin solid cancer cell specif genet engin express higher level mir17 mir17 cluster murin breast cancer colon cancer cell trigger natur killer nk cell recognit inhibit express mhc class mekk2 mek5 erk5 pathway result mous tumor studi recapitul use sampl human solid tumor togeth data indic mir17 mirna function tumor suppressor reprogram tumor cell nk cell mediat cytotox',\n",
       " 'titl mekk2 regul focal adhes stabil motil invas breast cancer cell abstract mek kinas mekk2 serin threonin kinas function mapk kinas kinas map3k regul activ mitogen activ protein kinas mapk recent demonstr ablat mekk2 express invas breast tumor cell dramat inhibit xenograft metastasi mechan mekk2 influenc metastasi relat tumor cell function unknown studi investig mekk2 function demonstr silenc mekk2 express breast tumor cell signific enhanc cell spread area focal adhes stabil reduc cell migrat show cell attach matrix protein fibronectin matrigel induc mekk2 activ local focal adhes reveal mekk2 ablat enhanc focal adhes size frequenc therebi link mekk2 function focal adhes stabil final show mekk2 knockdown inhibit fibronectin induc extracellular signal regul kinas erk5 signal focal adhes kinas fak autophosphoryl taken togeth result strong support role mekk2 regul signal modul breast tumor cell spread area migrat control focal adhes stabil',\n",
       " 'titl mekk2 kinas associ protein regul activ jun termin kinas abstract mekk2 map erk kinas kinas serin threonin kinas belong mekk ste11 famili map kinas kinas kinas map ks mekk2 integr stress mitogen signal activ nf jnk1 p38 erk5 pathway found mekk2 regul phosphoryl depend associ group adapt modul dimer associ protein found mekk2 phosphoryl thr result decreas activ loop phosphoryl ser consequ reduc activ mechanist found mekk2 associ inact mekk2 absenc bind led tran autophosphoryl ser enforc bind reduc ser tran autophosphoryl express t283a mekk2 within mekk2 background enhanc stress activ jun termin kinas activ elev express reduc erk activ correspond reduc prolifer rate result indic thr phosphoryl import regulatori mechan mekk2 activ',\n",
       " 'titl dietari influenc mapk signal pathway risk colon rectal cancer abstract mitogen activ protein kinas mapk pathway regul cellular function includ cell prolifer differenti migrat apoptosi associ gene dusp erk1 jnk p38 mapk signal pathway dietari factor associ growth factor inflamm oxid stress risk colon rectal cancer evalu data includ colon case control rectal case control statist signific interact observ mapk signal pathway adjust multipl comparison dusp gene interact carbohydr mutagen index calori calcium vitamin lycopen dietari fat folic acid selenium mapk1 mapk3 mapk1 raf1 within erk1 mapk signal pathway interact dietari fat crucifer veget within jnk mapk signal pathway interact map3k7 protein vitamin iron folic acid carbohydr crucifer veget map3k10 folic acid map3k9 lutein zeaxanthin mapk8 calcium map3k3 calcium lutein map3k1 crucifer veget interact within p38 signal pathway includ mapk14 calori carbohydr satur fat selenium vitamin map3k2 carbohydr folic acid data suggest dietari factor involv inflamm oxid stress interact mapk signal gene alter risk colorect cancer',\n",
       " 'titl develop valid high throughput intrins atpas activ assay discoveri mekk2 inhibitor abstract kinas mekk2 map3k2 recent implic tumor growth metastasi thus select inhibit mekk2 may novel strategi cancer therapi identifi inhibitor mekk2 kinas activ develop novel activ assay mekk2 base discoveri recombin purifi mekk2 intrins atpas activ mekk2 atpas assay valid enzym ident enzymat puriti multipl method includ mass spectrometri analysi test differ sourc mekk2 compar atpas assay ic50 data multipl inhibitor literatur valu ic50 data generat use mekk2 bind transphosphoryl assay taken togeth data indic genuin mekk2 activ measur assay atpas contribut signal miniatur version assay valid high throughput screen compound librari screen screen hit generat compar potenc mekk2 intrins atpas bind transphosphoryl assay identifi novel mekk2 inhibitor confirm crizotinib bosutinib potent vitro inhibitor mekk2 activ ic50 valu nm thus assay util discoveri small molecul inhibitor mekk2 activ',\n",
       " 'titl map kinas kinas kinas mekk2 regul hypertroph remodel right ventricl hypoxia induc pulmonari hypertens abstract pulmonari hypertens ph result pressur overload right ventricl rv heart initi patholog rv remodel ultim lead right heart failur substanti research indic signal mapk superfamili mediat patholog cardiac remodel consider led us test hypothesi regulatori protein mapkkk mekk2 contribut rv hypertrophi hypoxia induc ph transgen mice global knockout mekk2 mekk2 mice age match wild type wt mice expos chronic hypobar hypoxia wk compar anim normoxia exposur chronic hypoxia induc ph wt mekk2 mice respons ph wt mice show rv hypertrophi demonstr increas ratio rv weight bodi weight increas rv wall thick diastol increas cardiac myocyt size compar normox control anim contrast measur rv hypertrophi seen wt mice chronic hypoxia attenu mekk2 mice furthermor chronic hypoxia elicit alter program hypertroph inflammatori gene express consist patholog rv remodel wt mice mekk2 delet select inhibit inflammatori gene express compar wt mice action mekk2 mediat part regul abund phosphoryl effector erk5 conclus signal mekk2 contribut rv hypertrophi alter myocardi inflammatori gene express respons hypoxia induc ph therapi target mekk2 may protect myocardium hypertrophi patholog remodel human ph',\n",
       " 'titl map kinas gene colon rectal cancer abstract mitogen activ protein kinas mapk pathway regul mani cellular function includ cell prolifer differenti migrat apoptosi evalu genet variat jun termin kinas p38 extracellular regul kinas mapk signal pathway colon rectal cancer risk use data popul base case control studi colon case control rectal case control assess gene dusp1 dusp2 dusp4 dusp6 dusp7 map2k1 map3k1 map3k2 map3k3 map3k7 map3k9 map3k10 map3k11 mapk1 mapk3 mapk8 mapk12 mapk14 raf1 map2k1 rs8039880 odd ratio confid interv gg versus aa genotyp map3k9 rs11625206 recess model associ colon cancer adj valu dusp1 rs322351 tt versus cc mapk8 rs10857561 aa versus gg ga associ rectal cancer adj aspirin steroid anti inflammatori drug cigarett smoke bodi mass index interact sever gene alter cancer risk genet variant uniqu associ kras tp53 cimp tumor dusp2 rs1724120 hazard rate ratio hrr aa versus gg ga map3k10 rs112956 hrr ct tt versus cc map3k11 hrr tt versus gg gt influenc surviv diagnosi colon cancer map2k1 rs8039880 hrr gg versus ag gg raf1 rs11923427 hrr aa versus tt associ rectal cancer surviv data suggest genet variat mapk signal pathway influenc colorect cancer risk surviv diagnosi associ may modifi lifestyl factor influenc inflamm oxid stress',\n",
       " 'titl smooth muscl actin express stromal fibroblast head neck squamous cell carcinoma increas express galectin induct poor prognosi factor abstract tumor stroma activ part influenc biolog properti malign via molecular cross talk cancer associ fibroblast play signific role interact cell frequent express smooth muscl actin classifi myofibroblast adhes growth regulatori lectin galectin effector generat studi set presenc smooth muscl actin posit cancer associ fibroblast relat endogen lectin vivo competitor galectin squamous cell carcinoma head neck upregul galectin presenc high signific correl presenc smooth muscl actin posit cancer associ fibroblast tumor pinpoint correl molecular level appli microarray analys transcript profil correspond tumor signific correl sever transcript detect protein level galectin cancer associ fibroblast activ gene map3k2 trim23 ptplad1 fusip1 slc25a40 spin1 relat known squamous cell carcinoma poor prognosi factor nf upregul splice downregul result provid new insight signific presenc myofibroblast squamous cell carcinoma',\n",
       " 'titl microrna inhibit growth hepatoma cell target mekk2 cyclin d1 abstract grow evid indic deregul microrna mirna contribut tumorigenesi previous reveal microrna involv complement attack migrat breast cancer cell report show import mirna develop hepatocellular carcinoma hcc data show express level signific reduc clinic hcc tissu hepatoma cell line observ introduct dramat suppress growth hepatoma cell coloni format assay ethynyl deoxyuridin edu incorpor assay dimethylthiazol yl diphenyltetrazolium bromid mtt assay moreov ectop express abl inhibit growth hepatoma cell nude mice studi reveal mitogen activ protein kinas kinas kinas mekk2 cyclin d1 two direct target gene silenc mekk2 cyclin d1 abl inhibit growth hepatoma cell vitro vivo consist effect overexpress growth hepatoma cell addit signific decreas phosphoryl level jun termin kinas jnk downstream effector mekk2 retinoblastoma rb downstream effector cyclin d1 conclus abl inhibit growth hepatoma cell target mekk2 cyclin d1 vitro vivo find provid new insight role develop hcc implic potenti applic cancer therapi',\n",
       " 'titl kinas mekk2 mekk3 regul transform growth factor mediat helper cell differenti abstract mitogen activ protein kinas mapk key mediat cell receptor tcr signal role helper th cell differenti unclear show mapk kinas kinas mekk2 encod map3k2 mekk3 encod map3k3 negat regul transform growth factor tgf mediat th cell differenti map3k2 map3k3 lck cre mice show abnorm accumul regulatori treg th17 cell peripheri consist map3k2 map3k3 lck cre naiv cd4 cell differenti treg th17 cell higher frequenc wild type wt cell tgf stimul vitro addit map3k2 map3k3 lck cre mice develop sever experiment autoimmun encephalomyel map3k2 map3k3 lck cre cell exhibit impair phosphoryl smad2 smad3 protein linker region negat regul tgf respons cell thus crosstalk tcr induc mapk tgf signal pathway import regul th cell differenti',\n",
       " 'titl epigenom mitochondrion bioenerget environ correct abstract juli issu gene develop yoon colleagu pp report sirna knockdown survey gene mous c2c12 cell discov gene regul mitochondri biogenesi bioenerget mani gene studi previous import regul transcript protein nucleic acid modif signal transduct notabl exampl includ brac1 brac2 pax4 sin3a fyn fes map2k7 map3k2 calmodulin camk1 ube3a wnt yoon colleagu go valid observ extens document role wnt signal regul mitochondri function',\n",
       " 'titl chip depend termin mekk2 regul tempor erk activ requir proper hyperosmot respons abstract extracellular signal regul kinas erk pathway import signal pathway regul larg number cellular process includ prolifer differenti gene express hyperosmot stress activ erk pathway wherea littl known regulatori mechan physiolog function erk activ hyperosmot respons show mapk erk kinas kinas mekk2 member mapkkk famili mediat specif transient activ erk requir induct aquaporin aqp1 aqp5 gene express respons hyperosmot stress moreov identifi e3 ubiquitin ligas carboxyl terminus hsc70 interact protein chip bind partner mekk2 deplet chip small interfer rna gene target attenu degrad mekk2 prolong erk activ interest hyperosmol induc gene express aqp1 aqp5 suppress chip deplet revers inhibit prolong phase erk activ find show transient activ erk pathway depend mekk2 activ chip depend mekk2 degrad crucial proper gene express hyperosmot stress respons',\n",
       " 'titl integr genom analysi reveal oncomir oncogen cluster regul glioblastoma survivorship abstract use multidimension genom data set glioblastoma cancer genom atlas identifi hsa cooper compon frequent occur amplicon contain cdk4 centg1 two oncogen regul rb1 pi3 kinas akt pathway respect integr dna copi number mrna microrna dna methyl data identifi function relev target glioblastoma includ pten rb1 map3k2 mekk2 demonstr alon transform cell promot glioblastoma cell growth vitro mous brain decreas pten rb1 map3k2 mekk2 protein express therebi increas akt activ promot prolifer decreas jun termin kinas depend apoptosi overexpress pten compet pten defici glioblastoma cell promot tumor growth vivo increas growth cell overexpress cdk4 centg1 import glioblastoma patient harbor amplif display mark decreas surviv thus hsa cdk4 centg1 compris function integr oncomir oncogen dna cluster promot aggress human cancer cooper target rb1 pi3k akt jnk pathway',\n",
       " 'titl imag mass spectrometri specif fragment mitogen activ protein kinas extracellular signal regul kinas kinas kinas discrimin cancer uninvolv prostat tissu abstract histopatholog standard approach tissu diagnost centerpiec patholog although current system provid prognost inform need molecular marker enhanc diagnosi better predict clinic prognosi abil local diseas specif molecular chang biopsi tissu would help improv critic patholog decis make direct profil protein tissu use matrix assist laser desorpt ionize imag mass spectrometri potenti supplement morpholog under molecular detail discoveri set prostat cancer pca contain benign prostat tissu section evalu protein express differ separ valid set tissu section pca benign use verifi result cryosect done yield tissu section analyz pathologist determin tissu morpholog mirror section imag mass spectrometri spectra acquir intens signal plot function locat within tissu express profil found discrimin pca normal tissu overexpress singl ion abl discrimin cancer uninvolv tissu tandem mass spectrometri identifi marker fragment mitogen activ protein kinas extracellular signal regul kinas kinas kinas mekk2 abil mekk2 discrimin tumor normal cell orthogon confirm studi highlight potenti approach uncov molecular detail correl patholog decis make addit identif mekk2 show abil discov protein relev pca biolog',\n",
       " 'titl nf kappab depend induct cathelicidin relat antimicrobi peptid murin mast cell lipopolysaccharid abstract import aspect innat immun respons pathogen product anti microbi peptid cathelicidin relat antimicrobi peptid cramp murin homologu human cathelicidin studi mechan regul lps induct cramp gene express mast cell investig nf kappab mapk pathway focus investig mous bone marrow deriv mast cell grown cultur stimul lps mapk nf kappab monitor immunoblot analysi erk jnk p38 mapk inhibit use sirna pharmacolog inhibitor accumul p65 compon nf kappab inhibit sirna nf kappab activ inhibit overexpress kappab alpha mekk2 mekk3 overexpress transfect effect treatment cramp gene express monitor rt pcr inhibit erk jnk p38 mapk littl discern effect lps induc cramp gene express overexpress mekk2 mekk3 likewis littl impact howev inhibit accumul p65 nf kappab prevent lps induc cramp mrna import role nf kappab cramp gene express confirm overexpress kappab alpha reduc basal induc level cramp mrna nf kappab mapk play import role lps mediat induct cramp gene mast cell defect inhibit nf kappab activ may increas suscept bacteri viral pathogen sensit cathelicidin',\n",
       " 'titl vivo profil endogen interact knock mammalian cell abstract precis identifi screen target specif protein protein interact endogen level introduc novel quantit proteom method term vivo profil endogen interact knock ipeik design mous embryon fibroblast mef deriv target gene knockout ko mice stabl isotop tag serv target free background light target protein specif protein complex form correspond wild type wt cell mass spectrometr analysi pair label versus heavi isotop label peptid signal deriv wt versus ko cell respect quantit measur abund differ protein complex immunoprecipit ip target express wt versus target absent ko cell respect protein detect littl presenc cell ko origin determin target specif interact partner dynam interactor could identifi differ ip mix scheme use ipeik identifi multipl interact partner previous known unknown associ mitogen activ protein kinas kinas kinas mekk2 avail larg librari knockout mice model various target protein biolog interest method general applic screen endogen target specif ppis physiolog relev',\n",
       " 'titl xiap regul bi phasic nf kappab induct involv physic interact ubiquitin mekk2 abstract transcript factor nf kappab transient activ wide varieti stress signal includ pro inflammatori mediat regul express gene immun inflammatori anti apoptot function strength kinet induct well ultim regul subject multipl level regul one regulatori protein chromosom link inhibitor apoptosi xiap besid anti apoptot properti shown enhanc nf kappab activ howev under molecular mechan remain elus show follow tnfalpha stimul xiap regul second wave nf kappab activ xiap interact ubiquitin mekk2 kinas previous associ bi phasic nf kappab activ conclud interact mekk2 xiap function ubiquitin ligas depend manner evok second wave nf kappab activ result modul nf kappab target gene express',\n",
       " 'titl high replic hepat virus induc apoptosi hepatocyt abstract investig effect high level hepat virus hbv replic hepatocyt qsg cell human hepatocyt line qsg cultur transfect plasmid puc18 hbv1 puc18 contain full length hbv dna standard calcium phosphat precipit method qsg cell transfect plasmid puc18 control cell growth curv drawn day transfect four day transfect hbv dna cultur medium detect use fluoresc quantit real time pcr cell apoptosi detect use termin deoxynucleotidyl transferas mediat dutp nick label electron microscopi differenti express gene analyz use oliga signal pathway micro array curv cell growth show amount control qsg cell increas time signific faster puc18 hbv1 transfect qsg cell increas time four day transfect hbsag posit rate puc18 hbv1 transfect cell apoptot rate hbv dna level cultur supernat peak day adder transfect maximum valu copi ml gene relat cell growth apoptosi casp3 casp7 apr cdc2 mapk6 map3k2 differenti express high replic hbv mark inhibit growth hepatocyt induc cell apoptosi',\n",
       " 'titl mitogen activ protein kinas kinas kinas protect nickel induc acut lung injuri abstract nickel compound environment occup hazard pose serious health problem causat factor acut lung injuri jun termin kinas jnks regul mitogen activ protein map kinas map2 kinas cascad implic nickel toxic studi use genet modifi cell mice investig involv two upstream map3k map3k1 nickel induc jnk activ acut lung injuri mous embryon fibroblast level jnk activ cytotox induc nickel similar map3k2 null wild type cell much lower map3k1 map3k2 doubl null cell convers level jnk activ cytotox unexpect much higher map3k1 null cell adult mous tissu map3k1 wide distribut abund express bronchiol epithelium lung accord map3k1 ablat mice result sever nickel induc acut lung injuri reduc surviv base find propos role map3k1 reduc jnk activ protect mice nickel induc acut lung injuri',\n",
       " 'titl epigallocatechin gallat inhibit prostaglandin d2 stimul hsp27 induct via suppress p44 p42 map kinas pathway osteoblast abstract previous report prostaglandin d2 pgd2 stimul heat shock protein hsp27 induct p38 mitogen activ protein map kinas stress activ protein kinas sapk jun termin kinas jnk p44 p42 map kinas osteoblast like mc3t3 e1 cell present studi investig whether epigallocatechin gallat egcg major polyphenol found green tea affect induct hsp27 cell mechan egcg signific reduc hsp27 induct stimul pgd2 without affect level hsp70 pgd2 induc phosphoryl p38 map kinas sapk jnk affect egcg contrari egcg mark suppress pgd2 induc phosphoryl p44 p42 map kinas mek1 howev pgd2 induc phosphoryl raf inhibit egcg result strong suggest egcg suppress pgd2 stimul induct hsp27 point raf mek1 osteoblast',\n",
       " 'titl negat regul mekk1 signal serin threonin kinas stk38 abstract mitogen activ protein kinas mapk activ kinas cascad mapk mapk kinas mapkk mapkk kinas mapkkk mapkkk phosphoryl activ downstream mapkk turn phosphoryl activ downstream mapk mapkkk protein relay upstream signal mapk cascad induc cellular respons howev molecular mechan given mapkkk regul remain larg unknown found serin threonin protein kinas stk38 physic interact mapkkk mekk1 mekk2 mekk1 carboxi terminus includ catalyt domain amino terminus mekk1 necessari interact stk38 stk38 inhibit mekk1 activ without prevent mekk1 bind substrat sek1 import stk38 suppress autophosphoryl mekk2 without interf mekk2 dimer format convert mekk2 phosphoryl nonphosphoryl form negat regul mekk1 due phosphoryl stk38 hand stk38 short hairpin rna enhanc sorbitol induc activ mekk2 phosphoryl downstream mapkk mkk3 taken togeth result indic stk38 negat regul activ mekk1 direct interact catalyt domain mekk1 suggest novel mechan mekk1 regul',\n",
       " 'titl endotheli nitric oxid synthas play obligatori role late phase ischem precondit activ protein kinas epsilon p44 mitogen activ protein kinas pser signal transduc activ transcription1 pathway abstract role endotheli nitric oxid synthas eno ischem precondit pc cardioprotect poor understood address issu use genet rather pharmacolog approach nonprecondit state eno mice exhibit infarct size similar wild type mice sequenc six minut coronari occlus minut reperfus cycl ischem pc induc late pc wild type mice genet delet eno abrog cardioprotect induc late pc wild type mice ischem pc induc membran transloc protein kinas pkc epsilon increas pser mek ptyr p44 mitogen activ protein kinas nuclear pser signal transduc activ transcript stat pser stat3 nuclear stat1 dna bind activ follow upregul cyclooxygenas protein activ hour later chang abrog eno mice donor diethylenetriamin recapitul effect ischem pc contrast previous report found basal eno activ modul infarct size nonprecondit state howev eno obligatorili requir develop cardioprotect effect late pc act trigger process activ pkc epsilon mek p44 mitogen activ protein kinas pathway lead ser phosphoryl stat1 stat3 consequ upregul stat depend gene cyclooxygenas effect eno deriv reproduc exogen donor impli nitrat upregul cardiac cyclooxygenas',\n",
       " 'titl inhibit tumour necrosi factor alpha secret epidermtm tissu novel small molecul utl abstract utl chlorophenyl carboxyamidyl methylisoxazol small molecul tumour necrosi factor tnf alpha modul investig potenti sever immun mediat diseas includ psoriasi aim determin whether utl repres potenti antipsoriasi agent first keratinocyt cell base studi use studi inhibit tnf alpha gene suppress utl vitro second multilay human epidermi tissu model consist normal human deriv epiderm keratinocyt use studi dose depend reduct tnf alpha utl well feasibl use utl lotion formul cell base studi show utl signific reduc tnf alpha secret keratinocyt reduct mug ml suppress jak3 map3k2 gene respect human epidermi tissu model reduct tnf alpha utl appear dose depend microg ml utl reduc cell death induc ultraviolet uv tissu treat utl preliminari lotion formul show signific reduct tnf alpha induc uvb result indic utl may worthi investig potenti topic agent psoriasi',\n",
       " 'titl noncanon function mekk2 mek5 pb1 domain coordin extracellular signal regul kinas jun termin kinas signal abstract mekk2 mek5 encod phox bem1p pb1 domain heterodimer one anoth mekk2 mek5 extracellular signal relat kinas erk5 form ternari complex interact involv mekk2 mek5 pb1 domain amino acid termin extens mek5 pb1 domain termin extens encod erk5 dock site requir mek5 activ erk5 pb1 domain bind front back arrang cluster basic amino acid front mekk2 pb1 domain bind back acid cluster mek5 pb1 domain termin moieti includ acid cluster mekk2 pb1 domain requir mek5 bind bind mkk7 quiescent mekk2 preferenti bind mek5 mekk2 activ result erk5 activ activ mekk2 bind activ mkk7 lead jnk activ find defin mekk2 mek5 pb1 domain uniqu use differenti bind two mitogen activ protein kinas kinas mek5 mkk7 coordin control erk5 jun termin kinas activ',\n",
       " 'titl palmit mediat downregul peroxisom prolifer activ receptor gamma coactiv skelet muscl cell involv mek1 nuclear factor kappab activ abstract mechan elev level free fatti acid caus insulin resist well understood previous studi report insulin resist state character reduct express peroxisom prolifer activ receptor gamma coactiv pgc transcript activ promot oxid capac skelet muscl cell howev littl known factor respons reduc pgc express express pgc mrna level assess c2c12 skelet muscl cell expos palmit either presenc absenc sever inhibitor studi biochem pathway involv report exposur c2c12 skelet muscl cell mmol palmit oleat reduc pgc mrna level wherea pgc express affect palmit led mitogen activ protein kinas mapk extracellular signal relat kinas erk mek1 activ addit pharmacolog inhibit pathway coincub palmit expos cell mek1 inhibitor pd98059 u0126 prevent downregul pgc furthermor nuclear factor kappab nf kappab activ involv palmit mediat pgc downregul sinc nf kappab inhibitor parthenolid prevent decreas pgc express find indic palmit reduc pgc express skelet muscl cell mechan involv mapk erk nf kappab activ',\n",
       " 'titl brain deriv neurotroph factor activ erk5 cortic neuron via rap1 mekk2 signal cascad abstract extracellular signal regul kinas erk5 activ neuron central nervous system neurotrophin includ brain deriv neurotroph factor bdnf although mek5 known mediat bdnf stimul erk5 central nervous system neuron upstream signal compon identifi report bdnf induc sustain activ erk5 rat cortic neuron activ rap1 small gtpase well mekk2 mek5 kinas data indic activ rap1 mekk2 suffici stimul erk5 wherea inhibit either rap1 mekk2 attenu bdnf activ erk5 furthermor bdnf stimul mekk2 regul rap1 evid indic ras mekk3 mek5 kinas neuron cell play signific role bdnf activ erk5 studi identifi rap1 mekk2 critic upstream signal molecul mediat bdnf stimul erk5 central nervous system neuron',\n",
       " 'titl cd36 depend signal cascad necessari macrophag foam cell format abstract accumul macrophag foam cell atherosclerot blood vessel intima critic compon atherogenesi mediat scaveng receptor depend intern oxid ldl demonstr coimmunoprecipit pull assay macrophag scaveng receptor cd36 associ signal complex contain lyn mekk2 map kinas jnk1 jnk2 specif phosphoryl macrophag expos oxldl use cell isol sra tlr2 cd36 null mice phospholipid ligand specif either sra cd36 show jnk activ mediat cd36 foam cell format activ jnk2 hyperlipidem mice diminish absenc cd36 furthermor inhibit src jnk block oxldl uptak inhibit foam cell format vitro vivo find show specif cd36 depend signal pathway initi oxldl necessari foam cell format identifi potenti target antiatherosclerosi therapi',\n",
       " 'titl protein kinas alpha betai betaii isozym regul cytokin product mast cell mekk2 erk5 depend independ pathway abstract regul mapk pathway pkc isoform examin murin bone marrow deriv mast cell bmmcs pkcalpha betai betaii isoform show robust activ fcepsilonr1 mediat stimul anti ovalbumin specif ige ovalbumin ige ova pkcalpha betai betaii involv activ jnk mekk2 erk5 differenti relat contribut isoform specif mapk pathway compon bmmcs mice lack mekk2 show reduc product tnf alpha stimul demonstr mekk2 depend independ pathway cytokin product cytokin product stimul express pkc cell mekk2 defici wild type mice activ erk5 occur bmmcs lack mekk2 indic mekk2 independ cytokin product erk5 independ sinc mapk modul differenti regul mast cell function includ degranul cytokin product suggest specif function could target inhibit specif pkc isoform',\n",
       " 'titl mip1 mekk2 interact protein control mekk2 dimer activ abstract mitogen activ protein kinas mapk cascad central compon intracellular signal network use eukaryot cell respond wide spectrum extracellular stimuli mapk activ mapk kinas turn activ mapk kinas kinas map3k howev littl known molecular aspect regul activ larg number map3k crucial relay upstream receptor mediat signal mapk cascad induc various physiolog respons studi identifi novel mekk2 interact protein mip1 regul mekk2 dimer activ form complex inact nonphosphoryl mekk2 particular mip1 prevent mekk2 activ block mekk2 dimer format turn block jnkk2 jun termin kinas jnk1 extracellular signal regul kinas ap report gene activ mekk2 furthermor found endogen mip1 mekk2 complex dissoci transient follow epiderm growth factor stimul contrast knockdown mip1 express sirna augment mekk2 mediat jnk ap report activ togeth data suggest novel model mekk2 regul activ',\n",
       " 'titl mekk2 regul coordin activ erk5 jnk respons fgf fibroblast abstract mitogen activ protein kinas mapk regul mapk kinas mkks turn regul mkk kinas mkkks singl mkkk regul sever differ mapk famili member sever mkkks often activ mapk emerg evid indic uniqu role individu mkkks act signal node coordin activ differ subset mapk respons specif cellular stimuli thus much appar overlap mapk regul differ mkkks mkkk serv specif purpos regul uniqu cellular function purpos studi defin specif role mekk2 mkkk mapk regul cell function mekk2 coordin activ erk5 jnk pathway target disrupt mekk2 express caus loss erk5 jnk activ respons fgf mous embryon fibroblast mef fgf receptor signal requir mekk2 induct mrna jun compon ap transcript complex fgf stimul mekk2 fibroblast jun phosphoryl inhibit consist loss jnk activ thus mekk2 regul ap activ two level regul express ap compon jun termin phosphoryl one function ap transcript complex regul cytokin gene express express tnfalpha inhibit mekk2 fibroblast bacteri lipopolysaccharid lps tnfalpha neither activ erk5 requir mekk2 jnk activ demonstr specif mekk2 fgf receptor signal control cytokin gene express',\n",
       " 'titl regul jun termin kinas mekk mitogen activ protein kinas kinas kinas rheumatoid arthriti abstract mitogen activ protein kinas mapk jun termin kinas jnk critic regul collagenas product rheumatoid arthriti ra mapk regul upstream kinas includ mapk kinas mapkk mapk kinas kinas map3k present studi design evalu express regul jnk pathway map3k arthriti rt pcr studi map3k gene express ra osteoarthr synovi tissu demonstr mitogen activ protein kinas erk kinas kinas mekk mekk2 apoptosi signal regul kinas tgf beta activ kinas tak1 gene express trace amount mekk3 mekk4 mlk3 mrna detect western blot analysi demonstr immunoreact mekk2 tak1 trace amount mekk3 mekk1 apoptosi signal regul kinas analysi map3k mrna cultur fibroblast like synoviocyt fls show map3k examin express western blot analysi fls demonstr mekk1 mekk2 tak1 readili detect subsequ focus function studi vitro kinas assay use mekk2 immunoprecipit demonstr increas mekk2 mediat phosphoryl key mapkk activ jnk mapk kinas mkk mkk7 furthermor mekk2 immunoprecipit activ jun depend manner activ inhibit select jnk inhibitor sp600125 interest mekk1 immunoprecipit stimul fls appear activ jun jnk pathway tak1 activ jun depend jnk erk p38 data indic mekk2 potent activ jnk pathway fls signal complex includ mekk2 mkk4 mkk7 jnk potenti therapeut target ra',\n",
       " 'titl wnk1 activ erk5 mekk2 depend mechan abstract wnk1 belong uniqu protein kinas famili lack catalyt lysin normal posit mutat human wnk1 wnk4 implic caus famili form hypertens report overexpress wnk1 led increas activ cotransfect erk5 hek293 cell erk5 activ block mek5 inhibitor u0126 express domin negat mek5 mutant express domin negat mutant mekk2 mekk3 block activ erk5 wnk1 moreov mekk2 mekk3 coimmunoprecipit endogen wnk1 cell lysat wnk1 phosphoryl mekk2 vitro mekk3 activ wnk1 cell final erk5 activ epiderm growth factor attenu suppress wnk1 express use small interf rna taken togeth result place wnk1 erk5 map kinas pathway upstream mekk2',\n",
       " 'titl novel hydrogen peroxid induc phosphoryl ubiquitin pathway lead rna polymeras proteolysi abstract dna damag induc ubiquitin largest subunit rna polymeras rpb1 implic transcript coupl repair year studi far howev limit use bulki helix distort dna damag caus uv light cisplatin correct nucleotid excis repair pathway bulki helix distort damag caus high frequenc reactiv oxygen speci cell correct base excis repair pathway contrari classic view recent found second type dna lesion caus rna polymeras stall vitro paper show hydrogen peroxid caus signific ubiquitin proteasom degrad rpb1 mechan distinct employ uv irradi uv irradi treatment caus characterist chang protein kinas phosphoryl carboxyl termin domain ser induc ubiquitin like depend unusu ser phosphoryl erk1 moreov induc ubiquitin occur transcript engag polymeras without help cockayn syndrom protein hippel lindau tumor suppressor protein requir uv induc ubiquitin result suggest stall polymeras recogn ubiquitin differenti depend type dna lesion find may general implic basic mechan transcript coupl nucleotid excis repair base excis repair',\n",
       " 'titl mechan proinflammatori cytokin induc biphas nf kappab activ abstract transcript factor nf kappab regul gene involv innat adapt immun respons inflamm apoptosi oncogenesi proinflammatori cytokin induc activ nf kappab transient persist phase investig mechan biphas nf kappab activ result show mekk3 essenti regul rapid activ nf kappab wherea mekk2 import control delay activ nf kappab respons stimul cytokin tnf alpha mekk3 involv format ikappabalpha nf kappab ikk complex wherea mekk2 particip assembl ikappabbeta nf kappab ikk complex two distinct complex regul proinflammatori cytokin induc biphas nf kappab activ thus studi reveal novel mechan differ map3k ikappab isoform involv specif complex format ikk nf kappab regul biphas nf kappab activ find provid insight regul cytokin induc specif tempor gene express',\n",
       " 'titl mef2c regul jun tnf alpha gene express stimul mast cell abstract mitogen activ protein kinas mapk cascad play essenti role transduct extracellular signal cytoplasm nuclear effector mapk kinas kinas mekk2 essenti activ jun termin kinas jnk extracellular signal regul kinas erk5 pathway import express specif cytokin gene mast cell follow cross link high affin ige receptor fcepsilonri consequ erk5 activ activ transcript factor myocyt enhanc factor mef2c lead increas jun express investig role mef2c activ mast cell demonstr requir sequenti activ signal cascad mekk2 mek5 erk5 follow phosphoryl mef2c activ mef2c regul transcript jun tnf alpha inhibit erk5 mek5 activ activ mekk2 defici mast cell associ inhibit mef2c phosphoryl decreas jun express thus data defin activ modul mekk2 mek5 erk5 mef2c transcript activ jun mast cell follow fcepsilonri cross link result demonstr novel import mekk2 depend role mef2c induct jun express mast cell activ fcepsilonri pathway distinct involv mekk2 mek5 erk5 regul mast cell cytokin product',\n",
       " 'titl pb1 domain mekk2 mekk3 interact mek5 pb1 domain activ erk5 pathway abstract mekk2 mekk3 mapk kinas kinas activ erk5 pathway phosphoryl activ mapk kinas mek5 activ mek5 phosphoryl activ erk5 pb1 domain first defin p67phox bem1p protein shown mediat protein protein heterodimer pb1 domain encod within termin portion mekk2 mekk3 mek5 herein analyz function role mekk2 mekk3 mek5 pb1 domain erk5 activ pathway pb1 domain mekk2 mekk3 bind pb1 domain mek5 signific homo heterodimer one anoth vitro furthermor co immunoprecipit mekk2 mek5 cell lysat show form complex vivo delet mutat mekk2 pb1 domain abolish mekk2 mek5 complex demonstr pb1 domain interact requir mekk2 mek5 interact express cell mekk2 mekk3 pb1 domain inhibit erk5 activ wherea express mutant mekk2 unabl bind mek5 pb1 domain express p67phox pb1 domain effect erk5 activ find demonstr pb1 domain mediat associ mekk2 mekk3 mek5 respect pb1 domain kinas critic regul erk5 pathway free pb1 domain mekk2 mekk3 function effect inhibit erk5 pathway p38 jnk pathway demonstr specif uniqu requir mekk2 mekk3 pb1 domain regul erk5 activ',\n",
       " 'titl protein kinas associ kinas activ nfkappab kinas depend kinas independ manner abstract protein kinas associ kinas pkk known rip4 dik activ nfkappab overexpress cell line requir keratinocyt differenti vivo howev littl understood factor upstream pkk pkk activ nfkappab show certain catalyt inact mutant pkk activ nfkappab although lesser degre wild type pkk delet specif domain wild type pkk diminish abil enzym activ nfkappab delet made catalyt inact pkk background complet ablat nfkappab activ pkk may phosphoryl two specif mitogen activ protein kinas kinas kinas mekk2 mekk3 interact may part mediat critic activ loop residu thr184 catalyt inact pkk mutant block phorbol ester induc nfkappab activ interfer unexpect enhanc activ nfkappab two mitogen activ protein kinas kinas kinas taken togeth data indic pkk may function kinas depend well kinas independ manner activ nfkappab',\n",
       " 'titl translat inhibitor sensit prostat cancer cell apoptosi induc tumor necrosi factor relat apoptosi induc ligand trail activ jun termin kinas abstract tumor necrosi factor relat apoptosi induc ligand trail induc apoptosi sever human tumor vitro vivo howev tumor remain resist poor understood reason use quantit dna fragment assay apoptosi shown human prostat cancer cell resist wide rang trail dose ng ml howev translat inhibitor anisomycin cycloheximid emetin harringtonin puromycin unlik sever transcript inhibitor signific sensit pc3 neomycin pc3 neo cell trail induc apoptosi effect inhibit pc3 cell engin express bcl2 pc3 bcl2 translat inhibitor led activ jun termin kinas jnk play role sensit process inhibit jnk activ result protect trail plus translat inhibitor induc apoptosi jnk activ may requir process suffici activ jnk use mekk2 express vector mimic sensit effect translat inhibitor stress activ protein kinas erk p38 play insignific role determin apoptot sensit conclud activ jnk requir sensit pc3 cell trail induc apoptosi translat inhibitor cell otherwis trail resist howev addit jnk activ aspect translat inhibit suppress activ apoptosi inhibitori protein activ signal transduct pathway must involv',\n",
       " 'titl mutat protein kinas subdomain differenti affect mekk2 mekk1 activ abstract mapk erk kinas kinas mekk2 member mitogen activ protein kinas kinas kinas map3k famili protein kinas map3k compon three tier protein kinas pathway map3k phosphoryl activ mitogen activ protein kinas kinas map2k turn activ mitogen activ protein kinas mapk previous identifi residu within protein kinas subdomain map3k mekk1 critic interact map2k mkk4 mekk1 induc mkk4 activ report kinas subdomain play critic role mekk2 activ select point mutat subdomain impair mekk2 phosphoryl map2k mkk7 mek5 abolish mekk2 induc activ mapk jnk1 erk5 diminish mekk2 depend activ ap report gene interest spectrum mutat subdomain mekk2 affect activ overlap ident effect mekk1 thus mutat subdomain differenti affect mekk2 mekk1',\n",
       " 'titl mek kinas adaptor protein lad regul extracellular signal regul kinas activ epiderm growth factor via src abstract lad sh2 domain contain adaptor protein bind mek kinas mekk2 mitogen activ protein kinas mapk kinas kinas extracellular signal regul kinas erk5 jnk pathway lad mekk2 complex rest cell antisens knockdown lad express target gene disrupt mekk2 express result loss epiderm growth factor egf stress stimuli induc activ erk5 activ mekk2 erk5 pathway egf stress stimuli depend src kinas activ lad bind motif encod within amino acid terminus mekk2 express motif block lad mekk2 interact result inhibit src depend activ mekk2 erk5 jnk activ egf similar inhibit loss lad mekk2 express block interact mekk2 lad studi demonstr src kinas activ requir erk5 activ respons egf mekk2 express requir erk5 activ src lad mekk2 associ requir src activ erk5 egf src stimul erk5 regul mef2 depend promot activ requir function lad mekk2 signal complex',\n",
       " 'titl disrupt mekk2 mice reveal unexpect role mekk2 modul cell receptor signal transduct abstract mekk2 member mitogen activ protein kinas mapk kinas kinas gene famili involv regul multipl mapk signal pathway elucid vivo function mekk2 generat mice carri target mutat mekk2 locus mekk2 mice viabl fertil major subset thymic spleen cell mekk2 defici mice indistinguish wild type mice cell develop appear proceed similar bone marrow mekk2 defici wild type mice howev mekk2 cell prolifer augment respons anti cd3 monoclon antibodi mab stimul cell produc interleukin gamma interferon wild type cell suggest mekk2 may involv control strength cell receptor tcr signal consist mekk2 thymocyt suscept wild type thymocyt anti cd3 mab induc cell death furthermor tcr mediat jun termin kinas activ block moder enhanc mekk2 cell neither extracellular signal regul kinas p38 mapk activ affect mekk2 cell conclus found mekk2 may requir control strength tcr cd3 signal',\n",
       " 'titl rel ativ nf kappab cd28 costimul cell activ abstract costimulatori signal complement modifi signal provid lymphocyt antigen receptor product cell activ cd28 molecul appar import although costimulatori receptor cd28 provid signal least partial distinct deliv cell receptor tcr cd3 complex sever line evid indic nuclear factor nf kappab pathway perhap relev biochem transcript target costimulatori activ cd28 although mani question remain recent year wit signific progress understand signal transduct pathway lead tcr cd28 rel nf kappab depend transcript',\n",
       " 'titl role amino termin domain mekk activ nf kappa mapk pathway regul cell prolifer apoptosi abstract mitogen activ protein kinas mapk extracellular signal regul kinas erk kinas mek kinas mekk serin threonin kinas upstream regul mapk role amino termin termin domain mekk1 regul differ intracellular signal pathway apoptosi cell prolifer assess compar respons induc full length fl mekk induc kinas domain mekk pattern activ nf kappa erk mapk pathway jun termin kinas jnk mapk pathway mark differ kinas domain fl form similar cell prolifer apoptosi differ regul fl mekk correspond kinas domain data show termin domain mekk determin specif strength activ various intracellular signal pathway cellular respons',\n",
       " 'titl express galpha q226l induc p19 stem cell primit endoderm via mekk1 abstract galpha13 mediat abil morphogen retino acid promot primit endoderm format mous p19 embryon carcinoma stem cell process includ oblig activ jun termin kinas express constitut activ q226l gtpase defici form galpha13 mimic retino acid use investig signal upstream primit endoderm format jun termin kinas activ mek1 mkk4 mekk1 constitut activ clone stabli transfect express q226l galpha13 domin negat form mekk1 mekk4 express stabli clone harbor q226l galpha13 express domin negat version either mekk1 mekk4 effect block activ jun termin kinas well format primit endoderm deplet mekk1 antisens oligodeoxynucleotid suppress signal q226l galpha13 jnk1 primit endoderm format demonstr signal linkag map galpha13 activ primit endoderm format stem cell requir activ three level mitogen activ protein kinas cascad mekk1 map kinas kinas kinas mkk4 mek1 map kinas kinas jnk1 map kinas',\n",
       " 'titl vitro activ mekk2 mekk3 deterg function valin serin differ catalyt domain abstract mekk2 mekk3 mitogen activ protein kinas kinas kinas map3 kinas kda respect mark homolog kinas domain mekk2 mekk3 abl activ jun kinas pathway vivo howev follow routin immunoprecipit triton mekk2 mekk3 abl effect phosphoryl sek mek undergo autophosphoryl unexpect mekk2 mekk3 function vitro kinas assay cell solubil close relat deterg np given high homolog kinas set relat differenti sensit triton differ primari structur set chimer molecul generat loss activ triton map kinas domain iii specif serin mekk3 valin mekk2 residu immedi termin activ site lysin mutat serin mekk3 valin found mekk2 confer catalyt activ mekk3 triton wherea reciproc alter valin mekk2 serin confer lack activ triton mekk2 search protein databas identifi three kinas mekk3 pbs2p dd pki serin threonin site presenc serin threonin adjac activ site lysin protein kinas rare mekk3 result deterg instabl',\n",
       " 'titl mekk2 requir cell receptor signal jnk activ interleukin gene express abstract jun termin kinas jnks member mitogen activ protein kinas mapk gene famili essenti cell prolifer differenti apoptosi previous found activ jnk cell requir costimul cell receptor auxiliari receptor cd28 studi clone full length human mek kinas mekk cdna jurkat cell demonstr major upstream mapk kinas kinas jnk cascad cell human mekk2 cdna encod polypeptid amino acid human counterpart report murin mekk2 homolog human murin mekk3 catalyt domain homolog termin noncatalyt region northern blot analysi show mekk2 ubiquit express highest level peripher blood leukocyt cell mekk2 found strong activ jnk extracellular signal regul kinas mapk activ jnk depend ap report gene express mekk2 synerg anti cd3 antibodi activ jnk cell stimul cell led induct mekk2 tyrosin phosphoryl signific jnk activ induc anti cd3 anti cd28 antibodi tetradecanoylphorbol acet ionophor a23187 inhibit domin negat mekk2 mutant ap interleukin report gene induct cell inhibit domin negat mekk2 mutant taken togeth result show human mekk2 key signal molecul cell receptor cd3 mediat jnk mapk activ interleukin gene express',\n",
       " 'titl mekk1 activ human estrogen receptor alpha stimul agonist activ hydroxytamoxifen endometri ovarian cancer cell abstract estrogen mitogen stimul growth normal transform epitheli cell femal reproduct system effect estrogen mediat estrogen receptor ligand regul transcript factor tamoxifen select estrogen receptor modul function estrogen receptor antagonist breast agonist uterus current studi show coexpress constitut activ mekk1 raf mekk2 signific increas transcript activ receptor endometri ovarian cancer cell express wild type mekk1 activ rac1 function upstream mekk1 increas activ receptor coexpress domin negat mekk1 block rac1 induct indic endogen mekk1 capabl activ receptor addit experi demonstr mekk1 induc activ mediat jun termin kinas p38 hog1 independ known phosphoryl site receptor p38 jun termin kinas effici phosphoryl receptor immunocomplex kinas assay suggest differenti involv two kinas receptor activ import express constitut activ mekk1 increas agonist activ hydroxytamoxifen level compar estradiol fulli block antagonist activ find suggest uterin specif agonist activ tamoxifen compound may determin status kinas act downstream mekk1',\n",
       " 'titl mekk2 associ adapt protein lad ribp regul mek5 bmk1 erk5 pathway abstract mekk2 mekk3 two close relat mitogen activ protein kinas mapk kinas kinas kinas domain mekk2 mekk3 near ident although termin regulatori domain signific diverg yeast two hybrid librari screen identifi mek5 mapk kinas big mitogen activ protein kinas bmk1 erk5 pathway bind partner mekk2 mekk2 express stimul bmk1 erk5 activ downstream substrat mek5 compar mekk3 mekk2 activ bmk1 erk5 greater extent might correl higher affin mekk2 mek5 interact domin negat form mek5 block activ bmk1 erk5 mekk2 wherea activ jun termin kinas jnk unaffect show mek5 specif downstream effector mekk2 bmk1 erk5 pathway activ bmk1 erk5 epiderm growth factor h2o2 cos7 hek293 cell complet block kinas inact mekk3 mekk3kin wherea mekk2kin effect howev d10 cell express mekk2kin mekk3kin inhibit bmk1 erk5 activ two hybrid screen identifi lck associ adapt rlk itk bind protein lad ribp cell adapt protein bind partner mekk2 mekk2 lad ribp coloc cell contact site antigen load present cell demonstr cotransloc mekk2 lad ribp cell activ mekk3 neither bind lad ribp recruit cell contact antigen present cell mekk2 mekk3 differenti associ signal specif upstream receptor system wherea activ mek5 bmk1 erk5 pathway',\n",
       " 'titl mitogen activ protein kinas extracellular signal regul kinas pathway involv formyl methionyl leucyl phenylalanin induc p47phox phosphoryl human neutrophil abstract phosphoryl p47 phagocyt oxidas p47 phox one nadph oxidas compon essenti activ enzym superoxid product p47 phox phosphoryl multipl serin residu kinas involv process vivo remain character examin role extracellular signal regul kinas erk1 p38 mitogen activ protein kinas p47 phox phosphoryl inhibit erk1 activ pd98059 specif inhibitor erk kinas inhibit fmlp induc phosphoryl p47 phox howev pd98059 weak affect pma induc p47 phox phosphoryl even though erk1 activ abrog effect confirm use u0126 second erk kinas inhibitor unlik pd98059 u0126 p38 mitogen activ protein kinas inhibitor sb203580 inhibit phosphoryl p47 phox induc either fmlp pma two dimension phosphopeptid map analysi show fmlp induc p47 phox phosphoryl pd98059 affect phosphoryl major phosphopeptid suggest erk1 may regul p47 phox phosphoryl either direct indirect via kinas pma induc p47 phox phosphoryl gf109203x protein kinas inhibitor strong inhibit p47 phox phosphoryl howev fmlp induc p47 phox phosphoryl pd98059 gf109203x partial inhibit phosphoryl p47 phox test alon exert addit inhibitori effect p47 phox phosphoryl test togeth result show first time erk1 pathway particip phosphoryl p47 phox furthermor strong suggest p47 phox target sever kinas cascad intact neutrophil activ fmlp therefor converg point erk1 protein kinas',\n",
       " 'titl mekk2 gene disrupt caus loss cytokin product respons ige kit ligand stimul cell deriv mast cell abstract ligat high affin ige receptor fcepsilonri kit stimul cytokin product mast cell show mek kinas mekk2 mapk kinas kinas map3k regul jnk erk5 pathway requir cytokin product embryon stem cell deriv mast cell esmc target disrupt mekk2 mekk1 gene use abolish express respect kinas esmc transcript specif cytokin respons ige kit ligand mark reduc mekk2 esmc relat wild type esmc cytokin product mekk1 esmc similar wild type esmc demonstr specif mekk2 signal cytokin gene regul mekk2 esmc lost receptor mediat stimul jnk contrast jnk activ respons uv irradi normal show mekk2 requir receptor signal cellular stress respons mekk2 first map3k shown requir mast cell tyrosin kinas receptor signal control cytokin gene express',\n",
       " 'titl mek kinas bind activ protein kinas relat kinas bifurc kinas regulatori pathway level mapk kinas kinas abstract mek kinas mekk2 kda protein serin threonin kinas shown function mitogen activ protein kinas mapk kinas kinas mekk2 kinas domain cooh termin moieti protein nh termin moieti mekk2 signatur motif would suggest defin regulatori function yeast two hybrid screen perform identifi protein bind mekk2 protein kinas relat kinas prk2 found bind mekk2 prk2 previous shown bind rhoa src homolog domain nck prk2 bind mekk3 close relat mekk2 mekk2 bind site map amino acid prk2 distinct bind site rhoa nck sequenc diverg close relat kinas prk1 bind mekk2 cell mekk2 prk2 co immunoprecipit prk2 activ mekk2 similar purifi recombin mekk2 activ prk2 vitro mekk2 activ prk2 independ mekk2 regul jun nh termin kinas pathway mekk2 activ prk2 result bifurc signal dual control mapk pathway prk2 regul respons',\n",
       " 'titl synergist interact mek kinas jun termin kinas jnk kinas jnk1 result effici specif jnk1 activ abstract mitogen activ protein kinas mapk activ cascad modul consist mapk mapk kinas mapkk mapkk kinas mapkkk investig molecular basi activ jun termin kinas jnk subgroup mapk mapkkk mekk2 found strong specif jnk1 activ mekk2 mediat mapkk jnk kinas jnkk2 rather jnkk1 format tripartit complex consist mekk2 jnkk2 jnk1 scaffold protein requir mekk2 jnkk2 jnk1 tripartit complex format express jnk1 jnkk2 mekk2 signific augment coprecipit respect mekk2 jnkk2 mekk2 jnk1 jnkk2 jnk1 indic interact mekk2 jnkk2 jnk1 synergist final jnk1 activ effici mekk2 jnkk2 jnk1 complex jnk1 exclud complex thus format signal complex synergist interact mapkkk mapkk mapk molecul like mekk2 jnkk2 jnk1 like respons effici specif flow inform via mapk cascad',\n",
       " 'titl hepat epiderm growth factor regul mitogen activ protein kinas kinas kinas activ rat lack ident known form raf mekk abstract mitogen signal involv protein kinas phosphoryl mitogen activ protein kinas mapk activ mek rat basal hepat mek kinas activ low vivo adult rat late gestat fetal rat mark stimul intraperiton administr epiderm growth factor egf level stimul mek phosphoryl activ approxim time higher fetal liver adult liver identifi regul form two categori mek kinas raf mekk western immunoblot immunoprecipit kinas assay immunodeplet studi perform western immunoblot confirm raf raf raf mekk1 mekk2 present similar level e19 adult liver howev specif immunoprecipit kinas assay detect kinas could account mark egf sensit higher level activ e19 fetus immunodeplet studi produc mark reduct immunoreact raf mekk content activ minim decreas abil chromatographi fraction phosphoryl activ recombin mek result indic hepat egf sensit mek kinas activ may resid previous unidentifi physiolog relev form raf mekk',\n",
       " 'titl signal transduct pathway regul arsenat arsenit abstract arsenat arsenit activ jun termin kinas jnk howev mechan occur known express inhibitori mutant small gtp bind protein p21 activ kinas mitogen activ protein kinas extracellular signal regul kinas kinas kinas mekk identifi specif protein involv arsenat arsenit mediat activ jnk observ distinct differ arsenat arsenit signal demonstr arsenat arsenit capabl activ uniqu protein arsenat arsenit activ jnk requir rac rho neither arsenat arsenit signal inhibit domin negat mutant cdc42 ras arsenit stimul jnk requir wherea arsenat mediat activ jnk unaffect inhibitori mutant four mekk test mekk3 mekk4 involv arsenat mediat activ jnk contrast arsenit mediat jnk activ requir mekk2 mekk3 mekk4 result better defin mechan arsenat arsenit activ jnk demonstr differ regul signal transduct pathway inorgan arsenic speci',\n",
       " 'titl live cell fluoresc imag cell mekk2 redistribut activ respons antigen stimul cell receptor abstract cell activ requir engag cell receptor tcr interfac conjug form antigen present cell tcr engag accompani redistribut specif signal molecul cytoplasm region tcr complex studi immunocytochemistri live cell fluoresc imag demonstr cell mek kinas mekk2 transloc cell antigen present cell interfac respons antigen activ mekk2 transloc occur rapid antigen concentr increas biochem activ mekk2 follow tcr stimul express domin negat mekk2 inhibit tcr mediat conjug stabil erk p38 map kinas phosphoryl live cell fluoresc imag thus enabl character signal transduc dynam transloc follow tcr engag']"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tokenized_MAP3K2_docstrings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 259,
   "metadata": {},
   "outputs": [],
   "source": [
    "#try LDA topic modeling after code (at the end of this script) was inconclusive. \n",
    "#topic modeling on a docstring, say MAP3K 5 and 15, find 2 topics\n",
    "#let's try to find some putative functions of MAP3K4 by looking at MAP3K5, 6, 15 (phylotree)\n",
    "\n",
    "#MAP4k4 does the following:\n",
    "\n",
    "#modifies chromatin (HDAC6), gadd45a interaction, increased IL21 (inflammation),\n",
    "#extracellular acidification, cell migration, t-associated sex reversal,\n",
    "#TGF-induced phosphorylation and activation of p38 \n",
    "#regulates testes through controling p38-mediated control of sry\n",
    "#less IL1A means less MAP3k4 in crohn's disease\n",
    "#acetylation, JNK phosphorylation, MAPK signaling? look at paper Bogani et al. \n",
    "#activates jnk\n",
    "#wnt-jnk pathway\n",
    "#SO, mostly p38 and JNK pathways\n",
    "\n",
    "\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.decomposition import LatentDirichletAllocation\n",
    "\n",
    "combined_MAP3K_5_6_15=tokenized_MAP3K5_docstrings+tokenized_MAP3K15_docstrings + tokenized_MAP3K6_docstrings\n",
    "cv1 = CountVectorizer(stop_words = 'english', min_df=.05) #if occurs in 5% of text\n",
    "cv2 = CountVectorizer(stop_words = 'english', min_df=.02) #if occurs in 2% of text\n",
    "cv3 = CountVectorizer(stop_words = 'english', min_df=2) #if occurs in at least 2 texts\n",
    "cv4 = CountVectorizer(stop_words = 'english', min_df=4) #if occurs in at least 3 texts\n",
    "\n",
    "X1 = cv1.fit_transform(combined_MAP3K_5_6_15)\n",
    "X2 = cv2.fit_transform(combined_MAP3K_5_6_15)\n",
    "X3 = cv3.fit_transform(combined_MAP3K_5_6_15)\n",
    "X4 = cv4.fit_transform(combined_MAP3K_5_6_15)\n",
    "\n",
    "def display_topics(model, feature_names, no_top_words):\n",
    "    for topic_idx, topic in enumerate(model.components_):\n",
    "        print (\"Topic %d:\" % (topic_idx))\n",
    "        print (\" \".join([feature_names[i]\n",
    "                        for i in topic.argsort()[:-no_top_words - 1:-1]]))\n",
    "                \n",
    "no_top_words = 50"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "metadata": {},
   "outputs": [],
   "source": [
    "#3 diff docstrings #n_topics is now n_components\n",
    "\n",
    "lda_1=LatentDirichletAllocation(n_components=2,  #because when we use 3 topics, MAP3K15 has too few publications\n",
    "                                max_iter=5, \n",
    "                                learning_method='online', \n",
    "                                learning_offset=50., \n",
    "                                random_state=0).fit(X1)\n",
    "lda_2=LatentDirichletAllocation(n_components=3, \n",
    "                                max_iter=5, \n",
    "                                learning_method='online', \n",
    "                                learning_offset=50., \n",
    "                                random_state=0).fit(X2)\n",
    "lda_3=LatentDirichletAllocation(n_components=3, \n",
    "                                learning_method='online', \n",
    "                                learning_offset=50., \n",
    "                                random_state=0).fit(X3)\n",
    "lda_4=LatentDirichletAllocation(n_components=3, \n",
    "                                max_iter=5, \n",
    "                                learning_method='online', \n",
    "                                learning_offset=50., \n",
    "                                random_state=0).fit(X4)\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 287,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(534, 318)\n",
      "(534, 693)\n",
      "(534, 2683)\n",
      "(534, 1515)\n"
     ]
    }
   ],
   "source": [
    "#documents, words\n",
    "\n",
    "print(X1.shape)\n",
    "print(X2.shape)\n",
    "print(X3.shape)\n",
    "print(X4.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 288,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 0:\n",
      "cell gene activ induc express apoptosi cancer kinas pathway signal death abstract titl studi regul human protein use ask1 respons stress inhibit identifi effect mediat result jnk ros mutat depend involv associ tumor mechan p38 increas treatment develop suggest factor signific genet level target novel caspas apoptot caus analysi relat patient genom addit reveal potenti role line inhibitor map3k5 receptor transcript includ dna famili diseas chang control alter investig molecular time growth base mice function ask assay acid indic speci mapk model rna differenti mitochondri data howev provid decreas protect new phosphoryl import observ reactiv promot mitogen screen remain specif anti oxygen trigger report type generat test sirna high affect exposur tissu demonstr reduc enhanc suppress normal detect molecul sensit recent compar oxid product process cytotox implic determin modul surviv present neuron larg moreov evid format known lead biolog releas select carcinoma knockdown unknown prolifer treat overexpress cellular multipl possibl sever contribut form clinic induct primari brain trx exhibit cycl common confirm infect similar therapeut examin deriv perform wide shown redox attenu vitro like togeth condit agent transfect compon therapi critic earli mrna cultur damag follow immun drug expos differ marker jun region character event furthermor previous respect small resist\n",
      "Topic 1:\n",
      "ask1 activ kinas signal induc apoptosi protein regul cell stress p38 inhibit phosphoryl pathway mice jnk titl abstract express mediat mapk oxid respons role interact increas result tnf function bind depend associ factor termin mechan jun mitogen suggest complex effect studi thioredoxin type trx inhibitor demonstr overexpress cascad stimul report cardiac play diseas death cellular enhanc signific reduc howev level promot suppress use treatment wild target identifi neuron involv defici novel domain human import endotheli specif vivo direct receptor injuri attenu requir form lead negat decreas ask alpha vitro indic caus furthermor product dissoci member necrosi ros site protect prevent erk format famili tumor block includ differenti mutant jnk1 beta downstream oxygen map high mous endogen macrophag upstream apoptot data compar molecular addit reactiv investig knockdown caspas model acid cytokin inflammatori growth induct provid various observ reveal previous impair speci link analysi potenti surviv transcript control like redox assay modul serin liver antioxid remain manner present shown intracellular known moreov knockout key develop subsequ determin critic brain extracellular stimuli molecul transfect initi tissu immun process peroxid examin therapeut mitochondri treat resist threonin metabol progress togeth follow essenti nuclear wherea differ generat sensit character transloc sever independ inflamm regulatori pathogenesi select\n"
     ]
    }
   ],
   "source": [
    "display_topics(lda_1, cv1.get_feature_names(), no_top_words) \n",
    "#topic 0 is map3k5, topic 2 is ask1 which is also map3k5, and topic 2 is also ask1... \n",
    "#basically tells us what ASK1 does"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 291,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "136\n"
     ]
    }
   ],
   "source": [
    "#finding similarity between topics:\n",
    "no_top_words = 200 #hypothesis is that there is not enough differentiation between topics. see whether coherence is poor. \n",
    "features = cv1.get_feature_names()  #basically IDs \n",
    "top_200_by_topic = dict()\n",
    "for topic_idx, topic in enumerate(lda_1.components_):\n",
    "    terms = set([features[i] for i in topic.argsort()[:-no_top_words - 1:-1]])\n",
    "    top_200_by_topic[topic_idx] = terms\n",
    "    \n",
    "    \n",
    "s = top_200_by_topic[0]\n",
    "for i in range(2):\n",
    "    t = top_200_by_topic[i]\n",
    "    ins = s.intersection(t) #measure intersection of number 0 to the other 6. \n",
    "    print(len(ins))\n",
    "    \n",
    "#so there's a high similarity between the topics. that's a good thing--those will be high confidence put. functs.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 236,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 0:\n",
      "ask1 gene kinas mice express signal respons protein regul use titl abstract studi identifi activ map3k6 pathway develop mutat stress associ apoptosi cell induc genet type role function target diseas cancer human involv genom analysi signific novel reveal cardiac patient result wild model includ factor variant gastric high case relat\n",
      "Topic 1:\n",
      "cell induc apoptosi express ask1 activ cancer regul ros p38 pathway kinas inhibit effect increas signal titl abstract stress protein death level human treatment result studi depend reactiv oxygen jnk speci oxid mediat respons ask trx tumor gastric product gene signific mitochondri mechan use inhibitor mapk suggest apoptot suppress reduc\n",
      "Topic 2:\n",
      "activ ask1 kinas signal induc cell protein apoptosi regul pathway jnk phosphoryl inhibit stress mediat p38 abstract titl death mapk depend interact oxid jun termin mechan factor tnf result role suggest express bind respons associ function mitogen cascad caspas complex studi increas neuron effect apoptot stimul inhibitor human thioredoxin overexpress\n"
     ]
    }
   ],
   "source": [
    "display_topics(lda_2, cv2.get_feature_names(), no_top_words)\n",
    "#topic 0 is ask1 (map3k5) and map3k6, topic 1 is ask1 (MAP3K5),\n",
    "#topic 2 is map3k5, "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 280,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "109\n",
      "98\n"
     ]
    }
   ],
   "source": [
    "no_top_words = 200 #hypothesis is that there is not enough differentiation between topics. see whether coherence is poor. \n",
    "features = cv2.get_feature_names()  #basically IDs \n",
    "top_200_by_topic = dict()\n",
    "for topic_idx, topic in enumerate(lda_2.components_):\n",
    "    terms = set([features[i] for i in topic.argsort()[:-no_top_words - 1:-1]])\n",
    "    top_200_by_topic[topic_idx] = terms\n",
    "    \n",
    "    \n",
    "s = top_200_by_topic[0]\n",
    "for i in range(3):\n",
    "    t = top_200_by_topic[i]\n",
    "    ins = s.intersection(t) #measure intersection of number 0 to the other 6. \n",
    "    print(len(ins))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 237,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 0:\n",
      "gene cancer gastric mutat use map3k6 identifi genet develop cdh1 genom variant autophagi germlin associ case titl patient abstract hdgc express human map3k5 studi gc novel sequenc base caus relat screen candid model respons ctnna1 cell assay analysi test includ new potenti alter variat hereditari syndrom pathway kinas famili diffus\n",
      "Topic 1:\n",
      "ask1 activ cell kinas induc apoptosi signal regul protein express pathway stress inhibit p38 jnk titl abstract mediat mice death respons phosphoryl mapk increas studi result effect depend mechan role oxid factor suggest human gene function ros associ tnf inhibitor involv level treatment mitogen signific jun type use cancer tumor\n",
      "Topic 2:\n",
      "ask1 kinas protein trx1 signal activ regul phosphoryl interact ask3 stress mapkkk5 complex pathway respons bind cell osmot apoptosi dynam arrestin mpk38 identifi abstract titl spak signalosom osr1 oxid pdk1 mapkkk6 use report beta demonstr mediat electrophil strap associ scaffold trx2 site ask2 hne ask tgf stoichiometri depend suggest age\n"
     ]
    }
   ],
   "source": [
    "display_topics(lda_3,cv3.get_feature_names(),no_top_words)  #this is the one i want. \n",
    "#topics 1 and 2 seem same with ask1 involved. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 278,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "70\n",
      "41\n"
     ]
    }
   ],
   "source": [
    "no_top_words = 200 #hypothesis is that there is not enough differentiation between topics. see whether coherence is poor. \n",
    "features = cv3.get_feature_names()  #basically IDs \n",
    "top_200_by_topic = dict()\n",
    "for topic_idx, topic in enumerate(lda_3.components_):\n",
    "    terms = set([features[i] for i in topic.argsort()[:-no_top_words - 1:-1]])\n",
    "    top_200_by_topic[topic_idx] = terms\n",
    "    \n",
    "    \n",
    "s = top_200_by_topic[0]\n",
    "for i in range(3):\n",
    "    t = top_200_by_topic[i]\n",
    "    ins = s.intersection(t) #measure intersection of number 0 to the other 6. \n",
    "    print(len(ins))\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Topic 0:\n",
      "ask1 activ cell kinas induc apoptosi signal regul protein express pathway stress inhibit abstract titl p38 jnk mediat respons phosphoryl death mapk studi result gene depend mice increas effect mechan role oxid factor human suggest associ use interact function ros inhibitor tnf cancer mitogen level involv treatment jun bind termin\n",
      "Topic 1:\n",
      "mice cardiac ask1 express protein hypertrophi map3k5 mpk38 apoptosi heart activ signific pdk1 kinas increas signal diabet regul hiv associ cell type compar induc tissu platelet ventricular titl function defici inhibit posit effect obes remodel impair abstract trx1 left diet wt treatment tgf cardiomyocyt attenu model junction pathway disrupt fat\n",
      "Topic 2:\n",
      "gene cancer map3k6 gastric mutat genet identifi cdh1 germlin variant use case hdgc genom patient develop autophagi abstract titl studi associ kinas sequenc caus screen base ctnna1 hereditari candid alter novel respons test syndrom relat variat diffus famili human chang express includ tumor model assay dna new risk addit rna\n"
     ]
    }
   ],
   "source": [
    "display_topics(lda_4,cv4.get_feature_names(),no_top_words) \n",
    "#map3k15 is ask3, map3k5 is ask1, map3k6 is topic 2. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 284,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "87\n",
      "71\n"
     ]
    }
   ],
   "source": [
    "no_top_words = 200 #hypothesis is that there is not enough differentiation between topics. see whether coherence is poor. \n",
    "features = cv4.get_feature_names()  #basically IDs \n",
    "top_200_by_topic = dict()\n",
    "for topic_idx, topic in enumerate(lda_4.components_):\n",
    "    terms = set([features[i] for i in topic.argsort()[:-no_top_words - 1:-1]])\n",
    "    top_200_by_topic[topic_idx] = terms\n",
    "    \n",
    "    \n",
    "s = top_200_by_topic[0]\n",
    "for i in range(3):\n",
    "    t = top_200_by_topic[i]\n",
    "    ins = s.intersection(t) #measure intersection of number 0 to the other 6. \n",
    "    print(len(ins))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 297,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "137\n"
     ]
    }
   ],
   "source": [
    "#let's look at MAP3K2 and MAP3K3 now!\n",
    "\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "from sklearn.decomposition import LatentDirichletAllocation\n",
    "\n",
    "combined_MAP3K_2_3= tokenized_MAP3K2_docstrings + tokenized_MAP3K3_docstrings\n",
    "\n",
    "cv_MAP3K_2_3 = CountVectorizer(stop_words = 'english', min_df=.05) #if occurs in 5% of text\n",
    "\n",
    "X1_MAP3K_2_3 = cv_MAP3K_2_3.fit_transform(combined_MAP3K_2_3)\n",
    "\n",
    "\n",
    "def display_topics(model, feature_names, no_top_words):\n",
    "    for topic_idx, topic in enumerate(model.components_):\n",
    "        print (\"Topic %d:\" % (topic_idx))\n",
    "        print (\" \".join([feature_names[i]\n",
    "                        for i in topic.argsort()[:-no_top_words - 1:-1]]))\n",
    "                \n",
    "no_top_words = 50\n",
    "\n",
    "lda_MAP3K_2_3=LatentDirichletAllocation(n_components=2,  #because when we use 3 topics, MAP3K15 has too few publications\n",
    "                                max_iter=5, \n",
    "                                learning_method='online', \n",
    "                                learning_offset=50., \n",
    "                                random_state=0).fit(X1_MAP3K_2_3)\n",
    "\n",
    "no_top_words = 200 #hypothesis is that there is not enough differentiation between topics. see whether coherence is poor. \n",
    "features = cv_MAP3K_2_3.get_feature_names()  #basically IDs \n",
    "top_200_by_topic = dict()\n",
    "for topic_idx, topic in enumerate(lda_MAP3K_2_3.components_):\n",
    "    terms = set([features[i] for i in topic.argsort()[:-no_top_words - 1:-1]])\n",
    "    top_200_by_topic[topic_idx] = terms\n",
    "    \n",
    "    \n",
    "s = top_200_by_topic[0]\n",
    "for i in range(2):\n",
    "    t = top_200_by_topic[i]\n",
    "    ins = s.intersection(t) #measure intersection of number 0 to the other 6. \n",
    "    print(len(ins))\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 298,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abstract',\n",
       " 'acid',\n",
       " 'activ',\n",
       " 'alter',\n",
       " 'analysi',\n",
       " 'apoptosi',\n",
       " 'assay',\n",
       " 'associ',\n",
       " 'bind',\n",
       " 'block',\n",
       " 'breast',\n",
       " 'cancer',\n",
       " 'cascad',\n",
       " 'cell',\n",
       " 'cellular',\n",
       " 'clinic',\n",
       " 'compar',\n",
       " 'complex',\n",
       " 'confirm',\n",
       " 'contribut',\n",
       " 'control',\n",
       " 'correl',\n",
       " 'critic',\n",
       " 'cytokin',\n",
       " 'data',\n",
       " 'decreas',\n",
       " 'demonstr',\n",
       " 'depend',\n",
       " 'determin',\n",
       " 'develop',\n",
       " 'differ',\n",
       " 'differenti',\n",
       " 'domin',\n",
       " 'effect',\n",
       " 'enhanc',\n",
       " 'erk',\n",
       " 'express',\n",
       " 'extracellular',\n",
       " 'factor',\n",
       " 'famili',\n",
       " 'form',\n",
       " 'format',\n",
       " 'function',\n",
       " 'furthermor',\n",
       " 'gene',\n",
       " 'growth',\n",
       " 'high',\n",
       " 'howev',\n",
       " 'human',\n",
       " 'identifi',\n",
       " 'import',\n",
       " 'includ',\n",
       " 'increas',\n",
       " 'indic',\n",
       " 'induc',\n",
       " 'inhibit',\n",
       " 'inhibitor',\n",
       " 'interact',\n",
       " 'investig',\n",
       " 'involv',\n",
       " 'jnk',\n",
       " 'jun',\n",
       " 'kinas',\n",
       " 'known',\n",
       " 'lead',\n",
       " 'level',\n",
       " 'map',\n",
       " 'map3k',\n",
       " 'map3k2',\n",
       " 'map3k3',\n",
       " 'mapk',\n",
       " 'mechan',\n",
       " 'mediat',\n",
       " 'mekk1',\n",
       " 'mekk2',\n",
       " 'mekk3',\n",
       " 'member',\n",
       " 'mice',\n",
       " 'migrat',\n",
       " 'mitogen',\n",
       " 'model',\n",
       " 'modul',\n",
       " 'molecul',\n",
       " 'molecular',\n",
       " 'mous',\n",
       " 'mutant',\n",
       " 'mutat',\n",
       " 'negat',\n",
       " 'novel',\n",
       " 'observ',\n",
       " 'overexpress',\n",
       " 'pathway',\n",
       " 'patient',\n",
       " 'phosphoryl',\n",
       " 'play',\n",
       " 'potenti',\n",
       " 'previous',\n",
       " 'profil',\n",
       " 'prolifer',\n",
       " 'promot',\n",
       " 'protein',\n",
       " 'provid',\n",
       " 'receptor',\n",
       " 'reduc',\n",
       " 'regul',\n",
       " 'relat',\n",
       " 'report',\n",
       " 'requir',\n",
       " 'respons',\n",
       " 'result',\n",
       " 'reveal',\n",
       " 'role',\n",
       " 'serin',\n",
       " 'shown',\n",
       " 'signal',\n",
       " 'signific',\n",
       " 'small',\n",
       " 'specif',\n",
       " 'stimul',\n",
       " 'stress',\n",
       " 'studi',\n",
       " 'suggest',\n",
       " 'suppress',\n",
       " 'taken',\n",
       " 'target',\n",
       " 'termin',\n",
       " 'tissu',\n",
       " 'titl',\n",
       " 'togeth',\n",
       " 'transcript',\n",
       " 'tumor',\n",
       " 'type',\n",
       " 'ubiquitin',\n",
       " 'upstream',\n",
       " 'use',\n",
       " 'vitro',\n",
       " 'wherea'}"
      ]
     },
     "execution_count": 298,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ins"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting matplotlib-venn\n",
      "  Downloading https://files.pythonhosted.org/packages/c7/05/e084c8331ff7ab8b0e01c7cdb7c18854852340bf3096193510c902ffa1f1/matplotlib-venn-0.11.5.zip (40kB)\n",
      "\u001b[K    100% |████████████████████████████████| 40kB 1.2MB/s ta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: matplotlib in /anaconda3/lib/python3.6/site-packages (from matplotlib-venn)\n",
      "Requirement already satisfied: numpy in /anaconda3/lib/python3.6/site-packages (from matplotlib-venn)\n",
      "Requirement already satisfied: scipy in /anaconda3/lib/python3.6/site-packages (from matplotlib-venn)\n",
      "Requirement already satisfied: six>=1.10 in /anaconda3/lib/python3.6/site-packages (from matplotlib->matplotlib-venn)\n",
      "Requirement already satisfied: python-dateutil>=2.0 in /anaconda3/lib/python3.6/site-packages (from matplotlib->matplotlib-venn)\n",
      "Requirement already satisfied: pytz in /anaconda3/lib/python3.6/site-packages (from matplotlib->matplotlib-venn)\n",
      "Requirement already satisfied: cycler>=0.10 in /anaconda3/lib/python3.6/site-packages (from matplotlib->matplotlib-venn)\n",
      "Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /anaconda3/lib/python3.6/site-packages (from matplotlib->matplotlib-venn)\n",
      "Building wheels for collected packages: matplotlib-venn\n",
      "  Running setup.py bdist_wheel for matplotlib-venn ... \u001b[?25ldone\n",
      "\u001b[?25h  Stored in directory: /Users/liangrj/Library/Caches/pip/wheels/d8/88/2f/e41518cb16e58109e8d14dc5dd7ef8166c82dce86c7c6d6c80\n",
      "Successfully built matplotlib-venn\n",
      "Installing collected packages: matplotlib-venn\n",
      "Successfully installed matplotlib-venn-0.11.5\n",
      "\u001b[33mYou are using pip version 9.0.3, however version 10.0.1 is available.\n",
      "You should consider upgrading via the 'pip install --upgrade pip' command.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!pip install matplotlib-venn #graphically looking at this shit. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Note... I just realized that I should have decreased my corpus size into ONLY the gene function papers... "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [],
   "source": [
    "#take a step back, let's make list of list of single stemmed words rather than list\n",
    "#of single strings. This will allow us to visualize bag of words approach! \n",
    "\n",
    "from nltk.tokenize import RegexpTokenizer\n",
    "import nltk\n",
    "import string\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "combined_docstring_list=[]\n",
    "for doc in tokenized_docstring_list:\n",
    "    combined_docstring_list.append(\" \".join(doc))\n",
    "\n",
    "    \n",
    "MAP3Kwords_list=[] #MAP3K2, 3, 5, 6, 15. len of list is 5. each item is list of words\n",
    "for strings in combined_docstring_list:\n",
    "    MAP3Kwords_list.append(strings.split())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 276,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intersect MAP3K2 and 3: 1289\n",
      "intersect MAP3K2 and 3: 1289\n",
      "intersect MAP3K5 and 6: 635\n",
      "intersect MAP3K5 and 15: 405\n",
      "intersect MAP3K6 and 15: 210\n",
      "intersect MAP3K5, 6, and 15: 209\n"
     ]
    }
   ],
   "source": [
    "#finding intersections between MAP3K2 and MAP3K3 lists:\n",
    "print(\"intersect MAP3K2 and 3: \" + \n",
    "      str(len(set(MAP3Kwords_list[0]) & set(MAP3Kwords_list[1])))) \n",
    "print(\"intersect MAP3K2 and 3: \" + \n",
    "      str(len(set(MAP3Kwords_list[0]) & set(MAP3Kwords_list[1]))))\n",
    "\n",
    "\n",
    "print(\"intersect MAP3K5 and 6: \" + \n",
    "      str(len(set(MAP3Kwords_list[2]) & set(MAP3Kwords_list[3])))) \n",
    "print(\"intersect MAP3K5 and 15: \" + \n",
    "      str(len(set(MAP3Kwords_list[2]) & set(MAP3Kwords_list[4]))))\n",
    "print(\"intersect MAP3K6 and 15: \" + \n",
    "      str(len(set(MAP3Kwords_list[3]) & set(MAP3Kwords_list[4])))) \n",
    "\n",
    "#intersect MAP3K5, 6, and 15\n",
    "print(\"intersect MAP3K5, 6, and 15: \" + \n",
    "      str(len(set(MAP3Kwords_list[2]) & set(MAP3Kwords_list[3]) & set(MAP3Kwords_list[4]))))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAQgAAAD7CAYAAACWhwr8AAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMS4wLCBo\ndHRwOi8vbWF0cGxvdGxpYi5vcmcvpW3flQAAIABJREFUeJztnXmYXFW1t99VXT3Pnc7QmQgZCSET\nYAYIM0TCYFC4IJOAoqBX5aqXqwJ+IYCoiHoVEScwoAEFhcsY0BBCgoQQyETmhMxDJ52k56HG9f2x\nT4dKp6u7utNdp6p6v89TT6rOuE76nN9Ze+291xJVxWKxWFrD47YBFoslcbECYbFYomIFwmKxRMUK\nhMViiYoVCIvFEhUrEBaLJSpWICwpi4isFZFz3bYjmUlqgRCR7SLiF5HSFstXioiKyJAWy+91lk9q\nsfxmEQmJSJ2I1Dj7X+asO1lEPhCRSuczX0ROjth3jog8EPF7jIjsE5HvOL8XikiTc+w6EdnYwWu8\nUESWi0i9iOwSkatj3O8UEXlDRA6KyDGDXY7XrmRAVceo6kK37UhmklogHLYB1zb/EJGxQHbLjURE\ngBuBw8BNrRxniarmAUXA48CzIlIC7AWuAkqAUuAl4K+tGSIiE4C3gB+q6s8iVn1dVfOcz6hYL8wR\noqeBu4FCYALwYYy7B4BngS+1sU2n7Ep0RMTrtg2pQioIxJ+BL0T8vgl4qpXtzgL6A3cAnxeRjNYO\npqph4AmMyAxV1SpV3a5myKkAIWB4y/0cr2Q+cJeq/vo4rieSe4Dfqeo8VQ2q6iFV/TiWHVV1o6o+\nDqw9XiNEJE1E7hKRj0WkVkQ+FJFBzrozRGSZiFQ7/54Rsd9CEXlARN51vJSXRaSXiMx1PLVlkV6e\n4919U0S2Op7PT0XE46wbJiILROSQs26uiBRF7LtdRL4rIquBehHxOssudNZPcjzBGhHZLyI/j9j3\nM05zpMqxeXSL4/63iKx2rvFvIpJ1vP+nSYOqJu0H2A5cCGwERgNpwC7gBECBIRHbPo55o6YDh4DP\nRay7GXjH+e7FiEgtUBixTRUQBMLAPRHL5wD/xHgmN7Zi40KgAjgI/Bs4twPXtxW4H/gI2Af8BSjp\n4P/RcPNn7rxdwJ2ODaMwIjke6IXxqioxnpkX48lVAr0izrEFGIbxgNYBm5y/mRcj5H+KOI9iPLAS\nYLCz7a0R13ERkAn0BhYB/9viXlgJDAKyI+8P5/uS5r8PkAdMcb6PBOqdY6cD/+PYnBFxjPcxL5cS\nYD1wu9v3ftyeMbcNOC7jPxGIe4AfARcD/3JuviMCAeQANcAVzu/fAS9GHOdm5+Gvch6Y95pvrBbn\nywW+BlwasWyOc+xtQGkr+0wG8p0b+yaM8AyL8fr8zjWOdG7qfwBzO/h/FE0gYrYLI8AzW1l+I/B+\ni2VLgJud7wuBuyPW/QyYF/H7cmBlxG8FLo74/TXgzSg2XQGsaHEvfLG1+8P5vgiY3fJvBPwAeDbi\ntwfYgyOYzjFuiFj/EPBbt+/9eH1SoYkBpplxHeZBb6158VmMALzm/J4LzBCR3hHbvKeqRapaqqpT\nVHV+y4Ooaj3wW+ApEekTsepRYBnwLxEpbrHPUlWtVVWfqj6JeVtfEuN1NWLesJtUtQ54sAP7tkkH\n7RoEtNa06Q/saLFsBzAg4vf+iO+NrfzOa7H/rhbH6g8gIn1E5K8iskdEajDeVGkb+7bkSxih3eA0\nbS5r7RrUNDF3tbiG8ojvDa3YnLKkhECo6g7MG/wS4PlWNrkJ80fdKSLlwHMYd/LaVrZtDw/GI4m8\ngULA9cBO4A0RKWjLXIybHgurne3jQVt27cI0E1qyF9Oci2Qw5g3cWQa1ONZe5/uPHBvHqWoBcAPH\n2hv1/0pVN6vqtUAf4CfA30UklxbX4ASzBx3nNaQMKSEQDl8Cznfe8kcQkQHABcBlmF6ACZg29E9o\nvTfjKETkIhGZ6ATqCoCfY9rZ6yO3U9UA8B+YJsprIpIrIkUi8mkRyXKCZtcDZwNvOMceIq10x0bw\nJ+AWERkqIjnAd4FXImzbLiI3R7FbnGBahvM7S0Qyne9t2tUKfwTuF5ERznHHiUgvjEc2UkSuc45z\nDXBypI2d4E4RKXaCoHcAf3OW5wN1QJXzN72zIwcVkRtEpLfjIVQ5i0OYuNSlInKBiKQD3wF8wLvH\ncQ0pQ8p0B2n06P6NmHbuPyMXisivgO+IyCntHLoIeAQYiHGJl2HayU2t2OAXkc9hHpCXMQL0AHAS\n5mbcgImDNI85GIRxb1t9W6nqEyJyArDUWfQ68E3H/gxMoPC9KHafgPGqmml0zjUE4z21ZVdLfo6J\nVfwT49ZvAD6rqrsdV/2XwGOY4N5lqnowynFi4UVMV24hJr7zuLN8Nqb5WO2c58/Atzpw3IuBnztC\nuwP4vPM33CgiN2D+xgMwgc7LVdV/HNeQMogTeLG4gIjcA1So6u86se804D8dtzklEDOga4SqbnHb\nFovBCoQlYbACkXikUgzCYrF0MdaDsFgsUbEehMViiYoVCIvFEhUrEBaLJSpWICwWS1SsQFgslqhY\ngbBYLFGxAmGxWKJiBcJisUTFCoTFYolKjxEIMZmr33HbDoslmUgpgRCRt0SkwklMukpEZsa437ki\nEpZPUsDXiUi7uSIsllQnZfJBONwBrFPVoIhMBuaLyEhV3RfDvntVdWA322exJBVx8SCczEd3OqnD\n60XkcRHpKyLzxKRRn9+cy1FEnhORcifF+CIRGRNxnDki8lsR+Zez39tOQhUAVHW1qgabf2ISowyi\nFcSkVH9HRAq78dItlqQmnk2MKzGpxUdishnPA+7CZCjy4GRKcpaPwOQOXI5JMBvJ9ZhU8KWY7D9H\nrReRV0SkCZOFaSHwQYv1HhH5AzAOmK6q1c6qPmLqJWwTkV84+Qotlh5NPJsYj6jqfgARWQwcUNUV\nzu8XMHkjUdUnmncQkXuBShEpjHiQX1XVRc76u4FqERmkqruc/S9zcgteCJzk5CBsJh14BnPdkWnF\nNmByVW7ApGp7EpNm7bYu/j+wWJKKeHoQ7aY/dxLD/lhMBacaTE0CODq9+ZHU5k4q+MM4qdEjlgdU\ndR7waRH5TMSq4cBMYHZkzkFVLVfVdaoaVtVtmOIpV3X2Qi2WVCHRejGuwzzAF2KSlg5xlkemNz8S\nUxCRPEy1o720jpej07WvB24B5olIW7UoO5Ka3mJJWRJNIPIxKccPYWpPPNjKNpeIyDQnq/P9wFJV\n3SUiJ4nIDBHJFpF0J1Px2cDbkTur6jOY2Md8ERkGR7o5Bzsp3QcBP8ZkV7ZYejSJ1s35FPBpTBr4\nw5iyaF9tsc3TwCxgKiaIeb2zXIB7MXUZQsBm4BpVXd7yJKr6pCMwC0TkHOBUTLCzGCNO/4cRkcTF\n1LzIAbIwKekzMTUwmssO0uLf5o8f06RrAJqABo6O01gsR0iqnJQiMgfYrar3uG1LXDA1HIoiPsWY\nplcOXev9NYtGHaYoUCVGoCux9SF6NInmQfRcTNWrMufTByMGGXE6e4bzKeTokoIgUo8Ri8OYwHI5\nrRQNsqQmViDcQiSbTwShDBNsTURync8nA85EKjGB4d3AXkzZQUsKklRNjKTH1PYc6nxaVqZOVsLA\nAUyX9Me0qI1qSW6sQHQ3RhSGYUShl8vWxINyTO3MrbYpkvxYgegOTA/DSMyQ8Z4gCq0RxjRDtgDb\nbDMkObEC0ZWI9AHGYLyFNJetSST8mGHsazCjXy1JghWI40XEgxGEsUBvl61JdBTYhhGKcreNsbSP\nFYjOIpIGjMbMCs1z2ZpkpAL4CBPYtDdhgmIFoqOICCa+cBpWGLqCKuB9VLe7bYjlWKxAdASRE4FP\nYUY1WrqWcmApTkoAS2JgBSIWRAYAk7AxhniwHeNRVLltiMUKRNuYuRBnYIKQlvihmF6P91H1uW1M\nT8YKRDRETgKmEL/5EJZjaQTeRfVjtw3pqViBaIlIEXAWZn6EJTHYCSy2w7jjjxWIZsx4hgnAROwg\np0TEj/EmNrltSE/CCgSASD4mzZ0NQiY+O4GFdp5HfLACYepqnIvJyGRJDuqA+agecNuQVKfnCoRp\nUnwKGO+2KZZOEcY0Oda5bUgq0zMFwhTFuQDo57YpluNmMyaAGWx3S0uH6XkCIdIfE2/IctsUS5dx\nGPgnqjVuG5Jq9CyBEBkBnEPipfu3HD9+4A1iK9RsiZGeIxAipwKnu22GpVsJAW/aiV9dR+oLhJl9\neSamXoYl9VFgEaob3TYkFUhtgTA9Fedj51L0RJaiusptI5Kd1BUIk9Dl08BAt02xuMZqVN9z24hk\nJjWDdcZzuAgrDj2dcYic5bYRyUzqCYSJOZwHDHbbFEtCMBqRSW4bkayknkCYmZjD3DbCklBMQMSO\nmO0EqSUQImcAJ7lthiUhmezk+LB0gNQRCJHTgVPcNsOS0Jzl5BW1xEhqCIR5M5zqthmWhEeA850c\no5YYSH6BMNWsznTbDEvSkAZchEih24YkA8ktECap7HRsBihLx8gApiOS7rYhiU7yCsQnYx1y3DbF\nkpQUY7rDLW2QvAIB04C+bhthSWqG2O7PtklOgRAZje3OtHQNn0LEJg6KQvIJhElLf4bbZlhSBg9w\nISLZbhuSiCSXQHwyjNoGJS1dSQ6myWppQXIJhKlbYVPTW7qDExGxaQFakDwCIVICnOa2GZaUZhoi\nNldpBMkhEKZL8zySxV5LspKFHXR3FMnywJ0K9HLbCEuPYBgiQ9w2IlFIfIEQKcbEHiyWeDENEVtp\njWQQCNOlmQx2WlKHHGwGdCDRHzzj6tmZdxY3GG0ndCWyQJjA5BS3zbD0WDxAj09Vl7gCAWOAAreN\nsPRoTkSkR8/3SUyBEMkAJrpthsVCknixInKziLzT1cdNTIEwvRZ2wIolEeibKGnqROQtEakQkRoR\nWSUiMzuwb28ReVpEqkSkUkTmxrKft/PmdhPGexjjthkWSwSTENmO+1Wm7gDWqWpQRCYD80VkpMZW\nsPh5YBlwAtBAjPlbE9GDOBmwmX4siUQhENWLEJHtInKniKwWkXoReVxE+orIPBGpFZH5YsbzICLP\niUi5iFSLyCIRGRNxnDki8lsR+Zez39sickLzelVdrarB5p+Y52RQFJt+KiLviEihiEx3trtTVatV\nNaCqK2K58MQSCNNzYTNTWxKRce2svxKT4WwkcDkwD7gLKMU8Z990tpsHjAD6AMuBlq7+9cD9zn4r\nW64XkVdEpAlYCiwEPmix3iMif3Dsna6q1Zg4ykbgSRE5JCLLROScWC460ZoYI7Ap5CyJSR9E+qK6\nP8r6R9RZJyKLgQPNb2kReQG4AEBVn2jeQUTuBSpFpNB5kAFeVdVFzvq7gWoRGaSqu5z9LxOTS/NC\n4CRVDUfYkA48g3muL1dVv7N8ICZ3663ALRgxe1FEhqvqwbYuOrE8CLDpvyyJTFteRKRwNLbyO09E\n0kTkxyLysYjUANud9aUR2+5q/qKqdcBhoH/kiZwmwjzg0yLymYhVw4GZwOwIcWg+/3ZVfdzZ96/O\nedqdmJY4AmHaWkVum2GxtMEQRPKPY//rMA/whZi4xhBnuURscySmICJ5QAmwN8rxvBxdZnI9xkOY\nJyKjIpavxsQsOkziCET7bTyLxW0EGHsc++cDPuAQpin9YCvbXCIi08T05t0PLFXVXSJykojMEJFs\nEUkXkRuAs4G3I3dW1WcwsY/5ItIsHi8AxSJyk+PFXIWZwvDv9gxOjBiESAFQ5rYZltQk6CVcXYy/\nMZdw0AshLwS9aMgL4TTIakBya5HcWtJy6vB6Q22+OEchsgzVQCdMeQr4NLAH03T4AfDVFts8DcwC\npmKCmNc7ywW4F9PLFwI2A9eo6vKWJ1HVJx2BWSAi56jqdqcp8hvgUWADMLO9+AOAuN+1C4icip09\nZ+kiagrxV5QRPNgXqS7B68/qWLe5108wt5Zgvz2E++8gPbfumP0XorqpC00GTDcnsFtV7+nqY3eW\nxPAgTNeQxdJpAumEdg7Dt2M4aQ35ZGKqZ3WKYAbe6l54q3vBxnGQVY+/714CA7bjLTlIJuZ+7XKB\nSETcFwgzGcZOyrJ0ioN9aNo2ivCBMrI0rXu6yJtyydgxgowdIyC3Bt+ItRT1zpaczEZt6I7zJRLu\nNzFEpmHaVRZLzNQW4F81hVBVL1ypZzGvkPLf9GONztKtbpw/XrgrEGbk5I2ATe9liYlgGuH1E2nc\nMYwcPEd1D8aVven4bhtKJlAOLNRZWuOWLd2J2wIxBDPCy2Jpl91DaFh7KhmBzARoGgNfOZHAvgzS\ngSBmItQaneW2S961uD0OYrDL57ckAQq6cjL1K6eSkyjiAHB2Dc2jFb2YbsnLZXZq1dVwWyBsvklL\nmwTTCC+5gKbdQ8l125aWTKo/5vnpB3xWZkvKjAh2TyBMQtDjGbZqSXEaswkumkHgcB93ApHtMbSJ\nzMww4RaL84ErZLakxMvPTQ+i1XnsFguYXopFM8AZ05CQeMEzvgF/K6sygBkyWwbG26auxk2BSPr/\nPEv30JRNcMkFSCLFG6IxpZZQlFUeYLrMlj7xtKercUcgTPemnXthOYagl/C7FxLq6PBot5jY0Kad\nXuDiZI5JuOVB9MWmlbO0wrKz8DXkJW6zoiWlQTLyQlG9CDDJl2fIbOn00G83cUsg+rl0XksCs2Ec\n9Yf6JWZAsi1GN7Yah4gkH4gpxVui4ZZAlLa/iaUnUVuAf8vo5Ew3OKYhpmQsJ8psSbpJiW4JRG+X\nzmtJUFZNIeTm0Onj4aSmmO0+Q2ZLUolg/AVCJAvIi/t5LQnLnsE0ujXpqis4wRdzPC2DJMt74oYH\nUeLCOS0JSshDeO1pid+d2RZ5YbylAYLtbwnAqGTq1bACYXGVTWNpTJYuzbYY4otZIASY3J22dCVu\nKLcVCAsAYUF3DI+9S9MfQr74InfnpFP1+8v59Tfm8aWKek7wCKE+uWz/0QX8JTvddDk+v56Rz6/n\nmrCSluml7k8zebj7rgQG+Ql/0P5mzZwgs6VEZ+nh7rOoa3BDIJLGvbJ0L/sG0RTMiD328PMlXFCU\nxT5/yOwzZSBLrz2FxwG+OY9bH/uAad+eytt7a8l+bh3X3TWNX43ty+GPK7t/zs9Af4fTyo8mhqzS\nbuNGEyPhZuVZ3GHHiNi3XVtB0ebDjD13CEdK3F8/ljUeAY/AoAK2VTZSDPDMGiYNL2HF2L4cBhhW\nTG1X296S/n7SOrjLCJktCR97cUMgkqqbx9I9+LIIHi4l5twJj77PNZ8fwz88cuybujFA2kcHmHJ6\nf9YAHKinb2OAnFte5Ds3vsDdjy5jSlfa3hp9Ah1+ljKAod1hS1cSX4EwXZwdVVpLCrJ9OL5Yxz38\neTVjc9KpvWgYO1tbf9cCrhtQwOaZJ7EFIKykHajnhIcv4pH7z+OXi3dw6dLddOukqZJgp5rrUSuG\nJwrxdnFSvnnRBDIY7i6Eqs3w60Fwp8+Mx6ce8gfBtg3w2Pdh/B9hpoB6IPR9ePYO2LIASq6Fryp4\nQpB2KSx4Cha5fV1dTfmg2F8UGw8yfEc1469+jlPCSnogTNY35vHFR2bwxL0LuawxQN5PL+IvzdsX\nZlKZm05drxz8vXLw98tj80cHGDR5IAe652rM1O+8EKG6tA69AMtktkgip6mLt0CkfPPiRrigH+xr\nxATSdsFPm9eNhtvPMSXd+RpseABWpQFzYcA34Ct3wKzToHoD/KQYgrsgcwzMWgKrpkJ1lFMmHQpa\nlx973YoHzucFTPk4nl/PyNe3MP2RGTzxy6VM21bFmF9dzM+9nk+aHuefyMrHV3CtL4inIYD3QD0n\njuzF/G64lKMoDnZYIDIwo4q7TbiOF+tBdCGLoWgZjP0qvPZHuChy3Q7I3AajXoM5AINMjUYAqiCz\n2dcu5JOZgTXgVZJz+HFb1BYS0LTOF7Zp5s2tXJ+dzqH/fI3vAYwoYfmsc3n1jEGUL9zO2ptf5P8J\n6Ph+vHP2CVEL4HYZRSHCu9rfrCX9SWCBiG9W6xQvsXcy3PYdmHcYsn4P0zfDr5vX3Q5TFsH4dfC7\n5mX/AxP+CJ9tgIKfwSP/CVsBFkHxlfCNKuh9NfxjLix04XK6jR3DaPhoUup5kz8po+mdZ7iNCsaS\nTi3fZTYAc7iScsbhIUQ2FVzHHHrRyKtMYjXn4TvSyzIOOFVVV7p2ES2Idy9G0o+Yi8Y9MLYQar9E\n64G0f8GkS+D9yGUPwcrDMOtB+M3Dpiw8AGdDZQXctwjuWQBTP0yx3J2VKTpVLz9EmHG8y6X86qgV\nJ7Ke7zCb/+E+8tnPy8wA4FLe5/t8X1UnYOrDbE8kcYD4C0TK9mC8B8M/gvEF8OC98OXtMGo8fBFg\nLeTugSHfg49a2/fbsPkw9P6oxSS2qVBdBnufhg6MGEh8qotdz6beLeSFgalsppD6o1acwzrSneS2\nA9iKM17DoVn8rwWeiYedHSHef6iUvDEA5sMLdfDdGrjrXvjDENi4Cp4A+AmcNhpWl/LJeP1XoXdz\nsOFPMDgEaSdD3WIoOuB4WhshZxsMPx32x/2CuhF/RmreBxkaw2jK9ZzJYDNewyHNqaVxDQkoEPEO\nUqbkjdEeC+FTX4LXI5f9EU79AkxNg1A6+H8If0gDFkHZ5+A/AAVkJvzzWtjjht3dRcibmveBpz15\neJoZCGFmsPSo5Y9xJtCgqmta39E9rEB0A3fCpjsjysPvhJ+13OYFeAPzOYq7Yf3dcF83m+gqobTU\nvA+krR6nl5nKPsZxG7845uoruZIE9B7ACoTFBVRSr+sW2vAgFjKGdXyam3iYvBb5K0MIAWYCZ3W7\ngZ0g3gKRskFKi0VA+Q23UslIguTxQ37CKbzEOmageHmKbwFQzFa+zFwA3mMEwj4N61Y3bY9Gws8m\ns6QeHkXDKTgALA2Er/HHY1bMbGNa95ls4kwe7E67jod4u/ztpQe39AAyG2POvpRU+FqZaRoj9e1v\n4g5WICxxJ7u+zUIzSUtdWqe8oqDOUl/7m7mDFQhL3MmtO6YidkpQ17mnqa6LzehS4h2DsAJhIa+m\n++MPgVBGqLKpd7DG1yvkC+aoP5wp4bCXNE9QM9KaSPf4JCe9VoqyDqTnZtR2yXPQSQ+i27NdHQ9W\nICxxpzsEIqwe3V83uGlXzUitauqT7gvlpBNjr1maBEL5mZX+srytOrBgc0aWt7FTz0UnBSKhR8nG\nWyAStq1liR/FB8lAUbpgPEStr8i/8dDpwf31gzPD6u1U8Z2QpqdVNfXJrmrqw/qDk7Ug81DT0OKP\nwgPzt2SLaMw21nk61WTf14l94ob1ICxxJ8NPWkElTTUlseekbElVU6lvXcXk8KHG/lnQlZWzRWp8\npVkry89jfcUk/6heHwYHF26MSSgqvR0e5xMmgXNBQPwFoiHO57MkKGW7CNd0okJKIJQRWn1gmm9v\n7bBskG6NZfhCuRmrD5ydsbXqlKbTy+Z78jOrogpRAMJVHReI/TpLE7pHJ969GAkdkLHEjwHbO54b\n5ED9wKYF268J760dntPd4hBJnb8k6+0dV3nXVUyqV5VWxzpUeTvVdZuQoycjia9AqIawXoQFyGkg\nPac29ibnlsPjG5bumZHpD2W7knRI8Xg+rpyQ++7uy5qCYe8x3bQHOy4QYZJAINwYal1DD0hea2mf\nATsIbD6l7dyUqqIry89p3F07Muo9s692X/Fb29+6JRAKFAI6oGDAoguHXrigsrEy5/Utr3/FF/L1\nykzLPDRjxIzfF2UVNVQ1VeW8seWNm3whX29BAmcMOuPJYSXDYspZebixLPvtHVf6zhj4Slp2ev2R\n5+dAeofHduzQWdrYwX3ijhuzK1MmO7Pl+Bi2nuy0QNtv3mV7L2pTHADSPGnh08pO+/tNE26a9bnR\nn/vR7prd522t3Fr2zs53ZpRkl2y4ecLNPyjJLtnwzs53LgZYvGPxjILMgl1fGP+F+6YNnvanpXuW\nXtMRuxsChZmLd35Wm4I5R4aM78vo8DDrtR3c3hXcEIgqF85pSUC8QTwnbqIp2vpV+8+q318/pF1v\ns09un+pRpaN2AuRn5vuyvdn7qpqqig41Hho/sWziEoCJZROXHGw4OAGgPlBfNrBg4AaAE4tPLPeH\n/L0q6is6lPfTF8pJ//euz4QCoYwQwPbMDj1L+3SWdnuW7a7ACoTFVYavbd2L2HxoQsPO6tEdLpOw\np2ZPr4ZAw6ARJSO2BcPBgj65farBiEgwHMwHyM/I372tattEgPUV64f4Q/5ehxoPFbd13NZoCBRk\nvrv7skBYPbols0NB12UdPZdbuCEQCV/y3BI/vCE8Qzcc7UVUNvX2bTh0eocHPdX56zIXbFtw++je\no5/Nz8yP6pmcfcLZrwdCgZynVj31g7UVa8/P9mbvSpO0Ts0PqfGVZn1w8IyG/Rkxx/N26Swt78y5\n3CD+QUrVWkQasIFKi8Ow9WTvGEHAn0V6MOwNf7D3Iqdmd+wEQoG0lze+fHu/vH5LpwycsgLA6/HW\nHKg/UNgnt0/1gfoDhV6PtxYgPzO/6cqTr3wSQFWZs3LOg/3y+h3srP3r6id5aKhsImdtewO/QsB7\nnT2PG7iVAi6hx59b4os3hGfiEtPMWFsxtbEpmNehkZGqyksbX/pCbkbuvouGXXSkxF6v7F6rVuxb\nMRVgxb4VU3tl91oFUN1Une0P+dMA3t7x9rSCzILNbXkc7VGRVRJm3397CKe354Us01la2dnzuEF8\nK2sdOauMg+4vyW5JLt4dX1jzpPc/8jvqPXy0/6PhS3YvuTPLm7UHkw2ck3uf/MLQ4qHbXt/y+lf8\nIX9JRlrG4RnDZ/yuOLu4Ye2BtUOX7V12iyDh7PTsfdOHTX+qKKuo0+Nz5mx69Xp/Q8VEvFrN9wP3\nR9lsH/BKIhfqbQ23BKIvEZWkLBaApfL9+mdH+9Lqsus6PUfDDZ7015/oyykNseLxm/le4F48vpae\nuQ94Xmdp0o0kdquJUQGpmVXI0jka6B2o0Mm5UzdP9XpD3qRJSdeYlhHwnfL5beSU1oMIlZe1bKqE\ngDeSURzALYFQDQOdDgpZUo/tXOaHNLICWd4pm6eE0kJpSfEC2ZddcrSYVc5M55P5GgosSKZei5a4\nWaciaf/TLF3PHs47Epgsri9j34wbAAAPJElEQVTOPGPTGcFk8CR25fY9uo0eKk6n8ZTmvCdLdJZu\ni79VXYebArHDxXNbEohDjGnyUXzUQKOihqLMaRumhdOD6QktEjty+2Yes7Dm3BCwWGclXim9juKm\nQOwHEn6yiqX7KWdKq82J/Kb8jLPWn6WZgcxAvG2Khar0XF+TN/PoHBAhCfOvb9XqLF3vklldinsC\nYbpPrBdhoZKTo96Huf7c9LPXny1F9UUJ9zLZldvHeDevff1WFj3wXfx1fXn26of4+IErRTqfLSuR\ncKeb88jZZTBwsXsGWNxGEZ3Hc+Ewme1mY9raZ2vD+gHrM8OecEKUcHxh8Fn+iqwiEzvZ06eBJRMz\nCaQ32/YvVZI6/gDul97bAwSg49mFLKlBA/2CYTJj+vsPPTA0p6yyLPjh0A8bK/MqO5WgtquoT8s0\n4uBLD/L+uAC7y1pOHSiG5BcId6ttmwxTu1y1weIqjZR2qDszO5DtnbZxWvbYHWMbMgIZrsUmtmcN\nDLB+aD0vXeBhd1lrYpUSLz23PQgwKjvUbSMs7tBEaadmUQ45OCTnhIMn6N7ivY1b+m3x1OTUHNub\n0A2khdJCQyqGNL1c/sU0QqVtxRmsQHQROzDp8LswdbklWWiiE6mtHQSRAZUDsgdUDqAqp8q3qWxT\n6EDhgSwV7VLPWMISLqkv8ZVVlungg4OzyrUgfR+l7d2vKXE/uy8QqkFENgGnuG2KJf6E8XZJlLyo\noShz0seTCHqC4YqCiqbywvLw4fzDaQ0ZDRmdKdCTEcgI9KnpE+hf2V9Ka0oz0zTtSDPiTQYGaV8A\nrEB0IeuwAtEjScPfpenrvWGvp6yqLKusqgyAoCcYrsqp8tVm14YDaQENeAME0gIEvAFPIC2AJ+zR\n7EC25vhyJMeXIzn+HE+2LzstK5iVTivNhBDoGwyOpTljmxhdhmoVIvuAMrdNscQXbzdXQfCGvZ7S\nutKs0rrSLjneKno31pAZS7KjpJhL0h7u9mIczTq3DbDEn0yq4lYApyv4C6NiHYORlLM3W5JIArEN\nO/S6x1HA1sTwYmNgM4VNmymKtbekpluNiROJ88dRDSOyEZjgtimW+JFDRboHX6jlSMr/4r9u2sWu\nsZlk1j7N07MB3uKtgXOYc32QYGYeeYfu477H+9K3CWAhCwf8iT/dECCQLUj4MR57sICCLp3o9Qwj\nOxJQTarUctFIJA8CYA0p0nazxE4u+44Z8HQu5777Vb76q8hlT/DEF67giufnMve+UzhlxaM8Oh3A\nh8/zB/7wpZu5ee7TPH3vj/jRz7LI6tL7qJwc3zL6dmT0ZkVXnt8tEksgVBuADW6bYYkvvVhzzMN8\nBVdsLqW0PnJZHXV9ZzJzM8B5nLd+M5tPBfg7fz+5hJLd53HeboDBDK7PIKNLJxk9y/COCE6damo0\nlxNLIAwrocN1Di1JzEAWxNTULaBg75/583iAV3jltEYaSwB2s7uvINzKrXfcwA13/5AfTu9K+6rJ\nCCxgYEe8h5SZpZx4AqFaj/UiehRFbM7MoLrdeRW3c/uTi1l87o3ceHcTTVkePEGAMGFPOeXD7+Ge\nxx/hkZ9uYtPE53jupK6y72WG+EN4BP6vL/T5wSefjF/CNReYra47D0rug173QsHVXXVut0mcIOXR\nLAdGkrj2WbqYfizx7+TiNgcXTWVq+VSm/hJgKUv7bGXrWIBe9Koso2zTEIbUAQxl6JrNbB5MF7xo\nqskI/B9DHe/hiv1whZPWvkmg10Nw2wp4aBQsmQAf3wfFO0HeP97zJgqJ50FAcywiKaofW7qGEfwt\nE0Jtxg0+5uN8gCBBmcvcSz/Fp94GuIRL1h3i0IBKKjN8+Dw72TlyMIP3dYVdf2ZUwIe3lefk4dFQ\nXAHnH4Ynz4Evz4PiILBOVQ90xbkTAXcTxrSFSCbweSAus/Qs7vMB32so58wcgK/z9VvLKR/px5+X\nSWbtWZz1UhNNmctZfh7AMIYtv5/7X/A477jHeGzyYhbPAHQIQ9Y8yIP/OF579pLju43zo9x/p90E\nJ+2AuQtNc+PUlbBiNBzYA/y3qiZNgd62SFyBABAZDZzlthmW+FDHAP9CHk2HtIQYXXkPk5tW0buV\nKd3VadDvIXjnXjitFkpnwagN8NadkOkF/gYM1YR+uGIjMZsYzaiuB1LGXbO0TR57MgbxZvdOzoiR\n5fRuaF0cAH58CpTtNOIAUFgJ09+BjNWq+j6mF65rJn+4TGILhGExTr1FS+ozht9nZ1DlahZrP57w\nI4xrY7r2K5PggogmxJkr4dE8VYIiMhIz1TslCkMlvkCoHsKMsLT0ALz4PBN5OARh114KzzOs8SDZ\nUXrQyjNg82j43opPlv34t1DRV0TWAH8FbkqF5gUkegyiGZF04Gog121TLPFhM1c1bOSmWKZVdynl\n5Phu59yMUOwVxsuBV1RTc3Bf4nsQAKoBYInbZljixwj+njOABXGNRwSR8AOcLh0QhzpgfqqKAySL\nQACobgW2uG2GJX5M4H+zS1kRN5GYy8imHRTEmiruMPCiajdnvHGZ5BEIwzukyDx7S/sIKpOZld2f\nt7v9IdxAUePfGRFrk2Yf8JIq9e1umeQkl0Co+oE3sZO5egyCyqk8nDOMv3fbw9hAWuiHfCrWHJJb\ngddU8XeXPYlEcgkEgGoFkDJj3S2xMZoncyfz/5pimdTVUf6XCf4qM8CpLRRYDbyp2nNyliRHL0Zr\niMwABrlthiW+BMgJreRbvv1M6ZIejrfp3/Awp7Z3rIPAYtXUSALTEZJZILKAq4C4d4VZ3KeSUb41\n3KbVjOh0Fe1ycnxf5+z01idjASbx7HJgk2rPHKyXvAIBINIXuAxIiGrPlvhzmNG+LVwVOsiErDAZ\nMTeZ6/AGv8nZWkFOy9iDAvuBTRhh6NHxruQWCACRocCFbpthcZcgmeFyzmjaw7lUMyzdT2HUoGMQ\nCd/F1MB6SppnajZhikjvAnap4ouHzclA8gsEgMh4YLLbZlgSBx9FwcOMDtTTX/0U4qeAALkIIf7K\npVt+xrQGoAHTbV7RU5sQ7ZEaAgEgMg042W0zLAnPMlRXtL+ZBZKxmzM6/wZ2um2EJaFZb8WhY6SO\nQBhX6E1SZJqtpcvZjhmJa+kAqSMQ0Dyp6zXgkNumWBKKrcB8orSnReRmEbHi0QqpJRAAqk3Aq5jJ\nNBbLVmCBwJsiUiEiNSKySkRmxrKziJSJyEsisldEVESGtFg/R0T8IlIX8UmZbvfUEwhoFolXsJ5E\nT2cL8CaqYeAOoExVC4CvAH8RkbIYjhEGXgeubGObh1Q1L+KTMkOxU7fuhGoTIi8DlwB93DbH0joF\n8OCFsHAxTKmC3uNh2e/ghavhll0wfABsex1+NwoaxsBXtsOIIKT3gd0/g7lXm5mVnAo3p0PgAPTe\nA0MLYGMdfKbJaVao6uqI0yqQjhmqf0x6fBH5KTAVuFRV9wO/EZHUfVbaIDU9iGbM7M9Xgb1um2KJ\nzntw6uvwi4Xwg3Uw7hL45g/ghf3wbQX5FpwPMBXWrIN7yuG/h8DOr8OtkcdZAZO/Aa9uh8sPwUIf\nzI1cLyKviEgTsBRYCHzQYr1HRP4AjAOmq2p1jJfwNRE5LCIfikhbnkbSkfqqqBpAZB5wNjDCbXMs\nx/JZWHCamffACbClEGpuNqMaOQNWLIeTAP4I7zbv8wS8PBIu+Biyh2EK5Y6Cj74Nz6C6HJElQLWI\nDFLVXQCqepmY9IUXAiepaXo0kw48g3kmLlfzcomFXwHfAaqB6cDfRKRcVf/d6f+QBCL1BQLAtAnf\nQqQSmOS2OZajGRCRBCgD/MWOWABkQcAHWU0gl8IVH8JpDZAvTqbzTZA3DBrDED4EK1BdDqCqdSJy\nGOiPIzbO8gAwT0TuEJGPVfUlZ9VwYDwwqQPigDrnc3hNROYCn8OMy0l6UruJ0RLVlcAbgKtp1S0d\n51swaSVMeBZ+0Qh3rIHvw5HMQQ1bYNc+KGjeXkTygBKiNy+9wLCI3+uBWzDiMeo4TFUgIQr/dAU9\nSyAAVHcAL2ISjlqShFrISoPgCKjfDxlfgM8C1EEl8EI9+IBLRGSaiGQA9wNLVXWXiJwkIjNEJFtE\n0kXkBkyT8+3Ic6jqM8BdwHwROSIeYlILNE/synR+N6+7SkTynPjFdOAG4CVShJ4nEACqh4HnMSnL\nLUnAj+C9Yjg0Ch46CWaPN+Mb+KbpxmxOR/c0MAszBuY04HpnuQD3Yqq0VWC6PK9p0TwAQFWfBO4D\nFkSMeWjkkxfKBud3M3cAe4Aq4KfAl1V14fFeb6KQOpO1OoOIABOBU+mpYpmc+IF/o7q5eYGIzAF2\nq+o9rlmVgvSMIGU0jDouR2Q3cB5Q6LJFlvYpB95CtbbdLS3HjX1rAqgeAP6BCVRZEpMwZtzCy1Yc\n4kfPbmK0hshg4Bwg221TLEeoARY4Qm6JI1YgWsNEqScBo0ihLqskJASsAlaiGnTbmJ6IFYi2ECkF\nzgT6um1KD2Q3JhAZ63BnSzdgBSIWRIYDU7Ap9uNBDfAeqtvdNsRiBSJ2zBj+icBYbJr97sAHrADW\ncPQcCYuLWIHoKCK5wATMBCIrFMdPE6ak3VonI5glgbAC0VmMUIzHCEXPHk/SORowAcj1NgCZuFiB\nOF5Mj8cY59PpMnA9iDpgJbCRFMq8lKpYgegqTMah4ZiuUdvrcTSKmXK9AdhpYwzJgxWI7kCkCCMU\nI+jZPR81wEaMt9DgtjGWjmMFojsR8WDyHo4CBtMzhrb7MQWMNqBqU/0lOVYg4oXpJh2AEYrBpJZn\nUQPswAjDPtuESB2sQLiFGaXZLBa9Sa4h3WHMrMqdmJhClcv2WLoJKxCJgMmAVIoRiuZPvqs2HU0d\nJtHKfudz0PZA9AysQCQqpvu0WTQKMfkWC+i+pkkQkyy2FiMINZjMTAedQkSWHogViGTDlHXLAXKd\nTzZmoFbkJy3iewiTpDcY8W/zdx/NomBFwNIKViAsFktUekK3m8Vi6SRWICwWS1SsQFgslqhYgbBY\nLFGxAmGxWKJiBcJisUTFCoTFYomKFQiLxRIVKxAWiyUqViAsFktUrEBYLJaoWIGwWCxRsQJhsVii\nYgXCYrFExQqExWKJihUIi8USFSsQFoslKlYgLBZLVKxAWCyWqFiBsFgsUbECYbFYomIFwmKxRMUK\nhMViiYoVCIvFEhUrEBaLJSpWICwWS1T+P466IQrfz50JAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1af50b9e80>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#visualizing intersections is a much easier than I thought!\n",
    "from matplotlib_venn import venn2, venn2_circles\n",
    "from matplotlib_venn import venn3, venn3_circles\n",
    "venn3([set(MAP3Kwords_list[2]), set(MAP3Kwords_list[3]), set(MAP3Kwords_list[4])],\n",
    "      set_labels = ('map3k5', 'map3k6', 'map3k15'))\n",
    "\n",
    "plt.title(\"MAP3K5, 6, 15 comparison\")\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAASIAAAD7CAYAAADO+JnlAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMS4wLCBo\ndHRwOi8vbWF0cGxvdGxpYi5vcmcvpW3flQAAIABJREFUeJztnXl4XOV1/z9nRqPRakmW5H0DG2OD\nMQbCFtYEHAeSNCzNj19WnGbhaZ80SZOmSQhFVhLSbCT9tSVtkpKSBSi0DQkQIKxlMRgw2NhgFtvy\nbsubJFu7Znl/f7xXMCia0TKjee+dOZ/nuY80c7dz78z9znnPe97zijEGRVEUl4RcG6AoiqJCpCiK\nc1SIFEVxjgqRoijOUSFSFMU5KkSKojhHhUgpKkTkoyLyoGs7lLejQpQFIrJdRAZEpGHI++tFxIjI\nvCHvr/LeP2PI+ytFJCEiXSJy1Nv//d66E0RkrYi0e8vDInJCyr63iMi3U16fKCL7ROTLIhIVkZtF\nZIeIdIrIOhG5ZAzX90URafFs2isiPxaRkrHdJX9hjLnVGPMe13Yob0eFKHu2AR8efCEiJwHlQzcS\nEQE+DrQBVw9znGeMMVVALXAzcKeITAb2An8OTAYagLuB/xzOEBFZBjwG3GCMuREoAXYBFwA1wN97\nx503ymu7BzjVGDMJWAKcDHx+lPv6jqCLaCGjQpQ9vwY+kfL6auBXw2x3HjAD+ALwf0WkdLiDGWOS\nwC+wYnasMabDGLPd2BR4ARLAgqH7eV7Ww8C1xph/8Y7VbYxZ5e2fNMbcixXO00ZzYcaYrcaYjsFT\nAMnUc3ve10Mi0iYi+0XkWu/9qIj8o+dF7fX+j3rrLhSR3SLydyJywPPeLhORS0XkDe9Y16acY5WI\n/LeI3OF5dS+KyMkp678mIlu9dZtE5PKUdStFZLXnybUBq7z3nvLWi7fugIgcEZENIrLEW1cjIr8S\nkYOeR3mdiIRSjvuUiPzQ81K3jcXTVIbBGKPLOBdgO3Ax8DqwGAhjPZC5gAHmpWx7M3AnEAEOA1ek\nrFsJPOX9X4IVq06gJmWbDiCOFYPrUt6/BXgQ62l9fAR7pwJ9wKIxXONHgKPe9RwETvberwb2AV8G\nyrzXZ3rrvgmsAaYAjcDTwLe8dRd613G9dy8+4x33Nu8YJ3o2HuttvwqIYb3CCPC3WDGNeOs/hBX4\nEHAV0A1MT7mvceCvvftaPuRerwBewHqh4n2Gg/v+Cvi9Z9M84A3gUynHjXm2h4G/xHqu4vo7GdTF\nuQFBXlKE6DrgH4D3Ag95X/o3hQio8B7my7zXPwV+n3KcwQemAzjkPcQXD3O+SuCvgPelvHeLd+xt\nQEMGWyNYj+mn47zW44BvAdO81x8G1qXZditwacrrFcB27/8LgV4g7L2u9u7VmSnbv5Byr1YBa1LW\nhbACeF6ac68HPphyX3cOWZ8qRO/2BOYsIJSyTRjoB05Iee8a4H9TjrElZV2Fdw3TXH8ng7po0yw3\n/BrrOaxk+GbZ5Vihuc97fStwiYg0pmyzxhhTa4xpMMacZYx5eOhBjDHdwL8BvxKRKSmrbgKeBx4S\nkbqh+3lNil8DA8Dnxnpx3rk3A68AP/Hemo0VnOGYAexIeb3De2+Qw8aYhPd/r/d3f8r6XqAq5fWu\nFDuSwO7B44nIJ7zgfoeIdGBjWQ3D7TvMNT0K/Av2/u0XkZ+JyCRv/9JhrmFmyuvWlOP0eP+m2qyM\nARWiHGCM2YH1SC4FfjvMJldjv6Q7RaQV+C+sh/LhYbYdiRD2Fzj1oUgAHwV2An/0HibgzSD5zdhm\n2ZXGmNg4zjlICTDf+39Xyv9D2Yttng4yx3tvvMwe/McT1VnAXhGZC/wcK671xpha4GVsM2uQjOUl\njDH/ZIw5DdskXAh8BeuVxoa5hj1ZXIOSARWi3PEp4N2e1/ImIjITuAh4P7DMW04GvsfwvWdvQ0SW\ni8gpIhL2BOZHQDvwaup2nsB8CPsQ3Scild6qf8XGPj5gjOllCF46wYVpzv3pQc/LSxn4OvCIt/pe\nYJrXxR8VkWoROdNbdztwnYg0ik1tuB74zUjXmoHTROQKr9fri9hm0xpsU3UwdoWIfBLrEY0KETld\nRM4UkQg2ttQHJDxv7U7gBu+65gJfyvIalAyoEOUIY3uY1g6z6uPAemPMg8aY1sEF+Cdg6WAvTQZq\nsQ/2EWxTaAHwXmNM3zA2DABXYB+oe7wH6Bqs+LWKzVPqEpGPAojILKAL2Jjm3OcAG0WkG9usvA+4\n1jtXJ7Ac+AC2mbIZeJe337eBtcAG79gveu+Nl99jA9Ht2Pt5hTEmZozZBNwIPINt2p0ErB7DcSdh\nPap2bNPrMPBDb91fY8WpBXgKG0z/RRbXoGRAvGCbUoSIyMeAE40xX3dtSzpEZBWwwBjzMde2KBOH\nJngVMcYYbWoovkCbZoqiOEebZoqiOEc9IkVRnKNCpCiKc1SIFEVxjgqRoijOUSFSFMU5KkSKojhH\nhUhRFOeoECmK4hwVIkVRnKNCpCiKc1SIFEVxjgqRoijOUSFSFMU5KkSKojhHhUhRFOeoECmK4hwV\nIkVRnKNCpCiKc1SIFEVxjgqRoijO0emEFGdIs5Rjp+Kuws7aWomdcz6S8jfCW99T4y1J7DTbMexk\nkkOXLuCIaTL9+boWJTt0Fg9lwpFmqQLqgckpf6uB8ASfuh87Q+7gchg4YJr+dOptxS0qREpOkWYJ\nAVOA6d4yBevd+Iku4EDqYppM0q1JxY0KkZI10iz1wBxgBjCV4DX5B4C9wC5gl2kyXY7tKTpUiJRx\nIc0yFTgGmAdMcmtNzukAtgNbTZM57NiWokCFSBk1nudzPHAsUOHYnHzRDmwBtpgm0+namEJFhUjJ\niDRLBFgALAIaHZvjmlbgVaynpDGlHKJCpAyLNMtk4CRgPsGL+Uw0vVhB2mSaTI9rYwoBFSLlbUiz\nTAOWYYPPSmaSwDbgJdNkDrk2JsioEPkVkRJsrk0VNqkPQNIsYHt+uoEeoBsztqaDNMscrABNy9r2\n4mQHsFaD2+NDhcglIrVALW8JzuDfKqAsy6P3MShKb/1tBw5izNE3TWiWmcAZaPwnV2zHClKba0OC\nhApRvhApxXobU7FJfo24S/QbODSFfb+4FHN3PVM3llN6tGTCs5yLja3As5qTNDpUiCYKkTA2s3gm\nNtGvgbeaUc7oLyP+8qkM7JtDOfKWPQdKGHixktgTkwi/XE7UiHtbC4A4sAFYb5pM3LUxfkaFKJeI\nCFZ8jsMm+/lqaMO24+h5bRnRxAjeT3eI+IYKBh6fhKytJNof0ioNWdIJPG2azA7XhvgVFaJcIFKP\nzbVZgB1B7iu6q4i9+E4SR+rHHneKQfLlCvrumkxoXWXWcatiZzvwpA66/VNUiMaLSCXW81mAHU3u\nOwyYN06id8tiykw4e6/mcJiBP9YS+0MtZRpTGjd9WDHa5toQP6FCNFZEJgGnYEXIt02W7kpiz19A\nsquGaK6PHYfkukr6/2cy8kqFeknjZAuwWmsmWVSIRovtaj8Vm2ns60Du7nn0bDidaDIPXsvmKH03\nTwEVpHHRAzxmmswe14a4RoVoJEQmYwXoGHwuQIkQyZfOpG/vvPwPSH2jjN5/byT0akXuPbACx2Dz\njta5NsQlKkTpsAHo07BlLnxPVzUDz12I6alyKwSbyuj99ymENperII2RHVjvaMC1IS5QIRqKSAQ4\nHTgRn3tAgxycRt/a84iM1C2fT1ZX0fMv04h2hf1jUwA4CjxYjFnZKkSpiMwBzsUOsQgE2xbS88op\nlBPyn2j2CIn/mMLAA7WUu7YlQMSBh02T2enakHyiQgQgUg6cgy34FRg2nE73zgX+y1saytYofTdO\nJ7Qr6q8ETx9jsF38r7k2JF+oEIksAs6E4MQ0EiGSz15If9vU4HgaCTD31NHzi0YqdPjIqHnRNJm1\nro3IB8UrRDYf6HzsOLDAEA+TfOYiBsaTJe0HtpfS1zyLkkMRLbY2Sl7DekcF/aAWpxCJzAXehc/G\ngo1EvITk0xczcLQumCI0SI+Q+MfpDDxTHRyPzjEtwKOFXJ62uITIDko9HVsALFDEIiRWLyfWVRNs\nEUrlwRq6fzKVioQ21UbDVqwYFeQDWzxCJFIGXIQtyxEoBkpJrF5OvHtScOJYo2VXKf3XzyKsTbVR\nsQWba1RwD21xCJEdnvFeAjj/VjxM8qkVDBSSJzSUzhDxb8wmua0sWE1lR7wBPF5oYuTbQZs5Q2QW\ncBkBFCED5rkL6S9kEQKoTlLyg52ET+mmz7UtAWAhNtWkoChsj0jkeGzPWCBjEGvPoad1TtFMZEgC\nzE1T6X2otniuOQvWmCazwbURuaJwhciK0AWuzRgvL59K9/bj/Z+sOBHcOZnuXzcW57WPkYdNk2lx\nbUQuKMymmchCAixC246jp1hFCOD/tFH5l/vpdm1HAHiXNMtU10bkgsITooCLUHs9/ZtOLeyY0Gi4\ntIPKTx5QMRqBMLBCmqXatSHZUlhCJHIcVoQCGRPqj5J4/nxCRovVA3BFO5VXHVIxGoEyYLk0S6Cr\nHBTOF15kAXAhARUhA+a5C4gNlL05q6sCfOwwlR9sQ+eXz0wDcLZrI7KhMIRIZD52yEYgRQhg4+n0\nBHX82ETz6YNUrOhQMRqBE6RZ5rs2YrwEX4hEphBwEdo/g94glPNwyV/tp/z0LnQansycL81S69qI\n8RBsIbJ1hJYT4OuIRUisP0ubYyMRAvnqXkpn91OUpVRHSQS4SJolcM9D4Ax+E5EQcDE+nNBwLLz4\nTvpjUR1nNRqihvC3dkFVgoRrW3xMPXa6q0ARXCGyxcymuzYiG3bPo+fgDM0iHgv1CUr/frd6RSNw\nijRLvWsjxkIwhcj2kJ3k2oxs6C8jvvEdhTeaPh+c0Ef5pzTHKBMh4IIgNdECY+ib2Gl+zndtRras\nP4uBRERnuBgvl7VTqcHrjDQAJ7s2YrQES4hEotjgdKBjKgem03dwujbJsuVv9hGp0HhRJk6TZqlx\nbcRoCJYQ2aStwJXzSCUpmA1nBO6++5LqJCV/04rOHZ+eEAFJdAzOAyEyA1uLJdBsPpGevgotAJYr\nzuqi4pxObaJlYI40yxzXRoxEMITIdtWf69qMbOkvI751sWZP55rPtRLRLv2MnOX3wLWvjUvhFCCQ\nGaOpbDqFgaSPpoUuFKqSlHx5nzbRMlALLHJtRCb8L0QiNQRw1o2h9FQQ2zNHp8+ZKN7RTcWZ2kTL\nxGnSLL7t5PG/EMF5EHwv4rWTiflxfvpC4rMHCIuhQEuOZk05cIJrI9LhbyGyRc4CNRPrcPRUENur\n3tCEMyVO6fs71CvKwFK/1i3yrxCJlAJnuTYjF7y6TL2hfPGRQ5SWJSnYGVGzpAKfxor8K0SwFILf\nw9RTQWzfbPWG8kVVkpKPH1SvKAMn+7EHzXcGAYPe0BLXZuSCLSeoN5RvLumgvCFG3LUdPqUKH+bj\n+VOIrDcU+KS/RIjknnk6sDXfRCC08qCO0M/Aia4NGIr/hKiAvKFd8+nXga1ueGcnZZPimuSYhnpp\nlimujUjFf0JkuxgD7w0BtBzvy/tbFEQgdGWbTmGdAV915fvrQbFDOQrCGzrcSH9PtTbLXLLiCNGI\n9qClY740i2++n/4SIjgOCqM8xrbjtVngmsokJe89ol5RGsLA8a6NGMRvQrTUtQG5IBEieWC6ekN+\n4Iq2YNeummBUiP4EkUagzrUZuaB1Fv06uNUfNMQp1UqOaamTZvHFM+cfIYIFrg3IFbuP1fFOfuK9\nHfp5ZOBY1waAX4RIRIDAzlKZSjxM8tCU4GeEFxLLeiiLatA6HSpEKUynQILUe+fSZ8K+ua8KUGoI\nndupQes01EmzTHZthF8emILwhgD2zXZtgTIcy4/45rvuR45xbYD7D8fmDvnCPcwFbY3aW+ZHFvUS\n1XKyaXH+8+leiGAWFMbD2zFZh3T4lTDIRUe0nGwaGqVZnI5m8IMQFUxvWessHfHtZ87u8sX33Y8I\njgsQuv1gbG+Z76c6GS0Hpqs35GcW9BHRUrJpKWIhgskUyADXeJjk0drCaGIWKlFDeFGvlgdJwyyX\nJ3ctRFMdnz9ndNRrAbQgcEa3Np/TUCvN4iz/zbUQTXN8/pzR3qA9MkFgWbc2nzPQ6OrEroWoYDyi\n9gbXFiijYV4/pVoaJC3OvsXuhEikAqh2dv4cc7ROR3kHgRIInaBxonQUoRAVkDcULyHZV07EtR3K\n6FjYpx5RGlSIgkzHZAYQDVQHhWP79bNKQ7Wrqo0uhahgAtWdNfoLGyTmqBBlwkl9IpdC5IuCTLmg\ne5ImyQWJqTFtRmfASdzWjRDZKYMK5svQXeW891EZA1FDuDFGzLUdPqWIhMjONlkw9FaqEAWN+X2a\n95WGSS5OqkKUA3ortOs+aMztVyFKQ1F5RJWOzptz4iUktfRH8JisMpQOFaIg0h/VX9YgUhvX5nQa\nyl2cVJtmWRKLatd9EKnRn490hKVZ8u7hqxBlSaxUu+6DyKSEekQZyHtSozbNsmQgqkIURKoSGtfL\nQN5rhLnq7clacX8HUz8Lnx183QENl8Pd74PXvwYfi0MkBIlmuO0a2H4FvOcpOBMgCaE2mP4KfGkx\n9GRjR9CE6IsPcPWuo5wUDdN525U0A3z9Ea7c3sHSkJCoLuXg9Rdwy4xqentjhL/2MB8/1MOcJIRP\nbOSZ687nAYDvr+bd61s5D5CTp/HkV8/hEacXNkYqk3kQop9yNQc5iQidfNXea27hSlpZSogE5Rzk\nI9xCPb30EeaXfIwjzAUM53AH5/AGAH2E+RUfpo3jgSSn8jvew7oJtDzvHpErIcraE7sM9l8G3wLo\nA6mH718D6z4NH/8s3LMKXmmCJTfAldfAjb+FB7EL34Clt8LF2YoQwEDAmmYXzuPpqlIe+8U6Pjn4\n3rKpvLrqAu6KlpD8xiNccdNzXHLDRfz2ly9xWtwQufVKvtneS+k197JqfSvPH+0nur6V8266lH+o\niBD/q/v4wnN72HjGTA64vLaxUGoQMRgzkWMEl/I0ZTzGg2/da47hVT7KXURIcgtXcA+XsJLf8gfO\nA+Dv+Cb7qOY3fJ6z+A5hDP/NpZTRydf4exIIhya8RVE0TbOcnveHsLgODr4b2gA6vMh/O5TXQsfQ\n7e+B098Fz+Xi3CZgkYbLFrG5oYLu1PeuWsKmaIkNui+op+XogB1+I4KJJyntjxPqHCASEhL15fS+\ncZjpU6toqStnIFpCck4NbzzUwjIX15MNJRP9E3I2m6l5+73mAjYR8To4ZtJCrzfUqZ3pzOI1AKbT\nSQk9rGMuADs5h8u5H4Awhql0TbDlef9WB9YjSuUuOP0CT1h+AHeshC/cDH9uQO6G76Vu2wqlb8CS\nO+D2XJw7GTAhGok1uzln2VTWAnxiKS9u2M+yj/yWHySSlF44jztn19BzQiN7Hm7hsp1HqKwuJdbS\nzpIplexwbftYCYEBhwNgX+Uc5tt7TQO72c7JxHiendTRxVzaqOMw+wH4bz7IIRZSwUEu53Zm0DmB\nluX9ngTeIzoC4U2w9EvwAsCP4IJr4M5O+Npn4M5PwdWp238Xls6BLblolhUa33ycSwSS17yDZwEe\nbmFeSEj+5nL+7sYVXLtmN8tf3EfDO2fT+s7ZPHDtI3zxSw/y+YYKdockeGkMTofg38YlCEkusfea\n97Gacjr4Md/gPq5iElsJkSROmAHqmMkWvsINNNLC7/nQBFuX91vjyiNKkiMx+i4smQ47T8P+QqyF\ns5+AOwB+CC/cBJ9I3f4BOH0FPJ+LcxcSNz3H2VvbWfqPK/hxyPsaPr6DMxY38HJ5hMT8OjqnVbH1\nhX3MPXU6hz5/JquB1QDXPcpldeW0OzR/XDgL7t3D2exjKdfw4zefgghJ/oI739zmRr7KdA7QSBch\nBriI9QCcwQvcwbkTbGHeb40rjyhnv573whkXpQhLFRz5ESwE+D4squOtAOpWKN8OC7+O96HmgFDg\n/IA/5faNnPjULlasuoCb6srfKqNaV0bbG4dZlDRwuIfS/d0cc9xkWgG2ttuhABv3M3lrO6dcdWLw\nxD3pwin6X05kEyv4KDdRlVKytotSOr1u8ydYjJDgRPYRAhrYwDP2O81GFlHF3gm2Mu9CJMY4+F0Q\nWUkOchVaoXQefPcV+MZ86AX4f7DgO3BVEkIRiH0LbvsU7AT4LJz9DCzZCD/P9tyDvH4S3ZuXBCcv\n6nP38enWLhYOJKiKltB53hzuXr2LS5KGkmjYBlanVtFy43u49VAP0esf4+qOPmYY4KQpPH3tebbn\nceXv+Ep/gsqQkLhsEf/1oRO8QGuA+LOFE9xr9hM+TTsLiVNFCZ0s4W42cQmGEkq8IHYdLXyGW2mh\nnjv5AoIhSgcf5JccYztf2MZk7uIviFNBhE4u55fM89ZNDA+YJrNzAo//J7gSok8AzuZQyiUtx9Oz\n6VQqXNuhjI04JC8/XrOr03C3aTKt+Tyhqw+iYGZRKO13bYEyHrrDOlg5A3l/Pl0JUffImwSDyIDW\nPw4iXaFgJaLmmbz/vKoQZUlpv45ZCiJHw8FLN8gjRSNEE50ZmjdKdUaIQHKkRIUoDUnTZOL5Pql6\nRFlS2q8BzyDSEdamWRr6XJxUPaIsicQIh+Ia+AwabVplPB0TOXQkLeoR5YCyXhWioLG7VD3ZNBSV\nEBWMRwRQ3q1CFDRaynTmlTQcdXFSN0JkTB8UzsNb2aWBzyARg+TeiApRGorKI4Jh6gQFlaqj2nMW\nJA5GiE3o0I5gU3RCtN/huXNK1VGNNwSJnToFVCacOAgqRDmgpo2IaxuU0bMt74VQA0OPaTJO6nSp\nEOWAaD/h0j5iru1QRsfWqDbL0nDI1YndCZExRymgKomTOlSIgkASzMaK/E+XExCKUIgsBeMV1R52\nbYEyGvaUMtAT1vGBaVAhCjqTDzq/l8ooeKmCvI+jChAqREGn7hARjI5f8jvPVak3lIYu02ScJRq7\nFqKDFEhiYyRGuLJT40R+Jg7Jl8s1PpSGPS5P7laIjEkCu53akEMa96nb72d2RBmIhZz/+PqVIhYi\ny1bXBuSKaXvU7fczz1cVhvc9QRS9EG2HwvAkJh+gNJTQcWd+5cEabZaloc00mV6XBrgXImPiELzp\niocjZJDaw/kvs6mMTEuUvoMRzYBPg/PwiHshsmxxbUCumLZbe878yKOT1FPNwDbXBvhFiHbhoGD3\nRDBzO1GSKkZ+IgHmkRp0hNnwdJkm4zyNxh9CZHvPnKtyLoj2E65tc1P3Vxme18rp69Js6nT44rnz\nhxBZCqb3bNY29Yj8xB9r9PPIgC+eOz8J0V4KpJb1zB1EJakxCT/QGSL+xCTKXdvhU7pMkzng2gjw\nkxAZY4CNrs3IBZEY4ckHCiPmFXT+UEd/QqsxpmOzawMG8Y8QWV7DwbzbE8E833zExcuAkLyrjjLX\ndvgUA7zq2ohB/CVExgxgxSjwTNtNWbRXx5655Ilq+rTkR1p2uhzkOhR/CZFlIwQ/viIgc7eoELki\nCeY/6zWBMQObXBuQiv+EyJhuCsQrOuZ1oqJDPpzwUgW9+0tViNLQiQ+yqVPxnxBZ1lEAXlEkRnja\nHg1a55skmJun6LxlGdhkmoyvUhr8KUQF5BUteEVjFPlmdTW9O6I6wDUN/fisWQZ+FSLLOgpgVH5N\nB6X1rTgd2VxMDAjJn01REcrAy6bJ+C526V8hsl7Ri67NyAUnrFOvKF/cW0tvR4k2y9IwgE9z9fwr\nRJYNQJtrI7KlpoPShtbCmTrJr3SGiN/WoFnUGdhkmowv8/T8LUR2MOyTrs3IBYvX6a/0RPPrBgb6\ntRRsOuLYH3Zf4v8PzZj9+CgDdLyoVzSxbI3Sd38dFa7t8DEvmSbj26oQ/hciy7MQ/IDvSc8T0VpF\nuWdASP7DDI3DZaAHeMm1EZkIhhDZoR/PuDYjWyq7iMxpCb6g+o1fN9CnyYsZed40GV/3QAdDiACM\n2YLPskHHw+L1REsGgp+W4BfeKKP3d5O1SZaBg6bJvO7aiJEIjhBZniTgJWUjMcKL1xdGhQHX9AuJ\n787QToARWO3agNEQLCEyphN4BIIdZ5m7lYpJ7VpONlv+o5F+nZkjI5v8UvhsJIIlRADG7AbWujYj\nW055mpAOiB0/a6ro+YP2kmWiC9vJEwiCJ0QAxqzDTswYWKqPUnrcKxq4Hg97I/R/f7oWPBuBJ/w4\nlCMdwRQiy2NAh2sjsmHhK1RO0hk/xkSPkLh+FiGdwz4jr5kmE6iOneB+mMbEgAch2MXH3vEk4VBc\n52QfDQkw35lJTLvqM9INrHFtxFgJrhABGNOB9YwCS0UPkcXrg90TmC9unkLvS5XaJMtAEnjEr+PJ\nMhFsIQIwZjvwvGszsuGYzVQ07tPhH5m4u5buezQ4PRJrTZNpdW3EeAi+EMFg8PoF12Zkw2lPUlbW\nrflFw/FgDd0/n0qlazt8zk7TZNa7NmK8FIYQARjzAgGuX1SSIHTWY6DxorfzeDU9/zxNRWgEugh4\niKJwhAjAmLXYyo6BpKqT0mVr1CsaZE0VPT+coc2xEUgAD5smE+g4Y2EJEYAxzwOBdVFn7KJ83uuF\nMfV2NqyroOc7M7TI2Sh4LCjZ05koPCECMOY5fF72IBNLXqSycW/xBq/XVNHTPItyo1NFj8Szpsm0\nuDYiFxSmEAEY8ywBFqPTn6C89nDxZV7fXUv3DTOp0PnqR2STaTKB/X4PpXCFCAbFaDUBHCQbMshZ\njxKt6CyOHKMEmJ9NoUd7x0bFLgIyqn60iPHXPGsTg8gs4GII3jQz/WXEn3gvpr+8cLOJY5D83gz6\nn63WmNAo2Afc7/dCZ2OlOIQIQKQWWAHUuDZlrHRXEVu9HAbKCk+MOkPEm2aR2FxO1LUtAaAVuK/Q\nRAiKSYgAREqBC4F5bg0ZOz0VxJ5aUVhitKmM3htmUnq0ROtNj4L9WBEK9NjKdBSXEA0icjJwBgQr\nINpbTvyp95DsrwheEzOVBJg76um5vUHjQaPkAFaECjbHrDiFCEBkOtY7qnZsyZjoKyf+1HKSfZXB\nFKOOMLHvzCD5aoU2xUbJXuDBQhYhKGYhAhApAd4BnESAvKP+KInVy4n3VAfrYX6xgp7vzSDaE9am\n2CjZik1YLPhKnsUtRIOINAAgHQW3AAAHeElEQVTnAw2uTRkt8TDJ5y6gv22q/3uaOsLEfjKV+DPa\nKzYWNpgmE7i6QuNFhWgQEcF6Ru+AYMwMYcBsPJ3enQv8OR4rAeaBWnr/o5EynQp6TDxjmsxG10bk\nExWioYhMAs4FZrk2ZbRsW0jPK6dQTsg/zcuWKH0/mk5oRzSYsSxHDACPmiaz07Uh+UaFKB0ixwKn\nAXWuTRkNB6fR98I5lMRL3Xpzh8MM3NpA/KFaf3ppPqYNG5Q+6toQF6gQjYQVpFOAetemjERfOfHn\nzyN+pD7/5VTbw8Ruryf2QK0OVh0Hm4EnCzFRcbSoEI0WkbnAqUCja1MyYcC8uozelkWUkwdB6AgT\nu6Oe+P21lOlA1TGTwMaDNrk2xDUqRGNFZDZWkKa6NiUTh6bQ98K5lMSiE9NU2xuh/+46Eg/UUq4C\nNC4OYrvmAz0lVq5QIRovNiHyeOAY8Oewi4FSEhvOYKB1dm66zWOQfK6Kvv+ZTFjHho2bJLak8TrT\npA/fICpE2WKTIucCC4DZ+LC0ysFp9K0/i/B4R/DvL2HgD3XE/1ijyYhZ0ob1gg67NsRvqBDlEpEy\n4FjgOHzWdIuHSW46ld6d86kYKXaUBLO7lP6nq0k8UU1kl3bBZ0scO8vMxmLIkh4PKkQThc1HmgvM\nAKbjk1pIHZPpf+lMTGft23vW4pB8vZz+1dWYJ6sp7SgJRlJnAGgB1pgm0+XaED+jQpQPbNZ2A1aQ\npgJTwO3I853H0nbvuZinJ1H2UgUlr5cR0fnkc8oh4OmgTniYb1SIXCFSiRWkGmwFgKqUv7nyRhJA\nj7d0A+3Y3ppDGNMjzRICFmN7AXUcWG5oB14olKL2+UKFyI/YWFOqMA0GmSXNAnZ4QDdviU4PxvSN\n6nTNEsEK0lLQjOhx0oGNA7Vob9jYUSFS3sTzkBYCJxPAkrqOOIydLWarCtD4USFS/gRpFsHmR52E\nz3r/fEIS2A68rDGg3KBCpGREmqUOWIRNScj7GDaf0Qu8hp1TrOhn480lKkTKqJBmCWMnHViETUko\nlmEdMaz3swXYo3lAE4MKkTJmpFnKsDlSxwAzoeCyrePAHqz47CjmUfH5QoVIyQqvx22Ot8zAcX5U\nFnRgZ1DdBewzTSbh2J6iQoVIySnSLJN4K5vcr8JksPk+B7xlt2Y+u0WFSJlQpFkqgMnYwnL13v+1\n5G9w8ABwxFsOY4XnoDa3/IUKkZJ3vHylSmyy5uBS6S2l2ATOwaUUm2k++EU12O7zBDaQ3DfM0oUn\nPqZpdEmdiltUiBwiIiuBTxtjznVti6K4RAc5TjAi8piIHBSRoyLykoh8cJT7vU9EnhKRDhFpFZGf\ni0igZqVVlNGiQjTxfAGYboyZBHwW+I3Y6o4jUQN8GxvwXYyd3ugHE2alojikaIVIRLaLyFdEZIOI\ndIvIzSIyVUTuF5FOEXlYROq8bf/L80qOiMgTInJiynFuEZF/E5GHvP0eF1toHwBjzAZj3gyMGmzc\nY3Yam37geUE1xpjbjDEPGGN6jDHtwM+BcybshiiKQ4pWiDyuBJZjB3p+ALgfuBZbOygEfN7b7n7s\nEIcp2HrDtw45zkeBb3n7rR+6XkTuFZE+4Fngf4G1Q9aHROTn2NHv7zHGHBnG1vOBV8ZzkYrid4q9\nCt8/G2P2A4jIk8ABY8w67/VdwEUAxphfDO4gIquAds9rGRSMPxhjnvDWfwM4IiKzjTG7vP3fLyIR\n4GJgkTFvGyYQAW7HfhYfMMYMDDVSRJYDVwNn5u7SFcU/FLtHtD/l/95hXleJSFhEvisiW0XkKHbc\nEVjvZ5Bdg/8YY7qwRdJnpJ7IGBMzxtwPrBCRP0tZtQD4INCcRoTOAm4D/twY88ZYL1BRgkCxC9Fo\n+AhWKC7GBpDnee+nDvp8M+YjIlXYpL29aY5XAsxPef0q8EngfhE5PnVDETkFuBv4C2PMI+O/BEXx\nNypEI1MN9GOzciuA7wyzzaUicq6IlGJjRc8aY3aJyCIRuUREykUkIiIfw8Z6Hk/d2RhzOzY29bCI\nzAcQkSXAA8BfG2PumbCrUxQfoEI0Mr8CdmBHY28C1gyzzW1AE7ZJdho2eA3Wa1qFN6wA25V/lTHm\nxaEHMMb8Evgm8KiIzAO+jJ3e+mYR6fIWDVb7ABFZKSJPubajkNDM6iwRkVuA3caY61zbouQOEXkM\nWAJEgW3A9caY33vrVpImI15E3gX8E7a5ngCeAD5njNmTJ9MDiXpEijI8401E3QSsMMbUYjssNgP/\nOnFmFgYqREqgCEAi6n5jTGpHRQLbM6pkQIUoS4wxK7VZlnd8nYgqInNEpAObAvK3wPezvuICp9gT\nGpVg4utEVGPMTqBWRCYDn8EW3FcyoB6REkR8n4jq7dsG/BL4vYjoj34GVIiUQsVZIuow+00BJo3B\n9qJDhUgpVFwlol4hIsd78aNG4EfAOs87UtKgQqQUKq4SUWdiM+I7gY3YsraX5+aSChdNaFSKEk1E\n9RfqESmK4hwVIkVRnKNNM0VRnKMekaIozlEhUhTFOSpEiqI4R4VIURTnqBApiuIcFSJFUZzz/wEA\nxlcJ7KNpCQAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1af5163ac8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "from matplotlib_venn import venn2, venn2_circles\n",
    "venn2([set(MAP3Kwords_list[0]), set(MAP3Kwords_list[1])],\n",
    "      set_labels = ('map3k2', 'map3k3'))\n",
    "plt.title(\"MAP3K2, 3comparison\")\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('activ', 369),\n",
       " ('kinas', 337),\n",
       " ('mekk2', 323),\n",
       " ('cell', 250),\n",
       " ('protein', 155),\n",
       " ('express', 138),\n",
       " ('regul', 138),\n",
       " ('signal', 136),\n",
       " ('pathway', 118),\n",
       " ('gene', 103)]"
      ]
     },
     "execution_count": 145,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#we had a LOT of intersecting words. how to weight them??? try frequency distributions\n",
    "#if that doesn't work, you can scroll up and see topic modeling. \n",
    "MAP3K2_freq_dist= nltk.FreqDist(MAP3Kwords_list[0])\n",
    "MAP3K3_freq_dist= nltk.FreqDist(MAP3Kwords_list[1])\n",
    "MAP3K5_freq_dist= nltk.FreqDist(MAP3Kwords_list[2])\n",
    "MAP3K6_freq_dist= nltk.FreqDist(MAP3Kwords_list[3])\n",
    "MAP3K15_freq_dist= nltk.FreqDist(MAP3Kwords_list[4])\n",
    "\n",
    "MAP3K2_freq_dist.most_common(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('activ', 0.02967669293871642),\n",
       " ('kinas', 0.02710310439118546),\n",
       " ('mekk2', 0.02597715940164066),\n",
       " ('cell', 0.020106160527585654),\n",
       " ('protein', 0.012465819527103104),\n",
       " ('express', 0.01109860061122728),\n",
       " ('regul', 0.01109860061122728),\n",
       " ('signal', 0.010937751327006594),\n",
       " ('pathway', 0.009490107769020428),\n",
       " ('gene', 0.008283738137365288)]"
      ]
     },
     "execution_count": 148,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "total_MAP3K2_words=MAP3K2_freq_dist.N()\n",
    "for word in MAP3K2_freq_dist:\n",
    "    MAP3K2_freq_dist[word] /= float(total_MAP3K2_words) # value /= 5 is value = value/5\n",
    "MAP3K2_freq_dist.most_common(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "metadata": {},
   "outputs": [],
   "source": [
    "#normalize freq dist. not going to think through a for-loop at 2 AM\n",
    "\n",
    "\n",
    "total_MAP3K3_words=MAP3K3_freq_dist.N()\n",
    "for word in MAP3K3_freq_dist:\n",
    "    MAP3K3_freq_dist[word] /= float(total_MAP3K3_words) # value /= 5 is value = value/5\n",
    "\n",
    "total_MAP3K5_words=MAP3K5_freq_dist.N()\n",
    "for word in MAP3K5_freq_dist:\n",
    "    MAP3K5_freq_dist[word] /= float(total_MAP3K5_words) # value /= 5 is value = value/5\n",
    "\n",
    "\n",
    "total_MAP3K6_words=MAP3K6_freq_dist.N()\n",
    "for word in MAP3K6_freq_dist:\n",
    "    MAP3K6_freq_dist[word] /= float(total_MAP3K6_words) # value /= 5 is value = value/5\n",
    "\n",
    "total_MAP3K15_words=MAP3K15_freq_dist.N()\n",
    "for word in MAP3K15_freq_dist:\n",
    "    MAP3K15_freq_dist[word] /= float(total_MAP3K15_words) # value /= 5 is value = value/5\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>word</th>\n",
       "      <th>freq</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>activ</td>\n",
       "      <td>0.029677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>kinas</td>\n",
       "      <td>0.027103</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>mekk2</td>\n",
       "      <td>0.025977</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>cell</td>\n",
       "      <td>0.020106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>protein</td>\n",
       "      <td>0.012466</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>express</td>\n",
       "      <td>0.011099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>regul</td>\n",
       "      <td>0.011099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>signal</td>\n",
       "      <td>0.010938</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>pathway</td>\n",
       "      <td>0.009490</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>gene</td>\n",
       "      <td>0.008284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>mapk</td>\n",
       "      <td>0.008284</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>cancer</td>\n",
       "      <td>0.007882</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>titl</td>\n",
       "      <td>0.007077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>abstract</td>\n",
       "      <td>0.007077</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>erk5</td>\n",
       "      <td>0.006675</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>induc</td>\n",
       "      <td>0.006354</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>map3k2</td>\n",
       "      <td>0.006032</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>inhibit</td>\n",
       "      <td>0.005710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>function</td>\n",
       "      <td>0.005308</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>jnk</td>\n",
       "      <td>0.005228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>phosphoryl</td>\n",
       "      <td>0.005067</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>target</td>\n",
       "      <td>0.004986</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>mitogen</td>\n",
       "      <td>0.004745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>associ</td>\n",
       "      <td>0.004665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>use</td>\n",
       "      <td>0.004584</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>role</td>\n",
       "      <td>0.004504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>studi</td>\n",
       "      <td>0.004504</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>domain</td>\n",
       "      <td>0.004102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>interact</td>\n",
       "      <td>0.004021</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>factor</td>\n",
       "      <td>0.003941</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>relat</td>\n",
       "      <td>0.002413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>apoptosi</td>\n",
       "      <td>0.002413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>erk</td>\n",
       "      <td>0.002332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>type</td>\n",
       "      <td>0.002252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>promot</td>\n",
       "      <td>0.002252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>complex</td>\n",
       "      <td>0.002252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>tissu</td>\n",
       "      <td>0.002252</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>control</td>\n",
       "      <td>0.002171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>lung</td>\n",
       "      <td>0.002171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>howev</td>\n",
       "      <td>0.002171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>format</td>\n",
       "      <td>0.002171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>includ</td>\n",
       "      <td>0.002091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>erk1</td>\n",
       "      <td>0.002091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>map</td>\n",
       "      <td>0.002091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>pb1</td>\n",
       "      <td>0.002091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>lysin</td>\n",
       "      <td>0.002011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>data</td>\n",
       "      <td>0.002011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>indic</td>\n",
       "      <td>0.002011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>mechan</td>\n",
       "      <td>0.002011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>extracellular</td>\n",
       "      <td>0.002011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>negat</td>\n",
       "      <td>0.001930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>reveal</td>\n",
       "      <td>0.001930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>ubiquitin</td>\n",
       "      <td>0.001930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>methyl</td>\n",
       "      <td>0.001930</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>suppress</td>\n",
       "      <td>0.001850</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>high</td>\n",
       "      <td>0.001850</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>novel</td>\n",
       "      <td>0.001850</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>report</td>\n",
       "      <td>0.001850</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>reduc</td>\n",
       "      <td>0.001769</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>differ</td>\n",
       "      <td>0.001769</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             word      freq\n",
       "0           activ  0.029677\n",
       "1           kinas  0.027103\n",
       "2           mekk2  0.025977\n",
       "3            cell  0.020106\n",
       "4         protein  0.012466\n",
       "5         express  0.011099\n",
       "6           regul  0.011099\n",
       "7          signal  0.010938\n",
       "8         pathway  0.009490\n",
       "9            gene  0.008284\n",
       "10           mapk  0.008284\n",
       "11         cancer  0.007882\n",
       "12           titl  0.007077\n",
       "13       abstract  0.007077\n",
       "14           erk5  0.006675\n",
       "15          induc  0.006354\n",
       "16         map3k2  0.006032\n",
       "17        inhibit  0.005710\n",
       "18       function  0.005308\n",
       "19            jnk  0.005228\n",
       "20     phosphoryl  0.005067\n",
       "21         target  0.004986\n",
       "22        mitogen  0.004745\n",
       "23         associ  0.004665\n",
       "24            use  0.004584\n",
       "25           role  0.004504\n",
       "26          studi  0.004504\n",
       "27         domain  0.004102\n",
       "28       interact  0.004021\n",
       "29         factor  0.003941\n",
       "..            ...       ...\n",
       "70          relat  0.002413\n",
       "71       apoptosi  0.002413\n",
       "72            erk  0.002332\n",
       "73           type  0.002252\n",
       "74         promot  0.002252\n",
       "75        complex  0.002252\n",
       "76          tissu  0.002252\n",
       "77        control  0.002171\n",
       "78           lung  0.002171\n",
       "79          howev  0.002171\n",
       "80         format  0.002171\n",
       "81         includ  0.002091\n",
       "82           erk1  0.002091\n",
       "83            map  0.002091\n",
       "84            pb1  0.002091\n",
       "85          lysin  0.002011\n",
       "86           data  0.002011\n",
       "87          indic  0.002011\n",
       "88         mechan  0.002011\n",
       "89  extracellular  0.002011\n",
       "90          negat  0.001930\n",
       "91         reveal  0.001930\n",
       "92      ubiquitin  0.001930\n",
       "93         methyl  0.001930\n",
       "94       suppress  0.001850\n",
       "95           high  0.001850\n",
       "96          novel  0.001850\n",
       "97         report  0.001850\n",
       "98          reduc  0.001769\n",
       "99         differ  0.001769\n",
       "\n",
       "[100 rows x 2 columns]"
      ]
     },
     "execution_count": 248,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "pd.DataFrame(MAP3K2_freq_dist.most_common(100), columns=['word', 'freq'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>word</th>\n",
       "      <th>freq</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>activ</td>\n",
       "      <td>0.025528</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>mekk3</td>\n",
       "      <td>0.023725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>cell</td>\n",
       "      <td>0.020178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>kinas</td>\n",
       "      <td>0.019771</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>protein</td>\n",
       "      <td>0.013781</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>express</td>\n",
       "      <td>0.012851</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>gene</td>\n",
       "      <td>0.010002</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>signal</td>\n",
       "      <td>0.009478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>induc</td>\n",
       "      <td>0.008839</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>pathway</td>\n",
       "      <td>0.007676</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      word      freq\n",
       "0    activ  0.025528\n",
       "1    mekk3  0.023725\n",
       "2     cell  0.020178\n",
       "3    kinas  0.019771\n",
       "4  protein  0.013781\n",
       "5  express  0.012851\n",
       "6     gene  0.010002\n",
       "7   signal  0.009478\n",
       "8    induc  0.008839\n",
       "9  pathway  0.007676"
      ]
     },
     "execution_count": 137,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(MAP3K3_freq_dist.most_common(10), columns=['word', 'freq'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>word</th>\n",
       "      <th>freq</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ask1</td>\n",
       "      <td>0.030907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>activ</td>\n",
       "      <td>0.024812</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>cell</td>\n",
       "      <td>0.022256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>kinas</td>\n",
       "      <td>0.018154</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>apoptosi</td>\n",
       "      <td>0.016817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>induc</td>\n",
       "      <td>0.016699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>signal</td>\n",
       "      <td>0.014772</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>regul</td>\n",
       "      <td>0.012046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>protein</td>\n",
       "      <td>0.010853</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>express</td>\n",
       "      <td>0.008009</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       word      freq\n",
       "0      ask1  0.030907\n",
       "1     activ  0.024812\n",
       "2      cell  0.022256\n",
       "3     kinas  0.018154\n",
       "4  apoptosi  0.016817\n",
       "5     induc  0.016699\n",
       "6    signal  0.014772\n",
       "7     regul  0.012046\n",
       "8   protein  0.010853\n",
       "9   express  0.008009"
      ]
     },
     "execution_count": 138,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(MAP3K5_freq_dist.most_common(10), columns=['word', 'freq'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>word</th>\n",
       "      <th>freq</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>gene</td>\n",
       "      <td>0.021266</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>map3k6</td>\n",
       "      <td>0.017804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>cancer</td>\n",
       "      <td>0.016320</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>express</td>\n",
       "      <td>0.013353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>gastric</td>\n",
       "      <td>0.012859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>respons</td>\n",
       "      <td>0.011869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>kinas</td>\n",
       "      <td>0.010880</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>mutat</td>\n",
       "      <td>0.010386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>mice</td>\n",
       "      <td>0.008408</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>cdh1</td>\n",
       "      <td>0.007913</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      word      freq\n",
       "0     gene  0.021266\n",
       "1   map3k6  0.017804\n",
       "2   cancer  0.016320\n",
       "3  express  0.013353\n",
       "4  gastric  0.012859\n",
       "5  respons  0.011869\n",
       "6    kinas  0.010880\n",
       "7    mutat  0.010386\n",
       "8     mice  0.008408\n",
       "9     cdh1  0.007913"
      ]
     },
     "execution_count": 139,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(MAP3K6_freq_dist.most_common(10), columns=['word', 'freq'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>word</th>\n",
       "      <th>freq</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>kinas</td>\n",
       "      <td>0.022831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>gene</td>\n",
       "      <td>0.021005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>protein</td>\n",
       "      <td>0.016438</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>express</td>\n",
       "      <td>0.014612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ask3</td>\n",
       "      <td>0.014612</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>pathway</td>\n",
       "      <td>0.013699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>stress</td>\n",
       "      <td>0.013699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>human</td>\n",
       "      <td>0.012785</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>cell</td>\n",
       "      <td>0.011872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>regul</td>\n",
       "      <td>0.011872</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      word      freq\n",
       "0    kinas  0.022831\n",
       "1     gene  0.021005\n",
       "2  protein  0.016438\n",
       "3  express  0.014612\n",
       "4     ask3  0.014612\n",
       "5  pathway  0.013699\n",
       "6   stress  0.013699\n",
       "7    human  0.012785\n",
       "8     cell  0.011872\n",
       "9    regul  0.011872"
      ]
     },
     "execution_count": 140,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(MAP3K15_freq_dist.most_common(10), columns=['word', 'freq'])"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
